setup.cfg

```diff
@@ -65,6 +65,6 @@ multi_line_output = 3
 
 [flake8]
 extend-ignore = E203, E501, E722, B950, E266
-select = C,E,F,W,T,B,B9,E711
+select = C,E,F,W,T,B,B9,E711,D103
```

Comment:
I think this was the only docs warning being used? It's not close to supporting all the warnings. :P

Reply:
<details>
<summary>Details:</summary>

```
src/uproot/model.py:3:1: D205 1 blank line required between summary line and description
src/uproot/model.py:3:1: D400 First line should end with a period
src/uproot/model.py:58:1: D205 1 blank line required between summary line and description
src/uproot/model.py:58:1: D400 First line should end with a period
src/uproot/model.py:58:1: D401 First line should be in imperative mood
src/uproot/model.py:75:1: D205 1 blank line required between summary line and description
src/uproot/model.py:75:1: D400 First line should end with a period
src/uproot/model.py:75:1: D401 First line should be in imperative mood
src/uproot/model.py:125:1: D205 1 blank line required between summary line and description
src/uproot/model.py:125:1: D400 First line should end with a period
src/uproot/model.py:125:1: D401 First line should be in imperative mood
src/uproot/model.py:139:1: D205 1 blank line required between summary line and description
src/uproot/model.py:139:1: D400 First line should end with a period
src/uproot/model.py:139:1: D401 First line should be in imperative mood
src/uproot/model.py:172:1: D205 1 blank line required between summary line and description
src/uproot/model.py:172:1: D400 First line should end with a period
src/uproot/model.py:172:1: D401 First line should be in imperative mood
src/uproot/model.py:206:1: D401 First line should be in imperative mood
src/uproot/model.py:226:1: D401 First line should be in imperative mood
src/uproot/model.py:274:1: D205 1 blank line required between summary line and description
src/uproot/model.py:274:1: D400 First line should end with a period
src/uproot/model.py:274:1: D401 First line should be in imperative mood
src/uproot/model.py:289:1: D205 1 blank line required between summary line and description
src/uproot/model.py:289:1: D400 First line should end with a period
src/uproot/model.py:289:1: D401 First line should be in imperative mood
src/uproot/model.py:302:1: D205 1 blank line required between summary line and description
src/uproot/model.py:302:1: D400 First line should end with a period
src/uproot/model.py:362:1: D105 Missing docstring in magic method
src/uproot/model.py:369:1: D105 Missing docstring in magic method
src/uproot/model.py:374:1: D105 Missing docstring in magic method
src/uproot/model.py:380:1: D401 First line should be in imperative mood; try rephrasing
src/uproot/model.py:391:1: D205 1 blank line required between summary line and description
src/uproot/model.py:391:1: D400 First line should end with a period
src/uproot/model.py:391:1: D401 First line should be in imperative mood; try rephrasing
src/uproot/model.py:403:1: D401 First line should be in imperative mood; try rephrasing
src/uproot/model.py:414:1: D205 1 blank line required between summary line and description
src/uproot/model.py:414:1: D400 First line should end with a period
src/uproot/model.py:414:1: D401 First line should be in imperative mood; try rephrasing
src/uproot/model.py:422:1: D205 1 blank line required between summary line and description
src/uproot/model.py:422:1: D400 First line should end with a period
src/uproot/model.py:422:1: D401 First line should be in imperative mood; try rephrasing
src/uproot/model.py:430:1: D205 1 blank line required between summary line and description
src/uproot/model.py:430:1: D400 First line should end with a period
src/uproot/model.py:430:1: D401 First line should be in imperative mood
src/uproot/model.py:439:1: D205 1 blank line required between summary line and description
src/uproot/model.py:439:1: D400 First line should end with a period
src/uproot/model.py:439:1: D401 First line should be in imperative mood; try rephrasing
src/uproot/model.py:451:1: D205 1 blank line required between summary line and description
src/uproot/model.py:451:1: D400 First line should end with a period
src/uproot/model.py:451:1: D401 First line should be in imperative mood; try rephrasing
src/uproot/model.py:459:1: D205 1 blank line required between summary line and description
src/uproot/model.py:459:1: D400 First line should end with a period
src/uproot/model.py:459:1: D401 First line should be in imperative mood; try rephrasing
src/uproot/model.py:459:1: D402 First line should not be the function's "signature"
src/uproot/model.py:472:1: D205 1 blank line required between summary line and description
src/uproot/model.py:472:1: D400 First line should end with a period
src/uproot/model.py:472:1: D401 First line should be in imperative mood; try rephrasing
src/uproot/model.py:481:1: D205 1 blank line required between summary line and description
src/uproot/model.py:481:1: D400 First line should end with a period
src/uproot/model.py:481:1: D401 First line should be in imperative mood; try rephrasing
src/uproot/model.py:492:1: D205 1 blank line required between summary line and description
src/uproot/model.py:492:1: D400 First line should end with a period
src/uproot/model.py:492:1: D401 First line should be in imperative mood
src/uproot/model.py:505:1: D205 1 blank line required between summary line and description
src/uproot/model.py:505:1: D400 First line should end with a period
src/uproot/model.py:544:1: D205 1 blank line required between summary line and description
src/uproot/model.py:544:1: D400 First line should end with a period
src/uproot/model.py:563:1: D205 1 blank line required between summary line and description
src/uproot/model.py:563:1: D400 First line should end with a period
src/uproot/model.py:563:1: D401 First line should be in imperative mood
src/uproot/model.py:583:1: D401 First line should be in imperative mood
src/uproot/model.py:598:1: D205 1 blank line required between summary line and description
src/uproot/model.py:598:1: D400 First line should end with a period
src/uproot/model.py:598:1: D401 First line should be in imperative mood; try rephrasing
src/uproot/model.py:629:1: D205 1 blank line required between summary line and description
src/uproot/model.py:629:1: D400 First line should end with a period
src/uproot/model.py:629:1: D401 First line should be in imperative mood; try rephrasing
src/uproot/model.py:639:1: D205 1 blank line required between summary line and description
src/uproot/model.py:639:1: D400 First line should end with a period
src/uproot/model.py:668:1: D205 1 blank line required between summary line and description
src/uproot/model.py:668:1: D400 First line should end with a period
src/uproot/model.py:691:1: D205 1 blank line required between summary line and description
src/uproot/model.py:691:1: D400 First line should end with a period
src/uproot/model.py:691:1: D401 First line should be in imperative mood
src/uproot/model.py:712:1: D205 1 blank line required between summary line and description
src/uproot/model.py:712:1: D400 First line should end with a period
src/uproot/model.py:712:1: D401 First line should be in imperative mood
src/uproot/model.py:730:1: D205 1 blank line required between summary line and description
src/uproot/model.py:730:1: D400 First line should end with a period
src/uproot/model.py:824:1: D205 1 blank line required between summary line and description
src/uproot/model.py:824:1: D400 First line should end with a period
src/uproot/model.py:848:1: D205 1 blank line required between summary line and description
src/uproot/model.py:848:1: D400 First line should end with a period
src/uproot/model.py:866:1: D205 1 blank line required between summary line and description
src/uproot/model.py:866:1: D400 First line should end with a period
src/uproot/model.py:897:1: D205 1 blank line required between summary line and description
src/uproot/model.py:897:1: D400 First line should end with a period
src/uproot/model.py:925:1: D205 1 blank line required between summary line and description
src/uproot/model.py:925:1: D400 First line should end with a period
src/uproot/model.py:925:1: D401 First line should be in imperative mood
src/uproot/model.py:932:1: D205 1 blank line required between summary line and description
src/uproot/model.py:932:1: D400 First line should end with a period
src/uproot/model.py:932:1: D401 First line should be in imperative mood
src/uproot/model.py:940:1: D205 1 blank line required between summary line and description
src/uproot/model.py:940:1: D400 First line should end with a period
src/uproot/model.py:940:1: D401 First line should be in imperative mood
src/uproot/model.py:948:1: D205 1 blank line required between summary line and description
src/uproot/model.py:948:1: D400 First line should end with a period
src/uproot/model.py:948:1: D401 First line should be in imperative mood
src/uproot/model.py:1006:1: D205 1 blank line required between summary line and description
src/uproot/model.py:1006:1: D400 First line should end with a period
src/uproot/model.py:1027:1: D205 1 blank line required between summary line and description
src/uproot/model.py:1027:1: D400 First line should end with a period
src/uproot/model.py:1041:1: D205 1 blank line required between summary line and description
src/uproot/model.py:1041:1: D400 First line should end with a period
src/uproot/model.py:1070:1: D105 Missing docstring in magic method
src/uproot/model.py:1085:1: D105 Missing docstring in magic method
src/uproot/model.py:1091:1: D205 1 blank line required between summary line and description
src/uproot/model.py:1091:1: D400 First line should end with a period
src/uproot/model.py:1114:1: D205 1 blank line required between summary line and description
src/uproot/model.py:1114:1: D400 First line should end with a period
src/uproot/model.py:1144:1: D205 1 blank line required between summary line and description
src/uproot/model.py:1144:1: D400 First line should end with a period
src/uproot/model.py:1169:1: D401 First line should be in imperative mood
src/uproot/model.py:1179:1: D205 1 blank line required between summary line and description
src/uproot/model.py:1179:1: D400 First line should end with a period
src/uproot/model.py:1179:1: D401 First line should be in imperative mood
src/uproot/model.py:1187:1: D401 First line should be in imperative mood
src/uproot/model.py:1230:1: D205 1 blank line required between summary line and description
src/uproot/model.py:1230:1: D400 First line should end with a period
src/uproot/model.py:1304:1: D205 1 blank line required between summary line and description
src/uproot/model.py:1304:1: D400 First line should end with a period
src/uproot/model.py:1334:1: D205 1 blank line required between summary line and description
src/uproot/model.py:1334:1: D400 First line should end with a period
src/uproot/model.py:1342:1: D205 1 blank line required between summary line and description
src/uproot/model.py:1342:1: D400 First line should end with a period
src/uproot/model.py:1342:1: D401 First line should be in imperative mood; try rephrasing
src/uproot/model.py:1360:1: D401 First line should be in imperative mood; try rephrasing
src/uproot/model.py:1367:1: D105 Missing docstring in magic method
src/uproot/model.py:1373:1: D205 1 blank line required between summary line and description
src/uproot/model.py:1373:1: D400 First line should end with a period
src/uproot/model.py:1430:1: D205 1 blank line required between summary line and description
src/uproot/model.py:1430:1: D400 First line should end with a period
src/uproot/model.py:1451:1: D102 Missing docstring in public method
src/uproot/model.py:1469:1: D205 1 blank line required between summary line and description
src/uproot/model.py:1469:1: D400 First line should end with a period
src/uproot/model.py:1476:1: D205 1 blank line required between summary line and description
src/uproot/model.py:1476:1: D400 First line should end with a period
src/uproot/model.py:1476:1: D401 First line should be in imperative mood; try rephrasing
src/uproot/model.py:1496:1: D401 First line should be in imperative mood; try rephrasing
src/uproot/model.py:1506:1: D205 1 blank line required between summary line and description
src/uproot/model.py:1506:1: D400 First line should end with a period
src/uproot/model.py:1563:1: D205 1 blank line required between summary line and description
src/uproot/model.py:1563:1: D400 First line should end with a period
src/uproot/model.py:1584:1: D102 Missing docstring in public method
src/uproot/model.py:1600:1: D105 Missing docstring in magic method
src/uproot/model.py:1607:1: D205 1 blank line required between summary line and description
src/uproot/model.py:1607:1: D400 First line should end with a period
src/uproot/model.py:1615:1: D105 Missing docstring in magic method
src/uproot/source/http.py:31:1: D205 1 blank line required between summary line and description
src/uproot/source/http.py:31:1: D400 First line should end with a period
src/uproot/source/http.py:57:1: D205 1 blank line required between summary line and description
src/uproot/source/http.py:57:1: D400 First line should end with a period
src/uproot/source/http.py:57:1: D401 First line should be in imperative mood
src/uproot/source/http.py:68:1: D205 1 blank line required between summary line and description
src/uproot/source/http.py:68:1: D400 First line should end with a period
src/uproot/source/http.py:68:1: D401 First line should be in imperative mood
src/uproot/source/http.py:84:1: D205 1 blank line required between summary line and description
src/uproot/source/http.py:84:1: D400 First line should end with a period
src/uproot/source/http.py:144:1: D205 1 blank line required between summary line and description
src/uproot/source/http.py:144:1: D400 First line should end with a period
src/uproot/source/http.py:156:1: D107 Missing docstring in __init__
src/uproot/source/http.py:164:1: D200 One-line docstring should fit on one line with quotes
src/uproot/source/http.py:164:1: D401 First line should be in imperative mood; try rephrasing
src/uproot/source/http.py:171:1: D200 One-line docstring should fit on one line with quotes
src/uproot/source/http.py:171:1: D401 First line should be in imperative mood; try rephrasing
src/uproot/source/http.py:178:1: D200 One-line docstring should fit on one line with quotes
src/uproot/source/http.py:178:1: D400 First line should end with a period
src/uproot/source/http.py:178:1: D401 First line should be in imperative mood
src/uproot/source/http.py:183:1: D105 Missing docstring in magic method
src/uproot/source/http.py:186:1: D105 Missing docstring in magic method
src/uproot/source/http.py:190:1: D205 1 blank line required between summary line and description
src/uproot/source/http.py:190:1: D400 First line should end with a period
src/uproot/source/http.py:241:1: D205 1 blank line required between summary line and description
src/uproot/source/http.py:241:1: D400 First line should end with a period
src/uproot/source/http.py:266:1: D205 1 blank line required between summary line and description
src/uproot/source/http.py:266:1: D400 First line should end with a period
src/uproot/source/http.py:356:1: D205 1 blank line required between summary line and description
src/uproot/source/http.py:356:1: D400 First line should end with a period
src/uproot/source/http.py:356:1: D401 First line should be in imperative mood; try rephrasing
src/uproot/source/http.py:373:1: D205 1 blank line required between summary line and description
src/uproot/source/http.py:373:1: D400 First line should end with a period
src/uproot/source/http.py:373:1: D401 First line should be in imperative mood; try rephrasing
src/uproot/source/http.py:388:1: D205 1 blank line required between summary line and description
src/uproot/source/http.py:388:1: D400 First line should end with a period
src/uproot/source/http.py:388:1: D401 First line should be in imperative mood; try rephrasing
src/uproot/source/http.py:461:1: D205 1 blank line required between summary line and description
src/uproot/source/http.py:461:1: D400 First line should end with a period
src/uproot/source/http.py:461:1: D401 First line should be in imperative mood; try rephrasing
src/uproot/source/http.py:484:1: D205 1 blank line required between summary line and description
src/uproot/source/http.py:484:1: D400 First line should end with a period
src/uproot/source/http.py:484:1: D401 First line should be in imperative mood
src/uproot/source/http.py:533:1: D205 1 blank line required between summary line and description
src/uproot/source/http.py:533:1: D400 First line should end with a period
src/uproot/source/http.py:551:1: D107 Missing docstring in __init__
src/uproot/source/http.py:571:1: D105 Missing docstring in magic method
src/uproot/source/http.py:576:1: D105 Missing docstring in magic method
src/uproot/source/http.py:580:1: D105 Missing docstring in magic method
src/uproot/source/http.py:591:1: D102 Missing docstring in public method
src/uproot/source/http.py:601:1: D102 Missing docstring in public method
src/uproot/source/http.py:630:1: D205 1 blank line required between summary line and description
src/uproot/source/http.py:630:1: D400 First line should end with a period
src/uproot/source/http.py:630:1: D401 First line should be in imperative mood; try rephrasing
src/uproot/source/http.py:637:1: D102 Missing docstring in public method
src/uproot/source/http.py:640:1: D105 Missing docstring in magic method
src/uproot/source/http.py:643:1: D105 Missing docstring in magic method
src/uproot/source/http.py:650:1: D200 One-line docstring should fit on one line with quotes
src/uproot/source/http.py:650:1: D401 First line should be in imperative mood; try rephrasing
src/uproot/source/http.py:656:1: D102 Missing docstring in public method
src/uproot/source/http.py:665:1: D200 One-line docstring should fit on one line with quotes
src/uproot/source/http.py:665:1: D401 First line should be in imperative mood; try rephrasing
src/uproot/source/http.py:672:1: D200 One-line docstring should fit on one line with quotes
src/uproot/source/http.py:672:1: D400 First line should end with a period
src/uproot/source/http.py:672:1: D401 First line should be in imperative mood; try rephrasing
src/uproot/source/http.py:679:1: D205 1 blank line required between summary line and description
src/uproot/source/http.py:679:1: D400 First line should end with a period
src/uproot/source/http.py:696:1: D205 1 blank line required between summary line and description
src/uproot/source/http.py:696:1: D400 First line should end with a period
src/uproot/source/http.py:707:1: D107 Missing docstring in __init__
src/uproot/source/http.py:724:1: D200 One-line docstring should fit on one line with quotes
src/uproot/source/http.py:724:1: D401 First line should be in imperative mood; try rephrasing
src/uproot/source/http.py:730:1: D102 Missing docstring in public method
src/uproot/source/http.py:739:1: D200 One-line docstring should fit on one line with quotes
src/uproot/source/http.py:739:1: D401 First line should be in imperative mood; try rephrasing
src/uproot/source/http.py:746:1: D200 One-line docstring should fit on one line with quotes
src/uproot/source/http.py:746:1: D400 First line should end with a period
src/uproot/source/http.py:746:1: D401 First line should be in imperative mood; try rephrasing
src/uproot/behavior.py:3:1: D205 1 blank line required between summary line and description
src/uproot/behavior.py:16:1: D205 1 blank line required between summary line and description
src/uproot/behavior.py:16:1: D400 First line should end with a period
src/uproot/behavior.py:16:1: D401 First line should be in imperative mood
src/uproot/models/TObject.py:3:1: D200 One-line docstring should fit on one line with quotes
src/uproot/models/TObject.py:19:1: D200 One-line docstring should fit on one line with quotes
src/uproot/models/TObject.py:23:1: D102 Missing docstring in public method
src/uproot/models/TObject.py:26:1: D102 Missing docstring in public method
src/uproot/models/TObject.py:53:1: D102 Missing docstring in public method
src/uproot/models/TObject.py:68:1: D102 Missing docstring in public method
src/uproot/models/TObject.py:119:1: D105 Missing docstring in magic method
src/uproot/models/TObject.py:124:1: D102 Missing docstring in public method
src/uproot/models/TBasket.py:3:1: D205 1 blank line required between summary line and description
src/uproot/models/TBasket.py:3:1: D400 First line should end with a period
src/uproot/models/TBasket.py:32:1: D105 Missing docstring in magic method
src/uproot/models/TBasket.py:40:1: D205 1 blank line required between summary line and description
src/uproot/models/TBasket.py:40:1: D400 First line should end with a period
src/uproot/models/TBasket.py:40:1: D401 First line should be in imperative mood; try rephrasing
src/uproot/models/TBasket.py:51:1: D205 1 blank line required between summary line and description
src/uproot/models/TBasket.py:51:1: D400 First line should end with a period
src/uproot/models/TBasket.py:51:1: D401 First line should be in imperative mood; try rephrasing
src/uproot/models/TBasket.py:62:1: D205 1 blank line required between summary line and description
src/uproot/models/TBasket.py:62:1: D400 First line should end with a period
src/uproot/models/TBasket.py:62:1: D401 First line should be in imperative mood; try rephrasing
src/uproot/models/TBasket.py:73:1: D205 1 blank line required between summary line and description
src/uproot/models/TBasket.py:73:1: D400 First line should end with a period
src/uproot/models/TBasket.py:73:1: D401 First line should be in imperative mood; try rephrasing
src/uproot/models/TBasket.py:104:1: D205 1 blank line required between summary line and description
src/uproot/models/TBasket.py:104:1: D400 First line should end with a period
src/uproot/models/TBasket.py:104:1: D401 First line should be in imperative mood; try rephrasing
src/uproot/models/TBasket.py:124:1: D200 One-line docstring should fit on one line with quotes
src/uproot/models/TBasket.py:124:1: D401 First line should be in imperative mood; try rephrasing
src/uproot/models/TBasket.py:131:1: D200 One-line docstring should fit on one line with quotes
src/uproot/models/TBasket.py:131:1: D401 First line should be in imperative mood; try rephrasing
src/uproot/models/TBasket.py:138:1: D205 1 blank line required between summary line and description
src/uproot/models/TBasket.py:138:1: D400 First line should end with a period
src/uproot/models/TBasket.py:138:1: D401 First line should be in imperative mood; try rephrasing
src/uproot/models/TBasket.py:146:1: D200 One-line docstring should fit on one line with quotes
src/uproot/models/TBasket.py:146:1: D401 First line should be in imperative mood; try rephrasing
src/uproot/models/TBasket.py:153:1: D205 1 blank line required between summary line and description
src/uproot/models/TBasket.py:153:1: D400 First line should end with a period
src/uproot/models/TBasket.py:164:1: D401 First line should be in imperative mood; try rephrasing
src/uproot/models/TBasket.py:174:1: D205 1 blank line required between summary line and description
src/uproot/models/TBasket.py:174:1: D400 First line should end with a period
src/uproot/models/TBasket.py:174:1: D401 First line should be in imperative mood; try rephrasing
src/uproot/models/TBasket.py:185:1: D400 First line should end with a period
src/uproot/models/TBasket.py:202:1: D401 First line should be in imperative mood; try rephrasing
src/uproot/models/TBasket.py:209:1: D102 Missing docstring in public method
src/uproot/models/TBasket.py:212:1: D102 Missing docstring in public method
src/uproot/language/__init__.py:3:1: D205 1 blank line required between summary line and description
src/uproot/language/__init__.py:3:1: D400 First line should end with a period
src/uproot/language/__init__.py:14:1: D205 1 blank line required between summary line and description
src/uproot/language/__init__.py:14:1: D400 First line should end with a period
src/uproot/behaviors/TProfile3D.py:3:1: D200 One-line docstring should fit on one line with quotes
src/uproot/behaviors/TProfile3D.py:17:1: D200 One-line docstring should fit on one line with quotes
src/uproot/behaviors/TProfile3D.py:24:1: D102 Missing docstring in public method
src/uproot/behaviors/TProfile3D.py:27:1: D102 Missing docstring in public method
src/uproot/behaviors/TProfile3D.py:40:1: D102 Missing docstring in public method
src/uproot/behaviors/TProfile3D.py:44:1: D102 Missing docstring in public method
src/uproot/behaviors/TProfile3D.py:57:1: D102 Missing docstring in public method
src/uproot/behaviors/TProfile3D.py:116:1: D102 Missing docstring in public method
src/uproot/models/TGraph.py:3:1: D200 One-line docstring should fit on one line with quotes
src/uproot/models/TGraph.py:31:1: D200 One-line docstring should fit on one line with quotes
src/uproot/models/TGraph.py:37:1: D102 Missing docstring in public method
src/uproot/models/TGraph.py:107:1: D102 Missing docstring in public method
src/uproot/models/TGraph.py:192:1: D102 Missing docstring in public method
src/uproot/models/TGraph.py:244:1: D102 Missing docstring in public method
src/uproot/models/TGraph.py:376:1: D200 One-line docstring should fit on one line with quotes
src/uproot/models/TGraph.py:386:1: D200 One-line docstring should fit on one line with quotes
src/uproot/models/TGraph.py:392:1: D102 Missing docstring in public method
src/uproot/models/TGraph.py:423:1: D102 Missing docstring in public method
src/uproot/models/TGraph.py:452:1: D102 Missing docstring in public method
src/uproot/models/TGraph.py:480:1: D102 Missing docstring in public method
src/uproot/models/TGraph.py:576:1: D200 One-line docstring should fit on one line with quotes
src/uproot/models/TGraph.py:586:1: D200 One-line docstring should fit on one line with quotes
src/uproot/models/TGraph.py:592:1: D102 Missing docstring in public method
src/uproot/models/TGraph.py:635:1: D102 Missing docstring in public method
src/uproot/models/TGraph.py:678:1: D102 Missing docstring in public method
src/uproot/models/TGraph.py:712:1: D102 Missing docstring in public method
src/uproot/models/TGraph.py:828:1: D200 One-line docstring should fit on one line with quotes
src/uproot/models/TDatime.py:3:1: D200 One-line docstring should fit on one line with quotes
src/uproot/models/TDatime.py:19:1: D200 One-line docstring should fit on one line with quotes
src/uproot/models/TDatime.py:23:1: D102 Missing docstring in public method
src/uproot/models/TDatime.py:26:1: D102 Missing docstring in public method
src/uproot/models/TDatime.py:30:1: D102 Missing docstring in public method
src/uproot/models/TDatime.py:43:1: D102 Missing docstring in public method
src/uproot/interpretation/identify.py:3:1: D205 1 blank line required between summary line and description
src/uproot/interpretation/identify.py:3:1: D400 First line should end with a period
src/uproot/interpretation/identify.py:301:1: D205 1 blank line required between summary line and description
src/uproot/interpretation/identify.py:301:1: D400 First line should end with a period
src/uproot/interpretation/identify.py:1093:1: D205 1 blank line required between summary line and description
src/uproot/interpretation/identify.py:1093:1: D400 First line should end with a period
src/uproot/interpretation/identify.py:1135:1: D205 1 blank line required between summary line and description
src/uproot/interpretation/identify.py:1135:1: D400 First line should end with a period
src/uproot/interpretation/identify.py:1145:1: D205 1 blank line required between summary line and description
src/uproot/interpretation/identify.py:1145:1: D400 First line should end with a period
src/uproot/interpretation/identify.py:1158:1: D107 Missing docstring in __init__
src/uproot/interpretation/identify.py:1163:1: D105 Missing docstring in magic method
src/uproot/interpretation/identify.py:1166:1: D105 Missing docstring in magic method
src/uproot/interpretation/identify.py:1174:1: D102 Missing docstring in public method
src/uproot/interpretation/identify.py:1178:1: D102 Missing docstring in public method
src/uproot/interpretation/identify.py:1182:1: D102 Missing docstring in public method
src/uproot/interpretation/identify.py:1185:1: D102 Missing docstring in public method
src/uproot/interpretation/identify.py:1196:1: D102 Missing docstring in public method
src/uproot/interpretation/identify.py:1200:1: D102 Missing docstring in public method
src/uproot/interpretation/identify.py:1204:1: D102 Missing docstring in public method
src/uproot/interpretation/identify.py:1208:1: D102 Missing docstring in public method
src/uproot/interpretation/identify.py:1212:1: D102 Missing docstring in public method
src/uproot/interpretation/identify.py:1216:1: D102 Missing docstring in public method
src/uproot/interpretation/identify.py:1220:1: D102 Missing docstring in public method
src/uproot/interpretation/identify.py:1224:1: D102 Missing docstring in public method
src/uproot/interpretation/identify.py:1228:1: D102 Missing docstring in public method
src/uproot/interpretation/identify.py:1232:1: D102 Missing docstring in public method
src/uproot/interpretation/identify.py:1236:1: D102 Missing docstring in public method
src/uproot/interpretation/identify.py:1240:1: D102 Missing docstring in public method
src/uproot/interpretation/identify.py:1244:1: D102 Missing docstring in public method
src/uproot/behaviors/TAxis.py:3:1: D200 One-line docstring should fit on one line with quotes
src/uproot/behaviors/TAxis.py:19:1: D107 Missing docstring in __init__
src/uproot/behaviors/TAxis.py:22:1: D105 Missing docstring in magic method
src/uproot/behaviors/TAxis.py:27:1: D200 One-line docstring should fit on one line with quotes
src/uproot/behaviors/TAxis.py:27:1: D401 First line should be in imperative mood; try rephrasing
src/uproot/behaviors/TAxis.py:34:1: D200 One-line docstring should fit on one line with quotes
src/uproot/behaviors/TAxis.py:34:1: D401 First line should be in imperative mood; try rephrasing
src/uproot/behaviors/TAxis.py:43:1: D200 One-line docstring should fit on one line with quotes
src/uproot/behaviors/TAxis.py:48:1: D200 One-line docstring should fit on one line with quotes
src/uproot/behaviors/TAxis.py:48:1: D401 First line should be in imperative mood; try rephrasing
src/uproot/behaviors/TAxis.py:54:1: D401 First line should be in imperative mood
src/uproot/behaviors/TAxis.py:76:1: D200 One-line docstring should fit on one line with quotes
src/uproot/behaviors/TAxis.py:89:1: D205 1 blank line required between summary line and description
src/uproot/behaviors/TAxis.py:89:1: D400 First line should end with a period
src/uproot/behaviors/TAxis.py:89:1: D401 First line should be in imperative mood
src/uproot/behaviors/TAxis.py:100:1: D200 One-line docstring should fit on one line with quotes
src/uproot/behaviors/TAxis.py:124:1: D200 One-line docstring should fit on one line with quotes
src/uproot/behaviors/TAxis.py:124:1: D401 First line should be in imperative mood; try rephrasing
src/uproot/behaviors/TAxis.py:131:1: D401 First line should be in imperative mood
src/uproot/behaviors/TAxis.py:141:1: D401 First line should be in imperative mood; try rephrasing
src/uproot/behaviors/TAxis.py:151:1: D401 First line should be in imperative mood; try rephrasing
src/uproot/behaviors/TAxis.py:161:1: D200 One-line docstring should fit on one line with quotes
src/uproot/behaviors/TAxis.py:161:1: D401 First line should be in imperative mood; try rephrasing
src/uproot/behaviors/TAxis.py:161:1: D402 First line should not be the function's "signature"
src/uproot/behaviors/TAxis.py:173:1: D205 1 blank line required between summary line and description
src/uproot/behaviors/TAxis.py:173:1: D400 First line should end with a period
src/uproot/behaviors/TAxis.py:197:1: D205 1 blank line required between summary line and description
src/uproot/behaviors/TAxis.py:197:1: D400 First line should end with a period
src/uproot/behaviors/TAxis.py:233:1: D205 1 blank line required between summary line and description
src/uproot/behaviors/TAxis.py:233:1: D400 First line should end with a period
src/uproot/behaviors/TAxis.py:271:1: D205 1 blank line required between summary line and description
src/uproot/behaviors/TAxis.py:271:1: D400 First line should end with a period
src/uproot/behaviors/TAxis.py:291:1: D205 1 blank line required between summary line and description
src/uproot/behaviors/TAxis.py:291:1: D400 First line should end with a period
src/uproot/sink/file.py:18:1: D205 1 blank line required between summary line and description
src/uproot/sink/file.py:18:1: D400 First line should end with a period
src/uproot/sink/file.py:32:1: D205 1 blank line required between summary line and description
src/uproot/sink/file.py:32:1: D400 First line should end with a period
src/uproot/sink/file.py:61:1: D107 Missing docstring in __init__
src/uproot/sink/file.py:67:1: D200 One-line docstring should fit on one line with quotes
src/uproot/sink/file.py:67:1: D401 First line should be in imperative mood; try rephrasing
src/uproot/sink/file.py:79:1: D105 Missing docstring in magic method
src/uproot/sink/file.py:84:1: D105 Missing docstring in magic method
src/uproot/sink/file.py:89:1: D200 One-line docstring should fit on one line with quotes
src/uproot/sink/file.py:89:1: D401 First line should be in imperative mood
src/uproot/sink/file.py:96:1: D200 One-line docstring should fit on one line with quotes
src/uproot/sink/file.py:96:1: D401 First line should be in imperative mood
src/uproot/sink/file.py:104:1: D200 One-line docstring should fit on one line with quotes
src/uproot/sink/file.py:104:1: D401 First line should be in imperative mood; try rephrasing
src/uproot/sink/file.py:110:1: D205 1 blank line required between summary line and description
src/uproot/sink/file.py:110:1: D400 First line should end with a period
src/uproot/sink/file.py:110:1: D401 First line should be in imperative mood
src/uproot/sink/file.py:119:1: D105 Missing docstring in magic method
src/uproot/sink/file.py:122:1: D105 Missing docstring in magic method
src/uproot/sink/file.py:126:1: D102 Missing docstring in public method
src/uproot/sink/file.py:133:1: D205 1 blank line required between summary line and description
src/uproot/sink/file.py:133:1: D400 First line should end with a period
src/uproot/sink/file.py:146:1: D401 First line should be in imperative mood
src/uproot/sink/file.py:160:1: D205 1 blank line required between summary line and description
src/uproot/sink/file.py:160:1: D400 First line should end with a period
src/uproot/source/cursor.py:3:1: D205 1 blank line required between summary line and description
src/uproot/source/cursor.py:3:1: D400 First line should end with a period
src/uproot/source/cursor.py:31:1: D205 1 blank line required between summary line and description
src/uproot/source/cursor.py:31:1: D400 First line should end with a period
src/uproot/source/cursor.py:47:1: D107 Missing docstring in __init__
src/uproot/source/cursor.py:52:1: D105 Missing docstring in magic method
src/uproot/source/cursor.py:73:1: D205 1 blank line required between summary line and description
src/uproot/source/cursor.py:73:1: D400 First line should end with a period
src/uproot/source/cursor.py:81:1: D205 1 blank line required between summary line and description
src/uproot/source/cursor.py:81:1: D400 First line should end with a period
src/uproot/source/cursor.py:89:1: D205 1 blank line required between summary line and description
src/uproot/source/cursor.py:89:1: D400 First line should end with a period
src/uproot/source/cursor.py:89:1: D401 First line should be in imperative mood
src/uproot/source/cursor.py:98:1: D205 1 blank line required between summary line and description
src/uproot/source/cursor.py:98:1: D400 First line should end with a period
src/uproot/source/cursor.py:98:1: D401 First line should be in imperative mood; try rephrasing
src/uproot/source/cursor.py:112:1: D205 1 blank line required between summary line and description
src/uproot/source/cursor.py:112:1: D400 First line should end with a period
src/uproot/source/cursor.py:112:1: D401 First line should be in imperative mood
src/uproot/source/cursor.py:123:1: D205 1 blank line required between summary line and description
src/uproot/source/cursor.py:123:1: D400 First line should end with a period
src/uproot/source/cursor.py:130:1: D205 1 blank line required between summary line and description
src/uproot/source/cursor.py:130:1: D400 First line should end with a period
src/uproot/source/cursor.py:137:1: D205 1 blank line required between summary line and description
src/uproot/source/cursor.py:137:1: D400 First line should end with a period
src/uproot/source/cursor.py:155:1: D205 1 blank line required between summary line and description
src/uproot/source/cursor.py:155:1: D400 First line should end with a period
src/uproot/source/cursor.py:178:1: D205 1 blank line required between summary line and description
src/uproot/source/cursor.py:178:1: D400 First line should end with a period
src/uproot/source/cursor.py:200:1: D205 1 blank line required between summary line and description
src/uproot/source/cursor.py:200:1: D400 First line should end with a period
src/uproot/source/cursor.py:222:1: D205 1 blank line required between summary line and description
src/uproot/source/cursor.py:222:1: D400 First line should end with a period
src/uproot/source/cursor.py:242:1: D205 1 blank line required between summary line and description
src/uproot/source/cursor.py:242:1: D400 First line should end with a period
src/uproot/source/cursor.py:275:1: D205 1 blank line required between summary line and description
src/uproot/source/cursor.py:275:1: D400 First line should end with a period
src/uproot/source/cursor.py:293:1: D205 1 blank line required between summary line and description
src/uproot/source/cursor.py:293:1: D400 First line should end with a period
src/uproot/source/cursor.py:313:1: D205 1 blank line required between summary line and description
src/uproot/source/cursor.py:313:1: D400 First line should end with a period
src/uproot/source/cursor.py:337:1: D205 1 blank line required between summary line and description
src/uproot/source/cursor.py:337:1: D400 First line should end with a period
src/uproot/source/cursor.py:368:1: D205 1 blank line required between summary line and description
src/uproot/source/cursor.py:368:1: D400 First line should end with a period
src/uproot/source/cursor.py:389:1: D205 1 blank line required between summary line and description
src/uproot/source/cursor.py:389:1: D400 First line should end with a period
src/uproot/source/cursor.py:410:1: D205 1 blank line required between summary line and description
src/uproot/source/cursor.py:410:1: D400 First line should end with a period
src/uproot/source/cursor.py:428:1: D205 1 blank line required between summary line and description
src/uproot/source/cursor.py:428:1: D400 First line should end with a period
src/uproot/source/cursor.py:461:1: D102 Missing docstring in public method
src/uproot/source/cursor.py:471:1: D102 Missing docstring in public method
src/uproot/source/cursor.py:484:1: D205 1 blank line required between summary line and description
src/uproot/source/cursor.py:484:1: D400 First line should end with a period
src/uproot/models/TList.py:3:1: D200 One-line docstring should fit on one line with quotes
src/uproot/models/TList.py:17:1: D200 One-line docstring should fit on one line with quotes
src/uproot/models/TList.py:21:1: D102 Missing docstring in public method
src/uproot/models/TList.py:57:1: D105 Missing docstring in magic method
src/uproot/models/TList.py:69:1: D105 Missing docstring in magic method
src/uproot/models/TList.py:72:1: D105 Missing docstring in magic method
src/uproot/models/TList.py:76:1: D102 Missing docstring in public method
src/uproot/models/TList.py:79:1: D102 Missing docstring in public method
src/uproot/deserialization.py:3:1: D205 1 blank line required between summary line and description
src/uproot/deserialization.py:3:1: D400 First line should end with a period
src/uproot/deserialization.py:42:1: D205 1 blank line required between summary line and description
src/uproot/deserialization.py:42:1: D400 First line should end with a period
src/uproot/deserialization.py:96:1: D205 1 blank line required between summary line and description
src/uproot/deserialization.py:96:1: D400 First line should end with a period
src/uproot/deserialization.py:144:1: D205 1 blank line required between summary line and description
src/uproot/deserialization.py:144:1: D400 First line should end with a period
src/uproot/deserialization.py:182:1: D205 1 blank line required between summary line and description
src/uproot/deserialization.py:182:1: D400 First line should end with a period
src/uproot/deserialization.py:342:1: D107 Missing docstring in __init__
src/uproot/deserialization.py:349:1: D105 Missing docstring in magic method
src/uproot/deserialization.py:432:1: D205 1 blank line required between summary line and description
src/uproot/deserialization.py:432:1: D400 First line should end with a period
src/uproot/deserialization.py:486:1: D205 1 blank line required between summary line and description
src/uproot/deserialization.py:486:1: D400 First line should end with a period
src/uproot/deserialization.py:507:1: D205 1 blank line required between summary line and description
src/uproot/deserialization.py:507:1: D400 First line should end with a period
src/uproot/deserialization.py:507:1: D401 First line should be in imperative mood; try rephrasing
src/uproot/source/futures.py:3:1: D400 First line should end with a period
src/uproot/source/futures.py:33:1: D200 One-line docstring should fit on one line with quotes
src/uproot/source/futures.py:43:1: D205 1 blank line required between summary line and description
src/uproot/source/futures.py:43:1: D400 First line should end with a period
src/uproot/source/futures.py:48:1: D107 Missing docstring in __init__
src/uproot/source/futures.py:52:1: D200 One-line docstring should fit on one line with quotes
src/uproot/source/futures.py:52:1: D401 First line should be in imperative mood; try rephrasing
src/uproot/source/futures.py:59:1: D205 1 blank line required between summary line and description
src/uproot/source/futures.py:59:1: D400 First line should end with a period
src/uproot/source/futures.py:66:1: D105 Missing docstring in magic method
src/uproot/source/futures.py:70:1: D200 One-line docstring should fit on one line with quotes
src/uproot/source/futures.py:76:1: D200 One-line docstring should fit on one line with quotes
src/uproot/source/futures.py:76:1: D401 First line should be in imperative mood; try rephrasing
src/uproot/source/futures.py:86:1: D205 1 blank line required between summary line and description
src/uproot/source/futures.py:86:1: D400 First line should end with a period
src/uproot/source/futures.py:97:1: D107 Missing docstring in __init__
src/uproot/source/futures.py:105:1: D205 1 blank line required between summary line and description
src/uproot/source/futures.py:105:1: D400 First line should end with a period
src/uproot/source/futures.py:105:1: D401 First line should be in imperative mood
src/uproot/source/futures.py:132:1: D205 1 blank line required between summary line and description
src/uproot/source/futures.py:132:1: D400 First line should end with a period
src/uproot/source/futures.py:143:1: D107 Missing docstring in __init__
src/uproot/source/futures.py:150:1: D205 1 blank line required between summary line and description
src/uproot/source/futures.py:150:1: D400 First line should end with a period
src/uproot/source/futures.py:150:1: D401 First line should be in imperative mood; try rephrasing
src/uproot/source/futures.py:158:1: D205 1 blank line required between summary line and description
src/uproot/source/futures.py:158:1: D400 First line should end with a period
src/uproot/source/futures.py:158:1: D401 First line should be in imperative mood
src/uproot/source/futures.py:174:1: D205 1 blank line required between summary line and description
src/uproot/source/futures.py:174:1: D400 First line should end with a period
src/uproot/source/futures.py:186:1: D107 Missing docstring in __init__
src/uproot/source/futures.py:202:1: D105 Missing docstring in magic method
src/uproot/source/futures.py:209:1: D200 One-line docstring should fit on one line with quotes
src/uproot/source/futures.py:209:1: D401 First line should be in imperative mood; try rephrasing
src/uproot/source/futures.py:216:1: D200 One-line docstring should fit on one line with quotes
src/uproot/source/futures.py:216:1: D401 First line should be in imperative mood; try rephrasing
src/uproot/source/futures.py:216:1: D402 First line should not be the function's "signature"
src/uproot/source/futures.py:222:1: D205 1 blank line required between summary line and description
src/uproot/source/futures.py:222:1: D400 First line should end with a period
src/uproot/source/futures.py:233:1: D205 1 blank line required between summary line and description
src/uproot/source/futures.py:233:1: D400 First line should end with a period
src/uproot/source/futures.py:252:1: D205 1 blank line required between summary line and description
src/uproot/source/futures.py:252:1: D400 First line should end with a period
src/uproot/source/futures.py:262:1: D107 Missing docstring in __init__
src/uproot/source/futures.py:289:1: D205 1 blank line required between summary line and description
src/uproot/source/futures.py:289:1: D400 First line should end with a period
src/uproot/source/futures.py:303:1: D107 Missing docstring in __init__
src/uproot/source/futures.py:309:1: D200 One-line docstring should fit on one line with quotes
src/uproot/source/futures.py:309:1: D401 First line should be in imperative mood; try rephrasing
src/uproot/source/futures.py:315:1: D205 1 blank line required between summary line and description
src/uproot/source/futures.py:315:1: D400 First line should end with a period
src/uproot/source/futures.py:315:1: D401 First line should be in imperative mood
src/uproot/source/futures.py:332:1: D205 1 blank line required between summary line and description
src/uproot/source/futures.py:332:1: D400 First line should end with a period
src/uproot/source/futures.py:341:1: D107 Missing docstring in __init__
src/uproot/source/futures.py:354:1: D105 Missing docstring in magic method
src/uproot/source/futures.py:360:1: D205 1 blank line required between summary line and description
src/uproot/source/futures.py:360:1: D400 First line should end with a period
src/uproot/source/futures.py:378:1: D205 1 blank line required between summary line and description
src/uproot/source/futures.py:378:1: D400 First line should end with a period
src/uproot/source/futures.py:378:1: D401 First line should be in imperative mood
src/uproot/source/futures.py:386:1: D205 1 blank line required between summary line and description
src/uproot/source/futures.py:386:1: D400 First line should end with a period
src/uproot/source/futures.py:386:1: D401 First line should be in imperative mood; try rephrasing
src/uproot/source/futures.py:393:1: D105 Missing docstring in magic method
src/uproot/source/futures.py:397:1: D105 Missing docstring in magic method
src/uproot/behaviors/TH2.py:3:1: D205 1 blank line required between summary line and description
src/uproot/behaviors/TH2.py:3:1: D400 First line should end with a period
src/uproot/behaviors/TH2.py:16:1: D205 1 blank line required between summary line and description
src/uproot/behaviors/TH2.py:16:1: D400 First line should end with a period
src/uproot/behaviors/TH2.py:24:1: D102 Missing docstring in public method
src/uproot/behaviors/TH2.py:27:1: D102 Missing docstring in public method
src/uproot/behaviors/TH2.py:36:1: D102 Missing docstring in public method
src/uproot/behaviors/TH2.py:41:1: D102 Missing docstring in public method
src/uproot/behaviors/TH2.py:44:1: D102 Missing docstring in public method
src/uproot/behaviors/TH2.py:84:1: D205 1 blank line required between summary line and description
src/uproot/behaviors/TH2.py:84:1: D400 First line should end with a period
src/uproot/behaviors/TH2.py:104:1: D102 Missing docstring in public method
dev/make-models.py:1:1: D200 One-line docstring should fit on one line with quotes
src/uproot/interpretation/__init__.py:36:1: D205 1 blank line required between summary line and description
src/uproot/interpretation/__init__.py:36:1: D400 First line should end with a period
src/uproot/interpretation/__init__.py:36:1: D401 First line should be in imperative mood; try rephrasing
src/uproot/interpretation/__init__.py:44:1: D401 First line should be in imperative mood; try rephrasing
src/uproot/interpretation/__init__.py:54:1: D200 One-line docstring should fit on one line with quotes
src/uproot/interpretation/__init__.py:54:1: D401 First line should be in imperative mood; try rephrasing
src/uproot/interpretation/__init__.py:67:1: D205 1 blank line required between summary line and description
src/uproot/interpretation/__init__.py:67:1: D400 First line should end with a period
src/uproot/interpretation/__init__.py:92:1: D205 1 blank line required between summary line and description
src/uproot/interpretation/__init__.py:92:1: D400 First line should end with a period
src/uproot/interpretation/__init__.py:118:1: D205 1 blank line required between summary line and description
src/uproot/interpretation/__init__.py:118:1: D400 First line should end with a period
src/uproot/interpretation/__init__.py:142:1: D105 Missing docstring in magic method
src/uproot/interpretation/__init__.py:145:1: D105 Missing docstring in magic method
src/uproot/interpretation/__init__.py:149:1: D205 1 blank line required between summary line and description
src/uproot/interpretation/__init__.py:149:1: D400 First line should end with a period
src/uproot/interpretation/__init__.py:149:1: D401 First line should be in imperative mood
src/uproot/interpretation/__init__.py:159:1: D205 1 blank line required between summary line and description
src/uproot/interpretation/__init__.py:159:1: D400 First line should end with a period
src/uproot/interpretation/__init__.py:159:1: D401 First line should be in imperative mood
src/uproot/interpretation/__init__.py:169:1: D205 1 blank line required between summary line and description
src/uproot/interpretation/__init__.py:169:1: D400 First line should end with a period
src/uproot/interpretation/__init__.py:169:1: D401 First line should be in imperative mood
src/uproot/interpretation/__init__.py:179:1: D205 1 blank line required between summary line and description
src/uproot/interpretation/__init__.py:179:1: D400 First line should end with a period
src/uproot/interpretation/__init__.py:179:1: D401 First line should be in imperative mood
src/uproot/interpretation/__init__.py:187:1: D205 1 blank line required between summary line and description
src/uproot/interpretation/__init__.py:187:1: D400 First line should end with a period
src/uproot/interpretation/__init__.py:187:1: D401 First line should be in imperative mood
src/uproot/const.py:3:1: D200 One-line docstring should fit on one line with quotes
src/uproot/behaviors/TProfile2D.py:3:1: D200 One-line docstring should fit on one line with quotes
src/uproot/behaviors/TProfile2D.py:17:1: D200 One-line docstring should fit on one line with quotes
src/uproot/behaviors/TProfile2D.py:24:1: D102 Missing docstring in public method
src/uproot/behaviors/TProfile2D.py:27:1: D102 Missing docstring in public method
src/uproot/behaviors/TProfile2D.py:36:1: D102 Missing docstring in public method
src/uproot/behaviors/TProfile2D.py:40:1: D102 Missing docstring in public method
src/uproot/behaviors/TProfile2D.py:53:1: D102 Missing docstring in public method
src/uproot/behaviors/TProfile2D.py:104:1: D102 Missing docstring in public method
src/uproot/source/file.py:26:1: D205 1 blank line required between summary line and description
src/uproot/source/file.py:26:1: D400 First line should end with a period
src/uproot/source/file.py:33:1: D107 Missing docstring in __init__
src/uproot/source/file.py:42:1: D200 One-line docstring should fit on one line with quotes
src/uproot/source/file.py:42:1: D401 First line should be in imperative mood; try rephrasing
src/uproot/source/file.py:48:1: D102 Missing docstring in public method
src/uproot/source/file.py:51:1: D105 Missing docstring in magic method
src/uproot/source/file.py:54:1: D105 Missing docstring in magic method
src/uproot/source/file.py:58:1: D205 1 blank line required between summary line and description
src/uproot/source/file.py:58:1: D400 First line should end with a period
src/uproot/source/file.py:71:1: D205 1 blank line required between summary line and description
src/uproot/source/file.py:71:1: D400 First line should end with a period
src/uproot/source/file.py:90:1: D205 1 blank line required between summary line and description
src/uproot/source/file.py:90:1: D400 First line should end with a period
src/uproot/source/file.py:100:1: D107 Missing docstring in __init__
src/uproot/source/file.py:122:1: D105 Missing docstring in magic method
src/uproot/source/file.py:127:1: D105 Missing docstring in magic method
src/uproot/source/file.py:131:1: D105 Missing docstring in magic method
src/uproot/source/file.py:142:1: D102 Missing docstring in public method
src/uproot/source/file.py:158:1: D102 Missing docstring in public method
src/uproot/source/file.py:183:1: D200 One-line docstring should fit on one line with quotes
src/uproot/source/file.py:183:1: D401 First line should be in imperative mood; try rephrasing
src/uproot/source/file.py:190:1: D205 1 blank line required between summary line and description
src/uproot/source/file.py:190:1: D400 First line should end with a period
src/uproot/source/file.py:200:1: D102 Missing docstring in public method
src/uproot/source/file.py:206:1: D105 Missing docstring in magic method
src/uproot/source/file.py:214:1: D105 Missing docstring in magic method
src/uproot/source/file.py:224:1: D102 Missing docstring in public method
src/uproot/source/file.py:232:1: D205 1 blank line required between summary line and description
src/uproot/source/file.py:232:1: D400 First line should end with a period
src/uproot/source/file.py:243:1: D107 Missing docstring in __init__
src/uproot/source/file.py:258:1: D105 Missing docstring in magic method
src/uproot/source/file.py:263:1: D105 Missing docstring in magic method
src/uproot/_util.py:3:1: D205 1 blank line required between summary line and description
src/uproot/_util.py:3:1: D400 First line should end with a period
src/uproot/_util.py:25:1: D205 1 blank line required between summary line and description
src/uproot/_util.py:25:1: D400 First line should end with a period
src/uproot/_util.py:25:1: D401 First line should be in imperative mood
src/uproot/_util.py:25:1: D402 First line should not be the function's "signature"
src/uproot/_util.py:36:1: D205 1 blank line required between summary line and description
src/uproot/_util.py:36:1: D400 First line should end with a period
src/uproot/_util.py:36:1: D401 First line should be in imperative mood
src/uproot/_util.py:46:1: D205 1 blank line required between summary line and description
src/uproot/_util.py:46:1: D400 First line should end with a period
src/uproot/_util.py:46:1: D401 First line should be in imperative mood
src/uproot/_util.py:56:1: D200 One-line docstring should fit on one line with quotes
src/uproot/_util.py:56:1: D401 First line should be in imperative mood
src/uproot/_util.py:63:1: D200 One-line docstring should fit on one line with quotes
src/uproot/_util.py:63:1: D401 First line should be in imperative mood
src/uproot/_util.py:75:1: D205 1 blank line required between summary line and description
src/uproot/_util.py:75:1: D400 First line should end with a period
src/uproot/_util.py:75:1: D401 First line should be in imperative mood
src/uproot/_util.py:112:1: D200 One-line docstring should fit on one line with quotes
src/uproot/_util.py:112:1: D401 First line should be in imperative mood; try rephrasing
src/uproot/_util.py:119:1: D205 1 blank line required between summary line and description
src/uproot/_util.py:119:1: D400 First line should end with a period
src/uproot/_util.py:148:1: D200 One-line docstring should fit on one line with quotes
src/uproot/_util.py:148:1: D401 First line should be in imperative mood; try rephrasing
src/uproot/_util.py:158:1: D205 1 blank line required between summary line and description
src/uproot/_util.py:158:1: D400 First line should end with a period
src/uproot/_util.py:217:1: D200 One-line docstring should fit on one line with quotes
src/uproot/_util.py:217:1: D401 First line should be in imperative mood
src/uproot/_util.py:242:1: D200 One-line docstring should fit on one line with quotes
src/uproot/_util.py:276:1: D200 One-line docstring should fit on one line with quotes
src/uproot/_util.py:403:1: D205 1 blank line required between summary line and description
src/uproot/_util.py:403:1: D400 First line should end with a period
src/uproot/_util.py:462:1: D205 1 blank line required between summary line and description
src/uproot/_util.py:462:1: D400 First line should end with a period
src/uproot/_util.py:480:1: D200 One-line docstring should fit on one line with quotes
src/uproot/_util.py:480:1: D401 First line should be in imperative mood; try rephrasing
src/uproot/_util.py:529:1: D200 One-line docstring should fit on one line with quotes
src/uproot/_util.py:639:1: D205 1 blank line required between summary line and description
src/uproot/_util.py:639:1: D400 First line should end with a period
src/uproot/_util.py:780:1: D401 First line should be in imperative mood
src/uproot/_util.py:831:1: D200 One-line docstring should fit on one line with quotes
src/uproot/_util.py:831:1: D401 First line should be in imperative mood
src/uproot/_util.py:845:1: D200 One-line docstring should fit on one line with quotes
src/uproot/_util.py:845:1: D401 First line should be in imperative mood
src/uproot/_util.py:859:1: D400 First line should end with a period
src/uproot/_util.py:859:1: D401 First line should be in imperative mood
src/uproot/interpretation/library.py:3:1: D205 1 blank line required between summary line and description
src/uproot/interpretation/library.py:3:1: D400 First line should end with a period
src/uproot/interpretation/library.py:43:1: D205 1 blank line required between summary line and description
src/uproot/interpretation/library.py:43:1: D400 First line should end with a period
src/uproot/interpretation/library.py:61:1: D200 One-line docstring should fit on one line with quotes
src/uproot/interpretation/library.py:61:1: D401 First line should be in imperative mood
src/uproot/interpretation/library.py:67:1: D205 1 blank line required between summary line and description
src/uproot/interpretation/library.py:67:1: D400 First line should end with a period
src/uproot/interpretation/library.py:79:1: D205 1 blank line required between summary line and description
src/uproot/interpretation/library.py:79:1: D400 First line should end with a period
src/uproot/interpretation/library.py:91:1: D205 1 blank line required between summary line and description
src/uproot/interpretation/library.py:91:1: D400 First line should end with a period
src/uproot/interpretation/library.py:110:1: D205 1 blank line required between summary line and description
src/uproot/interpretation/library.py:110:1: D400 First line should end with a period
src/uproot/interpretation/library.py:137:1: D205 1 blank line required between summary line and description
src/uproot/interpretation/library.py:137:1: D400 First line should end with a period
src/uproot/interpretation/library.py:153:1: D205 1 blank line required between summary line and description
src/uproot/interpretation/library.py:153:1: D400 First line should end with a period
src/uproot/interpretation/library.py:162:1: D105 Missing docstring in magic method
src/uproot/interpretation/library.py:165:1: D105 Missing docstring in magic method
src/uproot/interpretation/library.py:170:1: D205 1 blank line required between summary line and description
src/uproot/interpretation/library.py:170:1: D400 First line should end with a period
src/uproot/interpretation/library.py:193:1: D102 Missing docstring in public method
src/uproot/interpretation/library.py:198:1: D102 Missing docstring in public method
src/uproot/interpretation/library.py:235:1: D102 Missing docstring in public method
src/uproot/interpretation/library.py:461:1: D205 1 blank line required between summary line and description
src/uproot/interpretation/library.py:461:1: D400 First line should end with a period
src/uproot/interpretation/library.py:490:1: D102 Missing docstring in public method
src/uproot/interpretation/library.py:493:1: D102 Missing docstring in public method
src/uproot/interpretation/library.py:603:1: D102 Missing docstring in public method
src/uproot/interpretation/library.py:716:1: D102 Missing docstring in public method
src/uproot/interpretation/library.py:807:1: D205 1 blank line required between summary line and description
src/uproot/interpretation/library.py:807:1: D400 First line should end with a period
src/uproot/interpretation/library.py:833:1: D102 Missing docstring in public method
src/uproot/interpretation/library.py:836:1: D102 Missing docstring in public method
src/uproot/interpretation/library.py:939:1: D102 Missing docstring in public method
src/uproot/interpretation/library.py:1085:1: D102 Missing docstring in public method
src/uproot/interpretation/library.py:1116:1: D102 Missing docstring in public method
src/uproot/behaviors/TTree.py:3:1: D205 1 blank line required between summary line and description
src/uproot/behaviors/TTree.py:3:1: D400 First line should end with a period
src/uproot/behaviors/TTree.py:28:1: D105 Missing docstring in magic method
src/uproot/behaviors/TTree.py:38:1: D200 One-line docstring should fit on one line with quotes
src/uproot/behaviors/TTree.py:45:1: D200 One-line docstring should fit on one line with quotes
src/uproot/behaviors/TTree.py:52:1: D200 One-line docstring should fit on one line with quotes
src/uproot/behaviors/TTree.py:59:1: D205 1 blank line required between summary line and description
src/uproot/behaviors/TTree.py:59:1: D400 First line should end with a period
src/uproot/behaviors/TTree.py:59:1: D401 First line should be in imperative mood; try rephrasing
src/uproot/behaviors/TTree.py:69:1: D401 First line should be in imperative mood; try rephrasing
src/uproot/behaviors/TTree.py:81:1: D200 One-line docstring should fit on one line with quotes
src/uproot/behaviors/TTree.py:81:1: D401 First line should be in imperative mood
src/uproot/behaviors/TTree.py:88:1: D205 1 blank line required between summary line and description
src/uproot/behaviors/TTree.py:88:1: D400 First line should end with a period
src/uproot/behaviors/TTree.py:88:1: D401 First line should be in imperative mood; try rephrasing
src/uproot/behaviors/TTree.py:100:1: D205 1 blank line required between summary line and description
src/uproot/behaviors/TTree.py:100:1: D400 First line should end with a period
src/uproot/behaviors/TTree.py:100:1: D401 First line should be in imperative mood; try rephrasing
src/uproot/behaviors/TTree.py:111:1: D205 1 blank line required between summary line and description
src/uproot/behaviors/TTree.py:111:1: D400 First line should end with a period
src/uproot/behaviors/TTree.py:111:1: D401 First line should be in imperative mood; try rephrasing
src/uproot/behaviors/TTree.py:122:1: D205 1 blank line required between summary line and description
src/uproot/behaviors/TTree.py:122:1: D400 First line should end with a period
src/uproot/behaviors/TTree.py:122:1: D401 First line should be in imperative mood; try rephrasing
src/uproot/behaviors/TTree.py:140:1: D205 1 blank line required between summary line and description
src/uproot/behaviors/TTree.py:140:1: D400 First line should end with a period
src/uproot/behaviors/TTree.py:140:1: D401 First line should be in imperative mood; try rephrasing
src/uproot/behaviors/TTree.py:158:1: D102 Missing docstring in public method
src/uproot/models/TArray.py:3:1: D200 One-line docstring should fit on one line with quotes
src/uproot/models/TArray.py:25:1: D102 Missing docstring in public method
src/uproot/models/TArray.py:28:1: D102 Missing docstring in public method
src/uproot/models/TArray.py:39:1: D105 Missing docstring in magic method
src/uproot/models/TArray.py:46:1: D102 Missing docstring in public method
src/uproot/models/TArray.py:49:1: D105 Missing docstring in magic method
src/uproot/models/TArray.py:52:1: D105 Missing docstring in magic method
src/uproot/models/TArray.py:55:1: D105 Missing docstring in magic method
src/uproot/models/TArray.py:73:1: D102 Missing docstring in public method
src/uproot/models/TArray.py:77:1: D102 Missing docstring in public method
src/uproot/containers.py:3:1: D205 1 blank line required between summary line and description
src/uproot/containers.py:3:1: D400 First line should end with a period
src/uproot/containers.py:113:1: D205 1 blank line required between summary line and description
src/uproot/containers.py:113:1: D400 First line should end with a period
src/uproot/containers.py:128:1: D205 1 blank line required between summary line and description
src/uproot/containers.py:128:1: D400 First line should end with a period
src/uproot/containers.py:128:1: D401 First line should be in imperative mood; try rephrasing
src/uproot/containers.py:136:1: D401 First line should be in imperative mood; try rephrasing
src/uproot/containers.py:152:1: D205 1 blank line required between summary line and description
src/uproot/containers.py:152:1: D400 First line should end with a period
src/uproot/containers.py:175:1: D205 1 blank line required between summary line and description
src/uproot/containers.py:175:1: D400 First line should end with a period
src/uproot/containers.py:196:1: D200 One-line docstring should fit on one line with quotes
src/uproot/containers.py:209:1: D205 1 blank line required between summary line and description
src/uproot/containers.py:209:1: D400 First line should end with a period
src/uproot/containers.py:230:1: D105 Missing docstring in magic method
src/uproot/containers.py:233:1: D105 Missing docstring in magic method
src/uproot/containers.py:238:1: D205 1 blank line required between summary line and description
src/uproot/containers.py:238:1: D400 First line should end with a period
src/uproot/containers.py:252:1: D107 Missing docstring in __init__
src/uproot/containers.py:257:1: D205 1 blank line required between summary line and description
src/uproot/containers.py:257:1: D400 First line should end with a period
src/uproot/containers.py:257:1: D401 First line should be in imperative mood; try rephrasing
src/uproot/containers.py:264:1: D105 Missing docstring in magic method
src/uproot/containers.py:274:1: D102 Missing docstring in public method
src/uproot/containers.py:281:1: D102 Missing docstring in public method
src/uproot/containers.py:287:1: D102 Missing docstring in public method
src/uproot/containers.py:307:1: D102 Missing docstring in public method
src/uproot/containers.py:315:1: D205 1 blank line required between summary line and description
src/uproot/containers.py:315:1: D400 First line should end with a period
src/uproot/containers.py:327:1: D107 Missing docstring in __init__
src/uproot/containers.py:330:1: D105 Missing docstring in magic method
src/uproot/containers.py:333:1: D105 Missing docstring in magic method
src/uproot/containers.py:337:1: D102 Missing docstring in public method
src/uproot/containers.py:341:1: D102 Missing docstring in public method
src/uproot/containers.py:344:1: D102 Missing docstring in public method
src/uproot/containers.py:354:1: D102 Missing docstring in public method
src/uproot/containers.py:359:1: D105 Missing docstring in magic method
src/uproot/containers.py:367:1: D205 1 blank line required between summary line and description
src/uproot/containers.py:367:1: D400 First line should end with a period
src/uproot/containers.py:392:1: D107 Missing docstring in __init__
src/uproot/containers.py:402:1: D205 1 blank line required between summary line and description
src/uproot/containers.py:402:1: D400 First line should end with a period
src/uproot/containers.py:402:1: D401 First line should be in imperative mood; try rephrasing
src/uproot/containers.py:409:1: D105 Missing docstring in magic method
src/uproot/containers.py:412:1: D105 Missing docstring in magic method
src/uproot/containers.py:419:1: D102 Missing docstring in public method
src/uproot/containers.py:423:1: D102 Missing docstring in public method
src/uproot/containers.py:429:1: D102 Missing docstring in public method
src/uproot/containers.py:451:1: D102 Missing docstring in public method
src/uproot/containers.py:481:1: D105 Missing docstring in magic method
src/uproot/containers.py:490:1: D205 1 blank line required between summary line and description
src/uproot/containers.py:490:1: D400 First line should end with a period
src/uproot/containers.py:505:1: D107 Missing docstring in __init__
src/uproot/containers.py:510:1: D205 1 blank line required between summary line and description
src/uproot/containers.py:510:1: D400 First line should end with a period
src/uproot/containers.py:510:1: D401 First line should be in imperative mood; try rephrasing
src/uproot/containers.py:517:1: D105 Missing docstring in magic method
src/uproot/containers.py:520:1: D105 Missing docstring in magic method
src/uproot/containers.py:530:1: D102 Missing docstring in public method
src/uproot/containers.py:537:1: D102 Missing docstring in public method
src/uproot/containers.py:543:1: D102 Missing docstring in public method
src/uproot/containers.py:553:1: D102 Missing docstring in public method
src/uproot/containers.py:558:1: D105 Missing docstring in magic method
src/uproot/containers.py:566:1: D205 1 blank line required between summary line and description
src/uproot/containers.py:566:1: D400 First line should end with a period
src/uproot/containers.py:579:1: D107 Missing docstring in __init__
src/uproot/containers.py:589:1: D200 One-line docstring should fit on one line with quotes
src/uproot/containers.py:596:1: D205 1 blank line required between summary line and description
src/uproot/containers.py:596:1: D400 First line should end with a period
src/uproot/containers.py:596:1: D401 First line should be in imperative mood; try rephrasing
src/uproot/containers.py:605:1: D200 One-line docstring should fit on one line with quotes
src/uproot/containers.py:605:1: D400 First line should end with a period
src/uproot/containers.py:605:1: D401 First line should be in imperative mood; try rephrasing
src/uproot/containers.py:610:1: D105 Missing docstring in magic method
src/uproot/containers.py:620:1: D102 Missing docstring in public method
src/uproot/containers.py:629:1: D102 Missing docstring in public method
src/uproot/containers.py:633:1: D102 Missing docstring in public method
src/uproot/containers.py:660:1: D102 Missing docstring in public method
src/uproot/containers.py:724:1: D205 1 blank line required between summary line and description
src/uproot/containers.py:724:1: D400 First line should end with a period
src/uproot/containers.py:733:1: D107 Missing docstring in __init__
src/uproot/containers.py:744:1: D105 Missing docstring in magic method
src/uproot/containers.py:749:1: D200 One-line docstring should fit on one line with quotes
src/uproot/containers.py:749:1: D401 First line should be in imperative mood; try rephrasing
src/uproot/containers.py:754:1: D105 Missing docstring in magic method
src/uproot/containers.py:762:1: D102 Missing docstring in public method
src/uproot/containers.py:766:1: D102 Missing docstring in public method
src/uproot/containers.py:769:1: D102 Missing docstring in public method
src/uproot/containers.py:786:1: D102 Missing docstring in public method
src/uproot/containers.py:868:1: D105 Missing docstring in magic method
src/uproot/containers.py:888:1: D205 1 blank line required between summary line and description
src/uproot/containers.py:888:1: D400 First line should end with a period
src/uproot/containers.py:897:1: D107 Missing docstring in __init__
src/uproot/containers.py:908:1: D105 Missing docstring in magic method
src/uproot/containers.py:913:1: D200 One-line docstring should fit on one line with quotes
src/uproot/containers.py:913:1: D401 First line should be in imperative mood; try rephrasing
src/uproot/containers.py:918:1: D105 Missing docstring in magic method
src/uproot/containers.py:926:1: D102 Missing docstring in public method
src/uproot/containers.py:930:1: D102 Missing docstring in public method
src/uproot/containers.py:933:1: D102 Missing docstring in public method
src/uproot/containers.py:953:1: D102 Missing docstring in public method
src/uproot/containers.py:992:1: D105 Missing docstring in magic method
src/uproot/containers.py:1017:1: D205 1 blank line required between summary line and description
src/uproot/containers.py:1017:1: D400 First line should end with a period
src/uproot/containers.py:1028:1: D107 Missing docstring in __init__
src/uproot/containers.py:1049:1: D105 Missing docstring in magic method
src/uproot/containers.py:1054:1: D200 One-line docstring should fit on one line with quotes
src/uproot/containers.py:1054:1: D401 First line should be in imperative mood; try rephrasing
src/uproot/containers.py:1061:1: D200 One-line docstring should fit on one line with quotes
src/uproot/containers.py:1061:1: D401 First line should be in imperative mood; try rephrasing
src/uproot/containers.py:1066:1: D105 Missing docstring in magic method
src/uproot/containers.py:1078:1: D102 Missing docstring in public method
src/uproot/containers.py:1086:1: D102 Missing docstring in public method
src/uproot/containers.py:1091:1: D102 Missing docstring in public method
src/uproot/containers.py:1128:1: D102 Missing docstring in public method
src/uproot/containers.py:1194:1: D105 Missing docstring in magic method
src/uproot/containers.py:1225:1: D200 One-line docstring should fit on one line with quotes
src/uproot/containers.py:1229:1: D105 Missing docstring in magic method
src/uproot/containers.py:1233:1: D205 1 blank line required between summary line and description
src/uproot/containers.py:1233:1: D400 First line should end with a period
src/uproot/containers.py:1241:1: D205 1 blank line required between summary line and description
src/uproot/containers.py:1241:1: D400 First line should end with a period
src/uproot/containers.py:1248:1: D107 Missing docstring in __init__
src/uproot/containers.py:1258:1: D105 Missing docstring in magic method
src/uproot/containers.py:1264:1: D105 Missing docstring in magic method
src/uproot/containers.py:1269:1: D105 Missing docstring in magic method
src/uproot/containers.py:1272:1: D105 Missing docstring in magic method
src/uproot/containers.py:1275:1: D105 Missing docstring in magic method
src/uproot/containers.py:1278:1: D105 Missing docstring in magic method
src/uproot/containers.py:1281:1: D105 Missing docstring in magic method
src/uproot/containers.py:1284:1: D105 Missing docstring in magic method
src/uproot/containers.py:1292:1: D102 Missing docstring in public method
src/uproot/containers.py:1299:1: D205 1 blank line required between summary line and description
src/uproot/containers.py:1299:1: D400 First line should end with a period
src/uproot/containers.py:1306:1: D107 Missing docstring in __init__
src/uproot/containers.py:1316:1: D105 Missing docstring in magic method
src/uproot/containers.py:1322:1: D105 Missing docstring in magic method
src/uproot/containers.py:1327:1: D105 Missing docstring in magic method
src/uproot/containers.py:1330:1: D105 Missing docstring in magic method
src/uproot/containers.py:1333:1: D105 Missing docstring in magic method
src/uproot/containers.py:1346:1: D105 Missing docstring in magic method
src/uproot/containers.py:1362:1: D102 Missing docstring in public method
src/uproot/containers.py:1369:1: D205 1 blank line required between summary line and description
src/uproot/containers.py:1369:1: D400 First line should end with a period
src/uproot/containers.py:1381:1: D205 1 blank line required between summary line and description
src/uproot/containers.py:1381:1: D400 First line should end with a period
src/uproot/containers.py:1387:1: D107 Missing docstring in __init__
src/uproot/containers.py:1413:1: D105 Missing docstring in magic method
src/uproot/containers.py:1419:1: D105 Missing docstring in magic method
src/uproot/containers.py:1425:1: D200 One-line docstring should fit on one line with quotes
src/uproot/containers.py:1431:1: D200 One-line docstring should fit on one line with quotes
src/uproot/containers.py:1437:1: D200 One-line docstring should fit on one line with quotes
src/uproot/containers.py:1442:1: D105 Missing docstring in magic method
src/uproot/containers.py:1462:1: D200 One-line docstring should fit on one line with quotes
src/uproot/containers.py:1484:1: D105 Missing docstring in magic method
src/uproot/containers.py:1487:1: D105 Missing docstring in magic method
src/uproot/containers.py:1490:1: D105 Missing docstring in magic method
src/uproot/containers.py:1503:1: D105 Missing docstring in magic method
src/uproot/containers.py:1520:1: D102 Missing docstring in public method
src/uproot/models/TNamed.py:3:1: D200 One-line docstring should fit on one line with quotes
src/uproot/models/TNamed.py:14:1: D200 One-line docstring should fit on one line with quotes
src/uproot/models/TNamed.py:18:1: D102 Missing docstring in public method
src/uproot/models/TNamed.py:41:1: D105 Missing docstring in magic method
src/uproot/models/TNamed.py:50:1: D102 Missing docstring in public method
src/uproot/writing/identify.py:3:1: D205 1 blank line required between summary line and description
src/uproot/writing/identify.py:3:1: D400 First line should end with a period
src/uproot/writing/identify.py:30:1: D205 1 blank line required between summary line and description
src/uproot/writing/identify.py:30:1: D400 First line should end with a period
src/uproot/writing/identify.py:184:1: D205 1 blank line required between summary line and description
src/uproot/writing/identify.py:184:1: D400 First line should end with a period
src/uproot/writing/identify.py:184:1: D401 First line should be in imperative mood
src/uproot/writing/identify.py:711:1: D205 1 blank line required between summary line and description
src/uproot/writing/identify.py:711:1: D400 First line should end with a period
src/uproot/writing/identify.py:711:1: D401 First line should be in imperative mood; try rephrasing
src/uproot/writing/identify.py:728:1: D205 1 blank line required between summary line and description
src/uproot/writing/identify.py:728:1: D400 First line should end with a period
src/uproot/writing/identify.py:728:1: D401 First line should be in imperative mood; try rephrasing
src/uproot/writing/identify.py:744:1: D205 1 blank line required between summary line and description
src/uproot/writing/identify.py:744:1: D400 First line should end with a period
src/uproot/writing/identify.py:775:1: D205 1 blank line required between summary line and description
src/uproot/writing/identify.py:775:1: D400 First line should end with a period
src/uproot/writing/identify.py:839:1: D205 1 blank line required between summary line and description
src/uproot/writing/identify.py:839:1: D400 First line should end with a period
src/uproot/writing/identify.py:961:1: D205 1 blank line required between summary line and description
src/uproot/writing/identify.py:961:1: D400 First line should end with a period
src/uproot/writing/identify.py:1176:1: D205 1 blank line required between summary line and description
src/uproot/writing/identify.py:1176:1: D400 First line should end with a period
src/uproot/writing/identify.py:1343:1: D205 1 blank line required between summary line and description
src/uproot/writing/identify.py:1343:1: D400 First line should end with a period
src/uproot/writing/identify.py:1520:1: D205 1 blank line required between summary line and description
src/uproot/writing/identify.py:1520:1: D400 First line should end with a period
src/uproot/writing/identify.py:1693:1: D205 1 blank line required between summary line and description
src/uproot/writing/identify.py:1693:1: D400 First line should end with a period
src/uproot/writing/identify.py:1877:1: D205 1 blank line required between summary line and description
src/uproot/writing/identify.py:1877:1: D400 First line should end with a period
src/uproot/behaviors/__init__.py:3:1: D205 1 blank line required between summary line and description
src/uproot/behaviors/__init__.py:3:1: D400 First line should end with a period
src/uproot/behaviors/TBranch.py:3:1: D205 1 blank line required between summary line and description
src/uproot/behaviors/TBranch.py:3:1: D400 First line should end with a period
src/uproot/behaviors/TBranch.py:70:1: D205 1 blank line required between summary line and description
src/uproot/behaviors/TBranch.py:70:1: D301 Use r""" if any backslashes in a docstring
src/uproot/behaviors/TBranch.py:70:1: D400 First line should end with a period
src/uproot/behaviors/TBranch.py:243:1: D205 1 blank line required between summary line and description
src/uproot/behaviors/TBranch.py:243:1: D301 Use r""" if any backslashes in a docstring
src/uproot/behaviors/TBranch.py:243:1: D400 First line should end with a period
src/uproot/behaviors/TBranch.py:401:1: D205 1 blank line required between summary line and description
src/uproot/behaviors/TBranch.py:401:1: D301 Use r""" if any backslashes in a docstring
src/uproot/behaviors/TBranch.py:401:1: D400 First line should end with a period
src/uproot/behaviors/TBranch.py:693:1: D205 1 blank line required between summary line and description
src/uproot/behaviors/TBranch.py:693:1: D400 First line should end with a period
src/uproot/behaviors/TBranch.py:724:1: D107 Missing docstring in __init__
src/uproot/behaviors/TBranch.py:730:1: D105 Missing docstring in magic method
src/uproot/behaviors/TBranch.py:739:1: D205 1 blank line required between summary line and description
src/uproot/behaviors/TBranch.py:739:1: D400 First line should end with a period
src/uproot/behaviors/TBranch.py:739:1: D401 First line should be in imperative mood; try rephrasing
src/uproot/behaviors/TBranch.py:748:1: D205 1 blank line required between summary line and description
src/uproot/behaviors/TBranch.py:748:1: D400 First line should end with a period
src/uproot/behaviors/TBranch.py:748:1: D401 First line should be in imperative mood; try rephrasing
src/uproot/behaviors/TBranch.py:756:1: D205 1 blank line required between summary line and description
src/uproot/behaviors/TBranch.py:756:1: D400 First line should end with a period
src/uproot/behaviors/TBranch.py:756:1: D401 First line should be in imperative mood; try rephrasing
src/uproot/behaviors/TBranch.py:764:1: D205 1 blank line required between summary line and description
src/uproot/behaviors/TBranch.py:764:1: D400 First line should end with a period
src/uproot/behaviors/TBranch.py:764:1: D401 First line should be in imperative mood; try rephrasing
src/uproot/behaviors/TBranch.py:772:1: D205 1 blank line required between summary line and description
src/uproot/behaviors/TBranch.py:772:1: D400 First line should end with a period
src/uproot/behaviors/TBranch.py:780:1: D205 1 blank line required between summary line and description
src/uproot/behaviors/TBranch.py:780:1: D400 First line should end with a period
src/uproot/behaviors/TBranch.py:788:1: D205 1 blank line required between summary line and description
src/uproot/behaviors/TBranch.py:788:1: D400 First line should end with a period
src/uproot/behaviors/TBranch.py:796:1: D205 1 blank line required between summary line and description
src/uproot/behaviors/TBranch.py:796:1: D400 First line should end with a period
src/uproot/behaviors/TBranch.py:804:1: D205 1 blank line required between summary line and description
src/uproot/behaviors/TBranch.py:804:1: D400 First line should end with a period
src/uproot/behaviors/TBranch.py:804:1: D401 First line should be in imperative mood; try rephrasing
src/uproot/behaviors/TBranch.py:812:1: D205 1 blank line required between summary line and description
src/uproot/behaviors/TBranch.py:812:1: D400 First line should end with a period
src/uproot/behaviors/TBranch.py:812:1: D401 First line should be in imperative mood; try rephrasing
src/uproot/behaviors/TBranch.py:820:1: D205 1 blank line required between summary line and description
src/uproot/behaviors/TBranch.py:820:1: D400 First line should end with a period
src/uproot/behaviors/TBranch.py:820:1: D401 First line should be in imperative mood; try rephrasing
src/uproot/behaviors/TBranch.py:832:1: D205 1 blank line required between summary line and description
src/uproot/behaviors/TBranch.py:832:1: D400 First line should end with a period
src/uproot/behaviors/TBranch.py:832:1: D401 First line should be in imperative mood
src/uproot/behaviors/TBranch.py:843:1: D205 1 blank line required between summary line and description
src/uproot/behaviors/TBranch.py:843:1: D400 First line should end with a period
src/uproot/behaviors/TBranch.py:862:1: D205 1 blank line required between summary line and description
src/uproot/behaviors/TBranch.py:862:1: D400 First line should end with a period
src/uproot/behaviors/TBranch.py:862:1: D401 First line should be in imperative mood; try rephrasing
src/uproot/behaviors/TBranch.py:881:1: D205 1 blank line required between summary line and description
src/uproot/behaviors/TBranch.py:881:1: D400 First line should end with a period
src/uproot/behaviors/TBranch.py:987:1: D205 1 blank line required between summary line and description
src/uproot/behaviors/TBranch.py:987:1: D400 First line should end with a period
src/uproot/behaviors/TBranch.py:1200:1: D205 1 blank line required between summary line and description
src/uproot/behaviors/TBranch.py:1200:1: D400 First line should end with a period
src/uproot/behaviors/TBranch.py:1415:1: D205 1 blank line required between summary line and description
src/uproot/behaviors/TBranch.py:1415:1: D400 First line should end with a period
src/uproot/behaviors/TBranch.py:1451:1: D205 1 blank line required between summary line and description
src/uproot/behaviors/TBranch.py:1451:1: D400 First line should end with a period
src/uproot/behaviors/TBranch.py:1488:1: D205 1 blank line required between summary line and description
src/uproot/behaviors/TBranch.py:1488:1: D400 First line should end with a period
src/uproot/behaviors/TBranch.py:1526:1: D205 1 blank line required between summary line and description
src/uproot/behaviors/TBranch.py:1526:1: D400 First line should end with a period
src/uproot/behaviors/TBranch.py:1564:1: D205 1 blank line required between summary line and description
src/uproot/behaviors/TBranch.py:1564:1: D400 First line should end with a period
src/uproot/behaviors/TBranch.py:1599:1: D205 1 blank line required between summary line and description
src/uproot/behaviors/TBranch.py:1599:1: D400 First line should end with a period
src/uproot/behaviors/TBranch.py:1636:1: D205 1 blank line required between summary line and description
src/uproot/behaviors/TBranch.py:1636:1: D400 First line should end with a period
src/uproot/behaviors/TBranch.py:1705:1: D205 1 blank line required between summary line and description
src/uproot/behaviors/TBranch.py:1705:1: D400 First line should end with a period
src/uproot/behaviors/TBranch.py:1735:1: D200 One-line docstring should fit on one line with quotes
src/uproot/behaviors/TBranch.py:1753:1: D205 1 blank line required between summary line and description
src/uproot/behaviors/TBranch.py:1753:1: D400 First line should end with a period
src/uproot/behaviors/TBranch.py:1834:1: D205 1 blank line required between summary line and description
src/uproot/behaviors/TBranch.py:1834:1: D400 First line should end with a period
src/uproot/behaviors/TBranch.py:1868:1: D105 Missing docstring in magic method
src/uproot/behaviors/TBranch.py:1928:1: D105 Missing docstring in magic method
src/uproot/behaviors/TBranch.py:1931:1: D105 Missing docstring in magic method
src/uproot/behaviors/TBranch.py:1954:1: D105 Missing docstring in magic method
src/uproot/behaviors/TBranch.py:1974:1: D205 1 blank line required between summary line and description
src/uproot/behaviors/TBranch.py:1974:1: D400 First line should end with a period
src/uproot/behaviors/TBranch.py:2106:1: D105 Missing docstring in magic method
src/uproot/behaviors/TBranch.py:2126:1: D200 One-line docstring should fit on one line with quotes
src/uproot/behaviors/TBranch.py:2133:1: D200 One-line docstring should fit on one line with quotes
src/uproot/behaviors/TBranch.py:2144:1: D205 1 blank line required between summary line and description
src/uproot/behaviors/TBranch.py:2144:1: D400 First line should end with a period
src/uproot/behaviors/TBranch.py:2144:1: D401 First line should be in imperative mood; try rephrasing
src/uproot/behaviors/TBranch.py:2175:1: D205 1 blank line required between summary line and description
src/uproot/behaviors/TBranch.py:2175:1: D400 First line should end with a period
src/uproot/behaviors/TBranch.py:2175:1: D401 First line should be in imperative mood; try rephrasing
src/uproot/behaviors/TBranch.py:2201:1: D205 1 blank line required between summary line and description
src/uproot/behaviors/TBranch.py:2201:1: D400 First line should end with a period
src/uproot/behaviors/TBranch.py:2201:1: D401 First line should be in imperative mood; try rephrasing
src/uproot/behaviors/TBranch.py:2215:1: D401 First line should be in imperative mood; try rephrasing
src/uproot/behaviors/TBranch.py:2231:1: D205 1 blank line required between summary line and description
src/uproot/behaviors/TBranch.py:2231:1: D400 First line should end with a period
src/uproot/behaviors/TBranch.py:2231:1: D401 First line should be in imperative mood; try rephrasing
src/uproot/behaviors/TBranch.py:2270:1: D200 One-line docstring should fit on one line with quotes
src/uproot/behaviors/TBranch.py:2270:1: D401 First line should be in imperative mood; try rephrasing
src/uproot/behaviors/TBranch.py:2303:1: D205 1 blank line required between summary line and description
src/uproot/behaviors/TBranch.py:2303:1: D400 First line should end with a period
src/uproot/behaviors/TBranch.py:2303:1: D401 First line should be in imperative mood; try rephrasing
src/uproot/behaviors/TBranch.py:2314:1: D205 1 blank line required between summary line and description
src/uproot/behaviors/TBranch.py:2314:1: D400 First line should end with a period
src/uproot/behaviors/TBranch.py:2314:1: D401 First line should be in imperative mood; try rephrasing
src/uproot/behaviors/TBranch.py:2322:1: D205 1 blank line required between summary line and description
src/uproot/behaviors/TBranch.py:2322:1: D400 First line should end with a period
src/uproot/behaviors/TBranch.py:2322:1: D401 First line should be in imperative mood; try rephrasing
src/uproot/behaviors/TBranch.py:2391:1: D205 1 blank line required between summary line and description
src/uproot/behaviors/TBranch.py:2391:1: D400 First line should end with a period
src/uproot/behaviors/TBranch.py:2400:1: D205 1 blank line required between summary line and description
src/uproot/behaviors/TBranch.py:2400:1: D400 First line should end with a period
src/uproot/behaviors/TBranch.py:2400:1: D401 First line should be in imperative mood; try rephrasing
src/uproot/behaviors/TBranch.py:2415:1: D205 1 blank line required between summary line and description
src/uproot/behaviors/TBranch.py:2415:1: D400 First line should end with a period
src/uproot/behaviors/TBranch.py:2415:1: D401 First line should be in imperative mood; try rephrasing
src/uproot/behaviors/TBranch.py:2429:1: D205 1 blank line required between summary line and description
src/uproot/behaviors/TBranch.py:2429:1: D400 First line should end with a period
src/uproot/behaviors/TBranch.py:2429:1: D401 First line should be in imperative mood; try rephrasing
src/uproot/behaviors/TBranch.py:2440:1: D205 1 blank line required between summary line and description
src/uproot/behaviors/TBranch.py:2440:1: D400 First line should end with a period
src/uproot/behaviors/TBranch.py:2440:1: D401 First line should be in imperative mood; try rephrasing
src/uproot/behaviors/TBranch.py:2457:1: D205 1 blank line required between summary line and description
src/uproot/behaviors/TBranch.py:2457:1: D400 First line should end with a period
src/uproot/behaviors/TBranch.py:2457:1: D401 First line should be in imperative mood; try rephrasing
src/uproot/behaviors/TBranch.py:2468:1: D205 1 blank line required between summary line and description
src/uproot/behaviors/TBranch.py:2468:1: D400 First line should end with a period
src/uproot/behaviors/TBranch.py:2468:1: D401 First line should be in imperative mood; try rephrasing
src/uproot/behaviors/TBranch.py:2479:1: D205 1 blank line required between summary line and description
src/uproot/behaviors/TBranch.py:2479:1: D400 First line should end with a period
src/uproot/behaviors/TBranch.py:2479:1: D401 First line should be in imperative mood; try rephrasing
src/uproot/behaviors/TBranch.py:2490:1: D205 1 blank line required between summary line and description
src/uproot/behaviors/TBranch.py:2490:1: D400 First line should end with a period
src/uproot/behaviors/TBranch.py:2490:1: D401 First line should be in imperative mood; try rephrasing
src/uproot/behaviors/TBranch.py:2498:1: D401 First line should be in imperative mood; try rephrasing
src/uproot/behaviors/TBranch.py:2520:1: D205 1 blank line required between summary line and description
src/uproot/behaviors/TBranch.py:2520:1: D400 First line should end with a period
src/uproot/behaviors/TBranch.py:2520:1: D401 First line should be in imperative mood
src/uproot/behaviors/TBranch.py:2558:1: D205 1 blank line required between summary line and description
src/uproot/behaviors/TBranch.py:2558:1: D400 First line should end with a period
src/uproot/behaviors/TBranch.py:2558:1: D401 First line should be in imperative mood; try rephrasing
src/uproot/behaviors/TBranch.py:2581:1: D205 1 blank line required between summary line and description
src/uproot/behaviors/TBranch.py:2581:1: D400 First line should end with a period
src/uproot/behaviors/TBranch.py:2581:1: D401 First line should be in imperative mood; try rephrasing
src/uproot/behaviors/TBranch.py:2595:1: D205 1 blank line required between summary line and description
src/uproot/behaviors/TBranch.py:2595:1: D400 First line should end with a period
src/uproot/behaviors/TBranch.py:2595:1: D401 First line should be in imperative mood; try rephrasing
src/uproot/behaviors/TBranch.py:2631:1: D205 1 blank line required between summary line and description
src/uproot/behaviors/TBranch.py:2631:1: D400 First line should end with a period
src/uproot/behaviors/TBranch.py:2631:1: D401 First line should be in imperative mood; try rephrasing
src/uproot/behaviors/TBranch.py:2653:1: D205 1 blank line required between summary line and description
src/uproot/behaviors/TBranch.py:2653:1: D400 First line should end with a period
src/uproot/behaviors/TBranch.py:2653:1: D401 First line should be in imperative mood
src/uproot/behaviors/TBranch.py:2679:1: D102 Missing docstring in public method
src/uproot/behaviors/TBranch.py:2747:1: D205 1 blank line required between summary line and description
src/uproot/behaviors/TBranch.py:2747:1: D400 First line should end with a period
src/uproot/behaviors/TBranch.py:2801:1: D205 1 blank line required between summary line and description
src/uproot/behaviors/TBranch.py:2801:1: D400 First line should end with a period
src/uproot/models/TObjString.py:3:1: D200 One-line docstring should fit on one line with quotes
src/uproot/models/TObjString.py:29:1: D102 Missing docstring in public method
src/uproot/models/TObjString.py:50:1: D102 Missing docstring in public method
src/uproot/models/TObjString.py:62:1: D102 Missing docstring in public method
src/uproot/models/TObjString.py:77:1: D105 Missing docstring in magic method
src/uproot/models/TObjString.py:90:1: D102 Missing docstring in public method
src/uproot/extras.py:3:1: D205 1 blank line required between summary line and description
src/uproot/extras.py:3:1: D400 First line should end with a period
src/uproot/extras.py:20:1: D200 One-line docstring should fit on one line with quotes
src/uproot/extras.py:20:1: D401 First line should be in imperative mood
src/uproot/extras.py:46:1: D200 One-line docstring should fit on one line with quotes
src/uproot/extras.py:46:1: D401 First line should be in imperative mood
src/uproot/extras.py:66:1: D205 1 blank line required between summary line and description
src/uproot/extras.py:66:1: D400 First line should end with a period
src/uproot/extras.py:66:1: D401 First line should be in imperative mood
src/uproot/extras.py:91:1: D400 First line should end with a period
src/uproot/extras.py:112:1: D205 1 blank line required between summary line and description
src/uproot/extras.py:112:1: D400 First line should end with a period
src/uproot/extras.py:130:1: D200 One-line docstring should fit on one line with quotes
src/uproot/extras.py:130:1: D401 First line should be in imperative mood
src/uproot/extras.py:145:1: D205 1 blank line required between summary line and description
src/uproot/extras.py:145:1: D400 First line should end with a period
src/uproot/extras.py:145:1: D401 First line should be in imperative mood
src/uproot/extras.py:155:1: D401 First line should be in imperative mood
src/uproot/extras.py:178:1: D401 First line should be in imperative mood
src/uproot/extras.py:201:1: D200 One-line docstring should fit on one line with quotes
src/uproot/extras.py:201:1: D401 First line should be in imperative mood
src/uproot/extras.py:221:1: D200 One-line docstring should fit on one line with quotes
src/uproot/extras.py:221:1: D401 First line should be in imperative mood
src/uproot/extras.py:241:1: D200 One-line docstring should fit on one line with quotes
src/uproot/extras.py:241:1: D401 First line should be in imperative mood
src/uproot/models/TTable.py:23:1: D200 One-line docstring should fit on one line with quotes
src/uproot/models/TTable.py:27:1: D102 Missing docstring in public method
src/uproot/models/TTable.py:64:1: D200 One-line docstring should fit on one line with quotes
src/uproot/models/TTable.py:129:1: D200 One-line docstring should fit on one line with quotes
src/uproot/models/TTable.py:133:1: D102 Missing docstring in public method
src/uproot/models/TTable.py:178:1: D200 One-line docstring should fit on one line with quotes
src/uproot/models/TTable.py:189:1: D200 One-line docstring should fit on one line with quotes
src/uproot/models/TTable.py:193:1: D102 Missing docstring in public method
src/uproot/models/TTable.py:247:1: D102 Missing docstring in public method
src/uproot/models/TTable.py:259:1: D200 One-line docstring should fit on one line with quotes
src/uproot/models/TString.py:3:1: D200 One-line docstring should fit on one line with quotes
src/uproot/models/TString.py:18:1: D102 Missing docstring in public method
src/uproot/models/TString.py:21:1: D102 Missing docstring in public method
src/uproot/models/TString.py:31:1: D102 Missing docstring in public method
src/uproot/models/TString.py:42:1: D105 Missing docstring in magic method
src/uproot/models/TString.py:51:1: D102 Missing docstring in public method
src/uproot/models/TString.py:55:1: D102 Missing docstring in public method
src/uproot/compression.py:3:1: D205 1 blank line required between summary line and description
src/uproot/compression.py:3:1: D400 First line should end with a period
src/uproot/compression.py:18:1: D200 One-line docstring should fit on one line with quotes
src/uproot/compression.py:22:1: D107 Missing docstring in __init__
src/uproot/compression.py:25:1: D105 Missing docstring in magic method
src/uproot/compression.py:30:1: D205 1 blank line required between summary line and description
src/uproot/compression.py:30:1: D400 First line should end with a period
src/uproot/compression.py:30:1: D401 First line should be in imperative mood
src/uproot/compression.py:38:1: D205 1 blank line required between summary line and description
src/uproot/compression.py:38:1: D400 First line should end with a period
src/uproot/compression.py:38:1: D401 First line should be in imperative mood
src/uproot/compression.py:51:1: D205 1 blank line required between summary line and description
src/uproot/compression.py:51:1: D400 First line should end with a period
src/uproot/compression.py:51:1: D401 First line should be in imperative mood; try rephrasing
src/uproot/compression.py:60:1: D205 1 blank line required between summary line and description
src/uproot/compression.py:60:1: D400 First line should end with a period
src/uproot/compression.py:60:1: D401 First line should be in imperative mood; try rephrasing
src/uproot/compression.py:72:1: D205 1 blank line required between summary line and description
src/uproot/compression.py:72:1: D400 First line should end with a period
src/uproot/compression.py:72:1: D401 First line should be in imperative mood; try rephrasing
src/uproot/compression.py:78:1: D105 Missing docstring in magic method
src/uproot/compression.py:97:1: D205 1 blank line required between summary line and description
src/uproot/compression.py:97:1: D400 First line should end with a period
src/uproot/compression.py:107:1: D107 Missing docstring in __init__
src/uproot/compression.py:113:1: D205 1 blank line required between summary line and description
src/uproot/compression.py:113:1: D400 First line should end with a period
src/uproot/compression.py:113:1: D401 First line should be in imperative mood; try rephrasing
src/uproot/compression.py:127:1: D102 Missing docstring in public method
src/uproot/compression.py:144:1: D205 1 blank line required between summary line and description
src/uproot/compression.py:144:1: D400 First line should end with a period
src/uproot/compression.py:156:1: D107 Missing docstring in __init__
src/uproot/compression.py:162:1: D205 1 blank line required between summary line and description
src/uproot/compression.py:162:1: D400 First line should end with a period
src/uproot/compression.py:162:1: D401 First line should be in imperative mood; try rephrasing
src/uproot/compression.py:176:1: D102 Missing docstring in public method
src/uproot/compression.py:196:1: D205 1 blank line required between summary line and description
src/uproot/compression.py:196:1: D400 First line should end with a period
src/uproot/compression.py:206:1: D107 Missing docstring in __init__
src/uproot/compression.py:212:1: D205 1 blank line required between summary line and description
src/uproot/compression.py:212:1: D400 First line should end with a period
src/uproot/compression.py:212:1: D401 First line should be in imperative mood; try rephrasing
src/uproot/compression.py:226:1: D102 Missing docstring in public method
src/uproot/compression.py:251:1: D205 1 blank line required between summary line and description
src/uproot/compression.py:251:1: D400 First line should end with a period
src/uproot/compression.py:261:1: D107 Missing docstring in __init__
src/uproot/compression.py:268:1: D205 1 blank line required between summary line and description
src/uproot/compression.py:268:1: D400 First line should end with a period
src/uproot/compression.py:268:1: D401 First line should be in imperative mood; try rephrasing
src/uproot/compression.py:283:1: D102 Missing docstring in public method
src/uproot/compression.py:289:1: D102 Missing docstring in public method
src/uproot/compression.py:312:1: D205 1 blank line required between summary line and description
src/uproot/compression.py:312:1: D400 First line should end with a period
src/uproot/compression.py:481:1: D205 1 blank line required between summary line and description
src/uproot/compression.py:481:1: D400 First line should end with a period
src/uproot/interpretation/numerical.py:3:1: D205 1 blank line required between summary line and description
src/uproot/interpretation/numerical.py:3:1: D400 First line should end with a period
src/uproot/interpretation/numerical.py:34:1: D400 First line should end with a period
src/uproot/interpretation/numerical.py:45:1: D102 Missing docstring in public method
src/uproot/interpretation/numerical.py:162:1: D205 1 blank line required between summary line and description
src/uproot/interpretation/numerical.py:162:1: D400 First line should end with a period
src/uproot/interpretation/numerical.py:176:1: D107 Missing docstring in __init__
src/uproot/interpretation/numerical.py:183:1: D105 Missing docstring in magic method
src/uproot/interpretation/numerical.py:191:1: D105 Missing docstring in magic method
src/uproot/interpretation/numerical.py:200:1: D205 1 blank line required between summary line and description
src/uproot/interpretation/numerical.py:200:1: D400 First line should end with a period
src/uproot/interpretation/numerical.py:200:1: D401 First line should be in imperative mood; try rephrasing
src/uproot/interpretation/numerical.py:217:1: D205 1 blank line required between summary line and description
src/uproot/interpretation/numerical.py:217:1: D400 First line should end with a period
src/uproot/interpretation/numerical.py:217:1: D401 First line should be in imperative mood; try rephrasing
src/uproot/interpretation/numerical.py:234:1: D205 1 blank line required between summary line and description
src/uproot/interpretation/numerical.py:234:1: D400 First line should end with a period
src/uproot/interpretation/numerical.py:234:1: D401 First line should be in imperative mood; try rephrasing
src/uproot/interpretation/numerical.py:244:1: D102 Missing docstring in public method
src/uproot/interpretation/numerical.py:249:1: D102 Missing docstring in public method
src/uproot/interpretation/numerical.py:252:1: D102 Missing docstring in public method
src/uproot/interpretation/numerical.py:270:1: D102 Missing docstring in public method
src/uproot/interpretation/numerical.py:307:1: D102 Missing docstring in public method
src/uproot/interpretation/numerical.py:325:1: D102 Missing docstring in public method
src/uproot/interpretation/numerical.py:367:1: D102 Missing docstring in public method
src/uproot/interpretation/numerical.py:374:1: D401 First line should be in imperative mood
src/uproot/interpretation/numerical.py:388:1: D205 1 blank line required between summary line and description
src/uproot/interpretation/numerical.py:388:1: D400 First line should end with a period
src/uproot/interpretation/numerical.py:393:1: D107 Missing docstring in __init__
src/uproot/interpretation/numerical.py:399:1: D200 One-line docstring should fit on one line with quotes
src/uproot/interpretation/numerical.py:421:1: D200 One-line docstring should fit on one line with quotes
src/uproot/interpretation/numerical.py:425:1: D107 Missing docstring in __init__
src/uproot/interpretation/numerical.py:429:1: D102 Missing docstring in public method
src/uproot/interpretation/numerical.py:434:1: D205 1 blank line required between summary line and description
src/uproot/interpretation/numerical.py:434:1: D400 First line should end with a period
src/uproot/interpretation/numerical.py:446:1: D200 One-line docstring should fit on one line with quotes
src/uproot/interpretation/numerical.py:453:1: D200 One-line docstring should fit on one line with quotes
src/uproot/interpretation/numerical.py:460:1: D200 One-line docstring should fit on one line with quotes
src/uproot/interpretation/numerical.py:460:1: D401 First line should be in imperative mood; try rephrasing
src/uproot/interpretation/numerical.py:467:1: D401 First line should be in imperative mood; try rephrasing
src/uproot/interpretation/numerical.py:481:1: D205 1 blank line required between summary line and description
src/uproot/interpretation/numerical.py:481:1: D400 First line should end with a period
src/uproot/interpretation/numerical.py:481:1: D401 First line should be in imperative mood; try rephrasing
src/uproot/interpretation/numerical.py:489:1: D200 One-line docstring should fit on one line with quotes
src/uproot/interpretation/numerical.py:489:1: D401 First line should be in imperative mood; try rephrasing
src/uproot/interpretation/numerical.py:496:1: D205 1 blank line required between summary line and description
src/uproot/interpretation/numerical.py:496:1: D400 First line should end with a period
src/uproot/interpretation/numerical.py:503:1: D105 Missing docstring in magic method
src/uproot/interpretation/numerical.py:509:1: D105 Missing docstring in magic method
src/uproot/interpretation/numerical.py:519:1: D102 Missing docstring in public method
src/uproot/interpretation/numerical.py:523:1: D102 Missing docstring in public method
src/uproot/interpretation/numerical.py:528:1: D102 Missing docstring in public method
src/uproot/interpretation/numerical.py:598:1: D205 1 blank line required between summary line and description
src/uproot/interpretation/numerical.py:598:1: D400 First line should end with a period
src/uproot/interpretation/numerical.py:609:1: D107 Missing docstring in __init__
src/uproot/interpretation/numerical.py:622:1: D401 First line should be in imperative mood; try rephrasing
src/uproot/interpretation/numerical.py:633:1: D102 Missing docstring in public method
src/uproot/interpretation/numerical.py:636:1: D102 Missing docstring in public method
src/uproot/interpretation/numerical.py:664:1: D205 1 blank line required between summary line and description
src/uproot/interpretation/numerical.py:664:1: D400 First line should end with a period
src/uproot/interpretation/numerical.py:675:1: D107 Missing docstring in __init__
src/uproot/interpretation/numerical.py:688:1: D401 First line should be in imperative mood; try rephrasing
src/uproot/interpretation/numerical.py:699:1: D102 Missing docstring in public method
src/uproot/interpretation/numerical.py:702:1: D102 Missing docstring in public method
src/uproot/behaviors/RooCurve.py:3:1: D200 One-line docstring should fit on one line with quotes
src/uproot/behaviors/RooCurve.py:50:1: D200 One-line docstring should fit on one line with quotes
src/uproot/behaviors/RooCurve.py:50:1: D401 First line should be in imperative mood; try rephrasing
src/uproot/behaviors/RooCurve.py:57:1: D200 One-line docstring should fit on one line with quotes
src/uproot/behaviors/RooCurve.py:57:1: D401 First line should be in imperative mood; try rephrasing
src/uproot/behaviors/RooCurve.py:64:1: D401 First line should be in imperative mood
src/uproot/behaviors/RooCurve.py:76:1: D205 1 blank line required between summary line and description
src/uproot/behaviors/RooCurve.py:76:1: D400 First line should end with a period
src/uproot/behaviors/RooCurve.py:91:1: D205 1 blank line required between summary line and description
src/uproot/behaviors/RooCurve.py:91:1: D400 First line should end with a period
src/uproot/behaviors/RooCurve.py:110:1: D205 1 blank line required between summary line and description
src/uproot/behaviors/RooCurve.py:110:1: D400 First line should end with a period
src/uproot/behaviors/RooCurve.py:125:1: D205 1 blank line required between summary line and description
src/uproot/behaviors/RooCurve.py:125:1: D400 First line should end with a period
src/uproot/behaviors/RooCurve.py:166:1: D205 1 blank line required between summary line and description
src/uproot/behaviors/RooCurve.py:166:1: D400 First line should end with a period
src/uproot/behaviors/TDatime.py:3:1: D200 One-line docstring should fit on one line with quotes
src/uproot/behaviors/TDatime.py:12:1: D200 One-line docstring should fit on one line with quotes
src/uproot/behaviors/TDatime.py:12:1: D400 First line should end with a period
src/uproot/behaviors/TDatime.py:16:1: D102 Missing docstring in public method
src/uproot/language/python.py:3:1: D205 1 blank line required between summary line and description
src/uproot/language/python.py:3:1: D400 First line should end with a period
src/uproot/language/python.py:238:1: D205 1 blank line required between summary line and description
src/uproot/language/python.py:238:1: D400 First line should end with a period
src/uproot/language/python.py:320:1: D107 Missing docstring in __init__
src/uproot/language/python.py:327:1: D105 Missing docstring in magic method
src/uproot/language/python.py:330:1: D105 Missing docstring in magic method
src/uproot/language/python.py:335:1: D200 One-line docstring should fit on one line with quotes
src/uproot/language/python.py:335:1: D401 First line should be in imperative mood
src/uproot/language/python.py:342:1: D205 1 blank line required between summary line and description
src/uproot/language/python.py:342:1: D400 First line should end with a period
src/uproot/language/python.py:349:1: D205 1 blank line required between summary line and description
src/uproot/language/python.py:349:1: D400 First line should end with a period
src/uproot/language/python.py:349:1: D401 First line should be in imperative mood
src/uproot/language/python.py:358:1: D205 1 blank line required between summary line and description
src/uproot/language/python.py:358:1: D400 First line should end with a period
src/uproot/language/python.py:398:1: D205 1 blank line required between summary line and description
src/uproot/language/python.py:398:1: D400 First line should end with a period
src/uproot/interpretation/jagged.py:3:1: D205 1 blank line required between summary line and description
src/uproot/interpretation/jagged.py:3:1: D400 First line should end with a period
src/uproot/interpretation/jagged.py:19:1: D205 1 blank line required between summary line and description
src/uproot/interpretation/jagged.py:19:1: D400 First line should end with a period
src/uproot/interpretation/jagged.py:36:1: D205 1 blank line required between summary line and description
src/uproot/interpretation/jagged.py:36:1: D400 First line should end with a period
src/uproot/interpretation/jagged.py:54:1: D107 Missing docstring in __init__
src/uproot/interpretation/jagged.py:64:1: D205 1 blank line required between summary line and description
src/uproot/interpretation/jagged.py:64:1: D400 First line should end with a period
src/uproot/interpretation/jagged.py:64:1: D401 First line should be in imperative mood; try rephrasing
src/uproot/interpretation/jagged.py:73:1: D200 One-line docstring should fit on one line with quotes
src/uproot/interpretation/jagged.py:73:1: D401 First line should be in imperative mood; try rephrasing
src/uproot/interpretation/jagged.py:80:1: D205 1 blank line required between summary line and description
src/uproot/interpretation/jagged.py:80:1: D400 First line should end with a period
src/uproot/interpretation/jagged.py:88:1: D105 Missing docstring in magic method
src/uproot/interpretation/jagged.py:96:1: D105 Missing docstring in magic method
src/uproot/interpretation/jagged.py:104:1: D102 Missing docstring in public method
src/uproot/interpretation/jagged.py:107:1: D102 Missing docstring in public method
src/uproot/interpretation/jagged.py:125:1: D102 Missing docstring in public method
src/uproot/interpretation/jagged.py:131:1: D102 Missing docstring in public method
src/uproot/interpretation/jagged.py:142:1: D102 Missing docstring in public method
src/uproot/interpretation/jagged.py:215:1: D102 Missing docstring in public method
src/uproot/interpretation/jagged.py:334:1: D205 1 blank line required between summary line and description
src/uproot/interpretation/jagged.py:334:1: D400 First line should end with a period
src/uproot/interpretation/jagged.py:348:1: D107 Missing docstring in __init__
src/uproot/interpretation/jagged.py:352:1: D105 Missing docstring in magic method
src/uproot/interpretation/jagged.py:357:1: D205 1 blank line required between summary line and description
src/uproot/interpretation/jagged.py:357:1: D400 First line should end with a period
src/uproot/interpretation/jagged.py:357:1: D401 First line should be in imperative mood
src/uproot/interpretation/jagged.py:365:1: D200 One-line docstring should fit on one line with quotes
src/uproot/interpretation/jagged.py:370:1: D105 Missing docstring in magic method
src/uproot/interpretation/jagged.py:373:1: D105 Missing docstring in magic method
src/uproot/interpretation/jagged.py:376:1: D105 Missing docstring in magic method
src/uproot/interpretation/jagged.py:384:1: D205 1 blank line required between summary line and description
src/uproot/interpretation/jagged.py:384:1: D400 First line should end with a period
src/uproot/models/THashList.py:3:1: D200 One-line docstring should fit on one line with quotes
src/uproot/models/THashList.py:12:1: D200 One-line docstring should fit on one line with quotes
src/uproot/models/THashList.py:16:1: D102 Missing docstring in public method
src/uproot/models/THashList.py:19:1: D102 Missing docstring in public method
src/uproot/models/THashList.py:39:1: D105 Missing docstring in magic method
src/uproot/models/THashList.py:51:1: D105 Missing docstring in magic method
src/uproot/models/THashList.py:54:1: D105 Missing docstring in magic method
src/uproot/source/object.py:18:1: D205 1 blank line required between summary line and description
src/uproot/source/object.py:18:1: D400 First line should end with a period
src/uproot/source/object.py:35:1: D107 Missing docstring in __init__
src/uproot/source/object.py:39:1: D102 Missing docstring in public method
src/uproot/source/object.py:43:1: D102 Missing docstring in public method
src/uproot/source/object.py:46:1: D105 Missing docstring in magic method
src/uproot/source/object.py:51:1: D105 Missing docstring in magic method
src/uproot/source/object.py:56:1: D205 1 blank line required between summary line and description
src/uproot/source/object.py:56:1: D400 First line should end with a period
src/uproot/source/object.py:69:1: D205 1 blank line required between summary line and description
src/uproot/source/object.py:69:1: D400 First line should end with a period
src/uproot/source/object.py:88:1: D205 1 blank line required between summary line and description
src/uproot/source/object.py:88:1: D400 First line should end with a period
src/uproot/source/object.py:109:1: D107 Missing docstring in __init__
src/uproot/cache.py:3:1: D205 1 blank line required between summary line and description
src/uproot/cache.py:3:1: D400 First line should end with a period
src/uproot/cache.py:23:1: D205 1 blank line required between summary line and description
src/uproot/cache.py:23:1: D400 First line should end with a period
src/uproot/cache.py:43:1: D200 One-line docstring should fit on one line with quotes
src/uproot/cache.py:43:1: D401 First line should be in imperative mood; try rephrasing
src/uproot/cache.py:48:1: D107 Missing docstring in __init__
src/uproot/cache.py:55:1: D105 Missing docstring in magic method
src/uproot/cache.py:60:1: D105 Missing docstring in magic method
src/uproot/cache.py:67:1: D105 Missing docstring in magic method
src/uproot/cache.py:76:1: D205 1 blank line required between summary line and description
src/uproot/cache.py:76:1: D400 First line should end with a period
src/uproot/cache.py:76:1: D401 First line should be in imperative mood; try rephrasing
src/uproot/cache.py:84:1: D200 One-line docstring should fit on one line with quotes
src/uproot/cache.py:84:1: D401 First line should be in imperative mood; try rephrasing
src/uproot/cache.py:90:1: D205 1 blank line required between summary line and description
src/uproot/cache.py:90:1: D400 First line should end with a period
src/uproot/cache.py:90:1: D401 First line should be in imperative mood
src/uproot/cache.py:104:1: D205 1 blank line required between summary line and description
src/uproot/cache.py:104:1: D400 First line should end with a period
src/uproot/cache.py:104:1: D401 First line should be in imperative mood
src/uproot/cache.py:118:1: D205 1 blank line required between summary line and description
src/uproot/cache.py:118:1: D400 First line should end with a period
src/uproot/cache.py:118:1: D401 First line should be in imperative mood
src/uproot/cache.py:131:1: D105 Missing docstring in magic method
src/uproot/cache.py:138:1: D105 Missing docstring in magic method
src/uproot/cache.py:152:1: D105 Missing docstring in magic method
src/uproot/cache.py:158:1: D105 Missing docstring in magic method
src/uproot/cache.py:163:1: D105 Missing docstring in magic method
src/uproot/cache.py:169:1: D205 1 blank line required between summary line and description
src/uproot/cache.py:169:1: D400 First line should end with a period
src/uproot/cache.py:196:1: D400 First line should end with a period
src/uproot/cache.py:196:1: D401 First line should be in imperative mood; try rephrasing
src/uproot/cache.py:212:1: D107 Missing docstring in __init__
src/uproot/cache.py:219:1: D105 Missing docstring in magic method
src/uproot/cache.py:228:1: D205 1 blank line required between summary line and description
src/uproot/cache.py:228:1: D400 First line should end with a period
src/uproot/cache.py:228:1: D401 First line should be in imperative mood; try rephrasing
src/uproot/cache.py:236:1: D200 One-line docstring should fit on one line with quotes
src/uproot/cache.py:236:1: D401 First line should be in imperative mood; try rephrasing
src/uproot/behaviors/TProfile.py:3:1: D200 One-line docstring should fit on one line with quotes
src/uproot/behaviors/TProfile.py:133:1: D200 One-line docstring should fit on one line with quotes
src/uproot/behaviors/TProfile.py:138:1: D102 Missing docstring in public method
src/uproot/behaviors/TProfile.py:142:1: D205 1 blank line required between summary line and description
src/uproot/behaviors/TProfile.py:142:1: D400 First line should end with a period
src/uproot/behaviors/TProfile.py:155:1: D205 1 blank line required between summary line and description
src/uproot/behaviors/TProfile.py:155:1: D400 First line should end with a period
src/uproot/behaviors/TProfile.py:168:1: D205 1 blank line required between summary line and description
src/uproot/behaviors/TProfile.py:168:1: D400 First line should end with a period
src/uproot/behaviors/TProfile.py:193:1: D205 1 blank line required between summary line and description
src/uproot/behaviors/TProfile.py:193:1: D400 First line should end with a period
src/uproot/behaviors/TProfile.py:220:1: D200 One-line docstring should fit on one line with quotes
src/uproot/behaviors/TProfile.py:227:1: D102 Missing docstring in public method
src/uproot/behaviors/TProfile.py:230:1: D102 Missing docstring in public method
src/uproot/behaviors/TProfile.py:237:1: D102 Missing docstring in public method
src/uproot/behaviors/TProfile.py:241:1: D102 Missing docstring in public method
src/uproot/behaviors/TProfile.py:252:1: D102 Missing docstring in public method
src/uproot/behaviors/TProfile.py:296:1: D102 Missing docstring in public method
src/uproot/models/TAtt.py:3:1: D200 One-line docstring should fit on one line with quotes
src/uproot/models/TAtt.py:18:1: D200 One-line docstring should fit on one line with quotes
src/uproot/models/TAtt.py:22:1: D102 Missing docstring in public method
src/uproot/models/TAtt.py:37:1: D102 Missing docstring in public method
src/uproot/models/TAtt.py:52:1: D102 Missing docstring in public method
src/uproot/models/TAtt.py:92:1: D200 One-line docstring should fit on one line with quotes
src/uproot/models/TAtt.py:96:1: D102 Missing docstring in public method
src/uproot/models/TAtt.py:111:1: D102 Missing docstring in public method
src/uproot/models/TAtt.py:126:1: D102 Missing docstring in public method
src/uproot/models/TAtt.py:184:1: D200 One-line docstring should fit on one line with quotes
src/uproot/models/TAtt.py:196:1: D200 One-line docstring should fit on one line with quotes
src/uproot/models/TAtt.py:200:1: D102 Missing docstring in public method
src/uproot/models/TAtt.py:213:1: D102 Missing docstring in public method
src/uproot/models/TAtt.py:227:1: D102 Missing docstring in public method
src/uproot/models/TAtt.py:264:1: D200 One-line docstring should fit on one line with quotes
src/uproot/models/TAtt.py:268:1: D102 Missing docstring in public method
src/uproot/models/TAtt.py:281:1: D102 Missing docstring in public method
src/uproot/models/TAtt.py:295:1: D102 Missing docstring in public method
src/uproot/models/TAtt.py:349:1: D200 One-line docstring should fit on one line with quotes
src/uproot/models/TAtt.py:360:1: D200 One-line docstring should fit on one line with quotes
src/uproot/models/TAtt.py:364:1: D102 Missing docstring in public method
src/uproot/models/TAtt.py:379:1: D102 Missing docstring in public method
src/uproot/models/TAtt.py:394:1: D102 Missing docstring in public method
src/uproot/models/TAtt.py:454:1: D200 One-line docstring should fit on one line with quotes
src/uproot/models/TAtt.py:462:1: D200 One-line docstring should fit on one line with quotes
src/uproot/models/TAtt.py:466:1: D102 Missing docstring in public method
src/uproot/models/TAtt.py:488:1: D102 Missing docstring in public method
src/uproot/models/TAtt.py:535:1: D102 Missing docstring in public method
src/uproot/models/TAtt.py:563:1: D102 Missing docstring in public method
src/uproot/models/TAtt.py:682:1: D200 One-line docstring should fit on one line with quotes
src/uproot/models/TAtt.py:690:1: D200 One-line docstring should fit on one line with quotes
src/uproot/models/TAtt.py:694:1: D102 Missing docstring in public method
src/uproot/models/TAtt.py:703:1: D102 Missing docstring in public method
src/uproot/models/TAtt.py:707:1: D102 Missing docstring in public method
src/uproot/models/TAtt.py:724:1: D102 Missing docstring in public method
src/uproot/models/TAtt.py:771:1: D200 One-line docstring should fit on one line with quotes
src/uproot/interpretation/grouped.py:3:1: D205 1 blank line required between summary line and description
src/uproot/interpretation/grouped.py:3:1: D400 First line should end with a period
src/uproot/interpretation/grouped.py:14:1: D205 1 blank line required between summary line and description
src/uproot/interpretation/grouped.py:14:1: D400 First line should end with a period
src/uproot/interpretation/grouped.py:33:1: D107 Missing docstring in __init__
src/uproot/interpretation/grouped.py:40:1: D200 One-line docstring should fit on one line with quotes
src/uproot/interpretation/grouped.py:40:1: D401 First line should be in imperative mood; try rephrasing
src/uproot/interpretation/grouped.py:47:1: D200 One-line docstring should fit on one line with quotes
src/uproot/interpretation/grouped.py:47:1: D401 First line should be in imperative mood; try rephrasing
src/uproot/interpretation/grouped.py:52:1: D105 Missing docstring in magic method
src/uproot/interpretation/grouped.py:55:1: D105 Missing docstring in magic method
src/uproot/interpretation/grouped.py:63:1: D102 Missing docstring in public method
src/uproot/interpretation/grouped.py:75:1: D102 Missing docstring in public method
src/uproot/interpretation/grouped.py:87:1: D102 Missing docstring in public method
src/uproot/interpretation/grouped.py:108:1: D102 Missing docstring in public method
src/uproot/interpretation/grouped.py:125:1: D102 Missing docstring in public method
src/uproot/pyroot.py:33:1: D205 1 blank line required between summary line and description
src/uproot/pyroot.py:33:1: D400 First line should end with a period
src/uproot/pyroot.py:71:1: D205 1 blank line required between summary line and description
src/uproot/pyroot.py:71:1: D400 First line should end with a period
src/uproot/pyroot.py:247:1: D205 1 blank line required between summary line and description
src/uproot/pyroot.py:247:1: D400 First line should end with a period
src/uproot/models/RNTuple.py:3:1: D200 One-line docstring should fit on one line with quotes
src/uproot/models/RNTuple.py:22:1: D200 One-line docstring should fit on one line with quotes
src/uproot/models/RNTuple.py:26:1: D102 Missing docstring in public method
src/uproot/models/RNTuple.py:92:1: D102 Missing docstring in public method
src/uproot/models/RNTuple.py:140:1: D102 Missing docstring in public method
src/uproot/writing/_cascadetree.py:3:1: D205 1 blank line required between summary line and description
src/uproot/writing/_cascadetree.py:3:1: D400 First line should end with a period
src/uproot/writing/_cascadetree.py:76:1: D107 Missing docstring in __init__
src/uproot/writing/_cascadetree.py:403:1: D105 Missing docstring in magic method
src/uproot/writing/_cascadetree.py:416:1: D102 Missing docstring in public method
src/uproot/writing/_cascadetree.py:420:1: D102 Missing docstring in public method
src/uproot/writing/_cascadetree.py:424:1: D102 Missing docstring in public method
src/uproot/writing/_cascadetree.py:428:1: D102 Missing docstring in public method
src/uproot/writing/_cascadetree.py:432:1: D102 Missing docstring in public method
src/uproot/writing/_cascadetree.py:436:1: D102 Missing docstring in public method
src/uproot/writing/_cascadetree.py:440:1: D102 Missing docstring in public method
src/uproot/writing/_cascadetree.py:444:1: D102 Missing docstring in public method
src/uproot/writing/_cascadetree.py:448:1: D102 Missing docstring in public method
src/uproot/writing/_cascadetree.py:452:1: D102 Missing docstring in public method
src/uproot/writing/_cascadetree.py:456:1: D102 Missing docstring in public method
src/uproot/writing/_cascadetree.py:460:1: D102 Missing docstring in public method
src/uproot/writing/_cascadetree.py:464:1: D102 Missing docstring in public method
src/uproot/writing/_cascadetree.py:467:1: D102 Missing docstring in public method
src/uproot/writing/_cascadetree.py:844:1: D102 Missing docstring in public method
src/uproot/writing/_cascadetree.py:1202:1: D102 Missing docstring in public method
src/uproot/writing/_cascadetree.py:1310:1: D102 Missing docstring in public method
src/uproot/writing/_cascadetree.py:1374:1: D102 Missing docstring in public method
src/uproot/writing/_cascadetree.py:1454:1: D200 One-line docstring should fit on one line with quotes
src/uproot/writing/_cascadetree.py:1454:1: D401 First line should be in imperative mood
src/uproot/writing/_cascadetree.py:1464:1: D200 One-line docstring should fit on one line with quotes
src/uproot/writing/_cascadetree.py:1464:1: D401 First line should be in imperative mood
src/uproot/source/chunk.py:3:1: D205 1 blank line required between summary line and description
src/uproot/source/chunk.py:3:1: D400 First line should end with a period
src/uproot/source/chunk.py:22:1: D205 1 blank line required between summary line and description
src/uproot/source/chunk.py:22:1: D400 First line should end with a period
src/uproot/source/chunk.py:32:1: D200 One-line docstring should fit on one line with quotes
src/uproot/source/chunk.py:32:1: D401 First line should be in imperative mood; try rephrasing
src/uproot/source/chunk.py:39:1: D205 1 blank line required between summary line and description
src/uproot/source/chunk.py:39:1: D400 First line should end with a period
src/uproot/source/chunk.py:50:1: D205 1 blank line required between summary line and description
src/uproot/source/chunk.py:50:1: D400 First line should end with a period
src/uproot/source/chunk.py:62:1: D205 1 blank line required between summary line and description
src/uproot/source/chunk.py:62:1: D400 First line should end with a period
src/uproot/source/chunk.py:93:1: D200 One-line docstring should fit on one line with quotes
src/uproot/source/chunk.py:93:1: D401 First line should be in imperative mood; try rephrasing
src/uproot/source/chunk.py:100:1: D200 One-line docstring should fit on one line with quotes
src/uproot/source/chunk.py:100:1: D401 First line should be in imperative mood; try rephrasing
src/uproot/source/chunk.py:107:1: D200 One-line docstring should fit on one line with quotes
src/uproot/source/chunk.py:107:1: D401 First line should be in imperative mood; try rephrasing
src/uproot/source/chunk.py:114:1: D205 1 blank line required between summary line and description
src/uproot/source/chunk.py:114:1: D400 First line should end with a period
src/uproot/source/chunk.py:114:1: D401 First line should be in imperative mood; try rephrasing
src/uproot/source/chunk.py:122:1: D200 One-line docstring should fit on one line with quotes
src/uproot/source/chunk.py:122:1: D401 First line should be in imperative mood; try rephrasing
src/uproot/source/chunk.py:128:1: D200 One-line docstring should fit on one line with quotes
src/uproot/source/chunk.py:135:1: D205 1 blank line required between summary line and description
src/uproot/source/chunk.py:135:1: D400 First line should end with a period
src/uproot/source/chunk.py:135:1: D401 First line should be in imperative mood; try rephrasing
src/uproot/source/chunk.py:143:1: D205 1 blank line required between summary line and description
src/uproot/source/chunk.py:143:1: D400 First line should end with a period
src/uproot/source/chunk.py:148:1: D105 Missing docstring in magic method
src/uproot/source/chunk.py:156:1: D102 Missing docstring in public method
src/uproot/source/chunk.py:166:1: D102 Missing docstring in public method
src/uproot/source/chunk.py:182:1: D200 One-line docstring should fit on one line with quotes
src/uproot/source/chunk.py:182:1: D400 First line should end with a period
src/uproot/source/chunk.py:182:1: D401 First line should be in imperative mood; try rephrasing
src/uproot/source/chunk.py:189:1: D205 1 blank line required between summary line and description
src/uproot/source/chunk.py:189:1: D400 First line should end with a period
src/uproot/source/chunk.py:189:1: D401 First line should be in imperative mood; try rephrasing
src/uproot/source/chunk.py:197:1: D205 1 blank line required between summary line and description
src/uproot/source/chunk.py:197:1: D400 First line should end with a period
src/uproot/source/chunk.py:197:1: D401 First line should be in imperative mood; try rephrasing
src/uproot/source/chunk.py:203:1: D105 Missing docstring in magic method
src/uproot/source/chunk.py:207:1: D105 Missing docstring in magic method
src/uproot/source/chunk.py:219:1: D205 1 blank line required between summary line and description
src/uproot/source/chunk.py:219:1: D400 First line should end with a period
src/uproot/source/chunk.py:250:1: D205 1 blank line required between summary line and description
src/uproot/source/chunk.py:250:1: D400 First line should end with a period
src/uproot/source/chunk.py:264:1: D107 Missing docstring in __init__
src/uproot/source/chunk.py:272:1: D105 Missing docstring in magic method
src/uproot/source/chunk.py:277:1: D200 One-line docstring should fit on one line with quotes
src/uproot/source/chunk.py:284:1: D200 One-line docstring should fit on one line with quotes
src/uproot/source/chunk.py:291:1: D205 1 blank line required between summary line and description
src/uproot/source/chunk.py:291:1: D400 First line should end with a period
src/uproot/source/chunk.py:299:1: D205 1 blank line required between summary line and description
src/uproot/source/chunk.py:299:1: D400 First line should end with a period
src/uproot/source/chunk.py:307:1: D205 1 blank line required between summary line and description
src/uproot/source/chunk.py:307:1: D400 First line should end with a period
src/uproot/source/chunk.py:315:1: D205 1 blank line required between summary line and description
src/uproot/source/chunk.py:315:1: D401 First line should be in imperative mood
src/uproot/source/chunk.py:335:1: D105 Missing docstring in magic method
src/uproot/source/chunk.py:344:1: D205 1 blank line required between summary line and description
src/uproot/source/chunk.py:344:1: D400 First line should end with a period
src/uproot/source/chunk.py:385:1: D401 First line should be in imperative mood; try rephrasing
src/uproot/source/chunk.py:396:1: D205 1 blank line required between summary line and description
src/uproot/source/chunk.py:396:1: D400 First line should end with a period
src/uproot/source/chunk.py:436:1: D205 1 blank line required between summary line and description
src/uproot/source/chunk.py:436:1: D400 First line should end with a period
src/uproot/source/__init__.py:3:1: D205 1 blank line required between summary line and description
src/uproot/source/__init__.py:3:1: D400 First line should end with a period
src/uproot/interpretation/objects.py:3:1: D205 1 blank line required between summary line and description
src/uproot/interpretation/objects.py:3:1: D400 First line should end with a period
src/uproot/interpretation/objects.py:29:1: D205 1 blank line required between summary line and description
src/uproot/interpretation/objects.py:29:1: D400 First line should end with a period
src/uproot/interpretation/objects.py:46:1: D205 1 blank line required between summary line and description
src/uproot/interpretation/objects.py:46:1: D400 First line should end with a period
src/uproot/interpretation/objects.py:62:1: D107 Missing docstring in __init__
src/uproot/interpretation/objects.py:68:1: D205 1 blank line required between summary line and description
src/uproot/interpretation/objects.py:68:1: D400 First line should end with a period
src/uproot/interpretation/objects.py:68:1: D401 First line should be in imperative mood; try rephrasing
src/uproot/interpretation/objects.py:76:1: D200 One-line docstring should fit on one line with quotes
src/uproot/interpretation/objects.py:76:1: D401 First line should be in imperative mood; try rephrasing
src/uproot/interpretation/objects.py:81:1: D105 Missing docstring in magic method
src/uproot/interpretation/objects.py:88:1: D105 Missing docstring in magic method
src/uproot/interpretation/objects.py:92:1: D102 Missing docstring in public method
src/uproot/interpretation/objects.py:96:1: D102 Missing docstring in public method
src/uproot/interpretation/objects.py:101:1: D102 Missing docstring in public method
src/uproot/interpretation/objects.py:107:1: D102 Missing docstring in public method
src/uproot/interpretation/objects.py:124:1: D102 Missing docstring in public method
src/uproot/interpretation/objects.py:174:1: D102 Missing docstring in public method
src/uproot/interpretation/objects.py:242:1: D205 1 blank line required between summary line and description
src/uproot/interpretation/objects.py:242:1: D400 First line should end with a period
src/uproot/interpretation/objects.py:242:1: D401 First line should be in imperative mood
src/uproot/interpretation/objects.py:360:1: D205 1 blank line required between summary line and description
src/uproot/interpretation/objects.py:360:1: D400 First line should end with a period
src/uproot/interpretation/objects.py:390:1: D107 Missing docstring in __init__
src/uproot/interpretation/objects.py:398:1: D205 1 blank line required between summary line and description
src/uproot/interpretation/objects.py:398:1: D400 First line should end with a period
src/uproot/interpretation/objects.py:398:1: D401 First line should be in imperative mood; try rephrasing
src/uproot/interpretation/objects.py:406:1: D205 1 blank line required between summary line and description
src/uproot/interpretation/objects.py:406:1: D400 First line should end with a period
src/uproot/interpretation/objects.py:406:1: D401 First line should be in imperative mood; try rephrasing
src/uproot/interpretation/objects.py:415:1: D205 1 blank line required between summary line and description
src/uproot/interpretation/objects.py:415:1: D400 First line should end with a period
src/uproot/interpretation/objects.py:423:1: D105 Missing docstring in magic method
src/uproot/interpretation/objects.py:431:1: D105 Missing docstring in magic method
src/uproot/interpretation/objects.py:435:1: D102 Missing docstring in public method
src/uproot/interpretation/objects.py:438:1: D102 Missing docstring in public method
src/uproot/interpretation/objects.py:463:1: D102 Missing docstring in public method
src/uproot/interpretation/objects.py:467:1: D102 Missing docstring in public method
src/uproot/interpretation/objects.py:475:1: D205 1 blank line required between summary line and description
src/uproot/interpretation/objects.py:475:1: D400 First line should end with a period
src/uproot/interpretation/objects.py:486:1: D205 1 blank line required between summary line and description
src/uproot/interpretation/objects.py:486:1: D400 First line should end with a period
src/uproot/interpretation/objects.py:494:1: D107 Missing docstring in __init__
src/uproot/interpretation/objects.py:499:1: D205 1 blank line required between summary line and description
src/uproot/interpretation/objects.py:499:1: D400 First line should end with a period
src/uproot/interpretation/objects.py:521:1: D107 Missing docstring in __init__
src/uproot/interpretation/objects.py:532:1: D105 Missing docstring in magic method
src/uproot/interpretation/objects.py:544:1: D205 1 blank line required between summary line and description
src/uproot/interpretation/objects.py:544:1: D400 First line should end with a period
src/uproot/interpretation/objects.py:544:1: D401 First line should be in imperative mood; try rephrasing
src/uproot/interpretation/objects.py:552:1: D200 One-line docstring should fit on one line with quotes
src/uproot/interpretation/objects.py:552:1: D401 First line should be in imperative mood; try rephrasing
src/uproot/interpretation/objects.py:559:1: D200 One-line docstring should fit on one line with quotes
src/uproot/interpretation/objects.py:566:1: D200 One-line docstring should fit on one line with quotes
src/uproot/interpretation/objects.py:573:1: D205 1 blank line required between summary line and description
src/uproot/interpretation/objects.py:573:1: D400 First line should end with a period
src/uproot/interpretation/objects.py:581:1: D205 1 blank line required between summary line and description
src/uproot/interpretation/objects.py:581:1: D400 First line should end with a period
src/uproot/interpretation/objects.py:589:1: D200 One-line docstring should fit on one line with quotes
src/uproot/interpretation/objects.py:597:1: D105 Missing docstring in magic method
src/uproot/interpretation/objects.py:600:1: D105 Missing docstring in magic method
src/uproot/interpretation/objects.py:631:1: D105 Missing docstring in magic method
src/uproot/interpretation/objects.py:660:1: D205 1 blank line required between summary line and description
src/uproot/interpretation/objects.py:660:1: D400 First line should end with a period
src/uproot/interpretation/objects.py:673:1: D107 Missing docstring in __init__
src/uproot/interpretation/objects.py:683:1: D200 One-line docstring should fit on one line with quotes
src/uproot/interpretation/objects.py:683:1: D401 First line should be in imperative mood; try rephrasing
src/uproot/interpretation/objects.py:690:1: D200 One-line docstring should fit on one line with quotes
src/uproot/interpretation/objects.py:696:1: D102 Missing docstring in public method
src/uproot/interpretation/objects.py:700:1: D105 Missing docstring in magic method
src/uproot/interpretation/objects.py:703:1: D105 Missing docstring in magic method
src/uproot/interpretation/objects.py:706:1: D105 Missing docstring in magic method
src/uproot/interpretation/objects.py:713:1: D105 Missing docstring in magic method
src/uproot/interpretation/objects.py:720:1: D102 Missing docstring in public method
src/uproot/reading.py:3:1: D205 1 blank line required between summary line and description
src/uproot/reading.py:3:1: D400 First line should end with a period
src/uproot/reading.py:32:1: D205 1 blank line required between summary line and description
src/uproot/reading.py:32:1: D301 Use r""" if any backslashes in a docstring
src/uproot/reading.py:32:1: D400 First line should end with a period
src/uproot/reading.py:211:1: D205 1 blank line required between summary line and description
src/uproot/reading.py:211:1: D400 First line should end with a period
src/uproot/reading.py:229:1: D205 1 blank line required between summary line and description
src/uproot/reading.py:229:1: D400 First line should end with a period
src/uproot/reading.py:229:1: D401 First line should be in imperative mood; try rephrasing
src/uproot/reading.py:237:1: D401 First line should be in imperative mood; try rephrasing
src/uproot/reading.py:279:1: D401 First line should be in imperative mood; try rephrasing
src/uproot/reading.py:294:1: D205 1 blank line required between summary line and description
src/uproot/reading.py:294:1: D400 First line should end with a period
src/uproot/reading.py:294:1: D401 First line should be in imperative mood; try rephrasing
src/uproot/reading.py:313:1: D205 1 blank line required between summary line and description
src/uproot/reading.py:313:1: D400 First line should end with a period
src/uproot/reading.py:313:1: D401 First line should be in imperative mood; try rephrasing
src/uproot/reading.py:325:1: D205 1 blank line required between summary line and description
src/uproot/reading.py:325:1: D400 First line should end with a period
src/uproot/reading.py:325:1: D401 First line should be in imperative mood; try rephrasing
src/uproot/reading.py:336:1: D205 1 blank line required between summary line and description
src/uproot/reading.py:336:1: D400 First line should end with a period
src/uproot/reading.py:346:1: D401 First line should be in imperative mood; try rephrasing
src/uproot/reading.py:355:1: D200 One-line docstring should fit on one line with quotes
src/uproot/reading.py:355:1: D401 First line should be in imperative mood; try rephrasing
src/uproot/reading.py:362:1: D205 1 blank line required between summary line and description
src/uproot/reading.py:362:1: D400 First line should end with a period
src/uproot/reading.py:362:1: D401 First line should be in imperative mood; try rephrasing
src/uproot/reading.py:370:1: D205 1 blank line required between summary line and description
src/uproot/reading.py:370:1: D400 First line should end with a period
src/uproot/reading.py:370:1: D401 First line should be in imperative mood; try rephrasing
src/uproot/reading.py:378:1: D205 1 blank line required between summary line and description
src/uproot/reading.py:378:1: D400 First line should end with a period
src/uproot/reading.py:378:1: D401 First line should be in imperative mood; try rephrasing
src/uproot/reading.py:386:1: D205 1 blank line required between summary line and description
src/uproot/reading.py:386:1: D400 First line should end with a period
src/uproot/reading.py:386:1: D401 First line should be in imperative mood; try rephrasing
src/uproot/reading.py:394:1: D205 1 blank line required between summary line and description
src/uproot/reading.py:394:1: D400 First line should end with a period
src/uproot/reading.py:394:1: D401 First line should be in imperative mood; try rephrasing
src/uproot/reading.py:402:1: D401 First line should be in imperative mood; try rephrasing
src/uproot/reading.py:420:1: D205 1 blank line required between summary line and description
src/uproot/reading.py:420:1: D400 First line should end with a period
src/uproot/reading.py:420:1: D401 First line should be in imperative mood; try rephrasing
src/uproot/reading.py:428:1: D205 1 blank line required between summary line and description
src/uproot/reading.py:428:1: D400 First line should end with a period
src/uproot/reading.py:428:1: D401 First line should be in imperative mood; try rephrasing
src/uproot/reading.py:436:1: D205 1 blank line required between summary line and description
src/uproot/reading.py:436:1: D400 First line should end with a period
src/uproot/reading.py:436:1: D401 First line should be in imperative mood; try rephrasing
src/uproot/reading.py:447:1: D205 1 blank line required between summary line and description
src/uproot/reading.py:447:1: D400 First line should end with a period
src/uproot/reading.py:466:1: D107 Missing docstring in __init__
src/uproot/reading.py:489:1: D205 1 blank line required between summary line and description
src/uproot/reading.py:489:1: D400 First line should end with a period
src/uproot/reading.py:550:1: D107 Missing docstring in __init__
src/uproot/reading.py:649:1: D105 Missing docstring in magic method
src/uproot/reading.py:654:1: D200 One-line docstring should fit on one line with quotes
src/uproot/reading.py:654:1: D401 First line should be in imperative mood; try rephrasing
src/uproot/reading.py:680:1: D401 First line should be in imperative mood; try rephrasing
src/uproot/reading.py:695:1: D105 Missing docstring in magic method
src/uproot/reading.py:699:1: D105 Missing docstring in magic method
src/uproot/reading.py:708:1: D205 1 blank line required between summary line and description
src/uproot/reading.py:708:1: D400 First line should end with a period
src/uproot/reading.py:708:1: D401 First line should be in imperative mood; try rephrasing
src/uproot/reading.py:718:1: D400 First line should end with a period
src/uproot/reading.py:718:1: D401 First line should be in imperative mood; try rephrasing
src/uproot/reading.py:748:1: D400 First line should end with a period
src/uproot/reading.py:748:1: D401 First line should be in imperative mood; try rephrasing
src/uproot/reading.py:780:1: D205 1 blank line required between summary line and description
src/uproot/reading.py:780:1: D400 First line should end with a period
src/uproot/reading.py:780:1: D401 First line should be in imperative mood; try rephrasing
src/uproot/reading.py:793:1: D205 1 blank line required between summary line and description
src/uproot/reading.py:793:1: D400 First line should end with a period
src/uproot/reading.py:842:1: D205 1 blank line required between summary line and description
src/uproot/reading.py:842:1: D400 First line should end with a period
src/uproot/reading.py:842:1: D401 First line should be in imperative mood; try rephrasing
src/uproot/reading.py:939:1: D205 1 blank line required between summary line and description
src/uproot/reading.py:939:1: D400 First line should end with a period
src/uproot/reading.py:939:1: D401 First line should be in imperative mood; try rephrasing
src/uproot/reading.py:955:1: D205 1 blank line required between summary line and description
src/uproot/reading.py:955:1: D400 First line should end with a period
src/uproot/reading.py:955:1: D401 First line should be in imperative mood
src/uproot/reading.py:974:1: D205 1 blank line required between summary line and description
src/uproot/reading.py:974:1: D400 First line should end with a period
src/uproot/reading.py:974:1: D401 First line should be in imperative mood
src/uproot/reading.py:996:1: D205 1 blank line required between summary line and description
src/uproot/reading.py:996:1: D400 First line should end with a period
src/uproot/reading.py:996:1: D401 First line should be in imperative mood
src/uproot/reading.py:1032:1: D205 1 blank line required between summary line and description
src/uproot/reading.py:1032:1: D400 First line should end with a period
src/uproot/reading.py:1047:1: D205 1 blank line required between summary line and description
src/uproot/reading.py:1047:1: D400 First line should end with a period
src/uproot/reading.py:1047:1: D401 First line should be in imperative mood; try rephrasing
src/uproot/reading.py:1068:1: D205 1 blank line required between summary line and description
src/uproot/reading.py:1068:1: D400 First line should end with a period
src/uproot/reading.py:1068:1: D401 First line should be in imperative mood; try rephrasing
src/uproot/reading.py:1088:1: D205 1 blank line required between summary line and description
src/uproot/reading.py:1088:1: D400 First line should end with a period
src/uproot/reading.py:1088:1: D401 First line should be in imperative mood
src/uproot/reading.py:1109:1: D205 1 blank line required between summary line and description
src/uproot/reading.py:1109:1: D400 First line should end with a period
src/uproot/reading.py:1109:1: D401 First line should be in imperative mood
src/uproot/reading.py:1185:1: D205 1 blank line required between summary line and description
src/uproot/reading.py:1185:1: D400 First line should end with a period
src/uproot/reading.py:1185:1: D401 First line should be in imperative mood
src/uproot/reading.py:1204:1: D205 1 blank line required between summary line and description
src/uproot/reading.py:1204:1: D400 First line should end with a period
src/uproot/reading.py:1204:1: D401 First line should be in imperative mood; try rephrasing
src/uproot/reading.py:1212:1: D205 1 blank line required between summary line and description
src/uproot/reading.py:1212:1: D400 First line should end with a period
src/uproot/reading.py:1212:1: D401 First line should be in imperative mood
src/uproot/reading.py:1222:1: D205 1 blank line required between summary line and description
src/uproot/reading.py:1222:1: D400 First line should end with a period
src/uproot/reading.py:1222:1: D401 First line should be in imperative mood
src/uproot/reading.py:1231:1: D205 1 blank line required between summary line and description
src/uproot/reading.py:1231:1: D400 First line should end with a period
src/uproot/reading.py:1231:1: D401 First line should be in imperative mood
src/uproot/reading.py:1239:1: D205 1 blank line required between summary line and description
src/uproot/reading.py:1239:1: D400 First line should end with a period
src/uproot/reading.py:1239:1: D401 First line should be in imperative mood
src/uproot/reading.py:1250:1: D205 1 blank line required between summary line and description
src/uproot/reading.py:1250:1: D400 First line should end with a period
src/uproot/reading.py:1250:1: D401 First line should be in imperative mood
src/uproot/reading.py:1260:1: D205 1 blank line required between summary line and description
src/uproot/reading.py:1260:1: D400 First line should end with a period
src/uproot/reading.py:1260:1: D401 First line should be in imperative mood
src/uproot/reading.py:1268:1: D205 1 blank line required between summary line and description
src/uproot/reading.py:1268:1: D400 First line should end with a period
src/uproot/reading.py:1268:1: D401 First line should be in imperative mood
src/uproot/reading.py:1276:1: D205 1 blank line required between summary line and description
src/uproot/reading.py:1276:1: D400 First line should end with a period
src/uproot/reading.py:1276:1: D401 First line should be in imperative mood
src/uproot/reading.py:1292:1: D205 1 blank line required between summary line and description
src/uproot/reading.py:1292:1: D400 First line should end with a period
src/uproot/reading.py:1363:1: D107 Missing docstring in __init__
src/uproot/reading.py:1461:1: D105 Missing docstring in magic method
src/uproot/reading.py:1468:1: D205 1 blank line required between summary line and description
src/uproot/reading.py:1468:1: D400 First line should end with a period
src/uproot/reading.py:1479:1: D205 1 blank line required between summary line and description
src/uproot/reading.py:1479:1: D400 First line should end with a period
src/uproot/reading.py:1490:1: D205 1 blank line required between summary line and description
src/uproot/reading.py:1490:1: D400 First line should end with a period
src/uproot/reading.py:1490:1: D401 First line should be in imperative mood; try rephrasing
src/uproot/reading.py:1498:1: D205 1 blank line required between summary line and description
src/uproot/reading.py:1498:1: D400 First line should end with a period
src/uproot/reading.py:1498:1: D401 First line should be in imperative mood; try rephrasing
src/uproot/reading.py:1512:1: D205 1 blank line required between summary line and description
src/uproot/reading.py:1512:1: D400 First line should end with a period
src/uproot/reading.py:1520:1: D205 1 blank line required between summary line and description
src/uproot/reading.py:1520:1: D400 First line should end with a period
src/uproot/reading.py:1520:1: D401 First line should be in imperative mood; try rephrasing
src/uproot/reading.py:1526:1: D105 Missing docstring in magic method
src/uproot/reading.py:1530:1: D105 Missing docstring in magic method
src/uproot/reading.py:1555:1: D205 1 blank line required between summary line and description
src/uproot/reading.py:1555:1: D400 First line should end with a period
src/uproot/reading.py:1555:1: D401 First line should be in imperative mood; try rephrasing
src/uproot/reading.py:1564:1: D205 1 blank line required between summary line and description
src/uproot/reading.py:1564:1: D400 First line should end with a period
src/uproot/reading.py:1564:1: D401 First line should be in imperative mood; try rephrasing
src/uproot/reading.py:1572:1: D401 First line should be in imperative mood; try rephrasing
src/uproot/reading.py:1586:1: D205 1 blank line required between summary line and description
src/uproot/reading.py:1586:1: D400 First line should end with a period
src/uproot/reading.py:1586:1: D401 First line should be in imperative mood; try rephrasing
src/uproot/reading.py:1599:1: D205 1 blank line required between summary line and description
src/uproot/reading.py:1599:1: D400 First line should end with a period
src/uproot/reading.py:1630:1: D205 1 blank line required between summary line and description
src/uproot/reading.py:1630:1: D400 First line should end with a period
src/uproot/reading.py:1661:1: D205 1 blank line required between summary line and description
src/uproot/reading.py:1661:1: D400 First line should end with a period
src/uproot/reading.py:1694:1: D205 1 blank line required between summary line and description
src/uproot/reading.py:1694:1: D400 First line should end with a period
src/uproot/reading.py:1726:1: D205 1 blank line required between summary line and description
src/uproot/reading.py:1726:1: D400 First line should end with a period
src/uproot/reading.py:1774:1: D205 1 blank line required between summary line and description
src/uproot/reading.py:1774:1: D400 First line should end with a period
src/uproot/reading.py:1805:1: D205 1 blank line required between summary line and description
src/uproot/reading.py:1805:1: D400 First line should end with a period
src/uproot/reading.py:1855:1: D205 1 blank line required between summary line and description
src/uproot/reading.py:1855:1: D400 First line should end with a period
src/uproot/reading.py:1898:1: D200 One-line docstring should fit on one line with quotes
src/uproot/reading.py:1903:1: D105 Missing docstring in magic method
src/uproot/reading.py:1910:1: D105 Missing docstring in magic method
src/uproot/reading.py:1918:1: D105 Missing docstring in magic method
src/uproot/reading.py:1922:1: D401 First line should be in imperative mood
src/uproot/reading.py:1937:1: D205 1 blank line required between summary line and description
src/uproot/reading.py:1937:1: D400 First line should end with a period
src/uproot/reading.py:1937:1: D401 First line should be in imperative mood
src/uproot/reading.py:1955:1: D205 1 blank line required between summary line and description
src/uproot/reading.py:1955:1: D400 First line should end with a period
src/uproot/reading.py:1955:1: D401 First line should be in imperative mood
src/uproot/reading.py:1974:1: D205 1 blank line required between summary line and description
src/uproot/reading.py:1974:1: D400 First line should end with a period
src/uproot/reading.py:1974:1: D401 First line should be in imperative mood
src/uproot/reading.py:1991:1: D205 1 blank line required between summary line and description
src/uproot/reading.py:1991:1: D400 First line should end with a period
src/uproot/reading.py:1991:1: D401 First line should be in imperative mood
src/uproot/reading.py:2054:1: D105 Missing docstring in magic method
src/uproot/reading.py:2100:1: D200 One-line docstring should fit on one line with quotes
src/uproot/reading.py:2127:1: D205 1 blank line required between summary line and description
src/uproot/reading.py:2127:1: D400 First line should end with a period
src/uproot/reading.py:2127:1: D401 First line should be in imperative mood; try rephrasing
src/uproot/reading.py:2135:1: D200 One-line docstring should fit on one line with quotes
src/uproot/reading.py:2135:1: D401 First line should be in imperative mood; try rephrasing
src/uproot/reading.py:2142:1: D200 One-line docstring should fit on one line with quotes
src/uproot/reading.py:2149:1: D200 One-line docstring should fit on one line with quotes
src/uproot/reading.py:2156:1: D205 1 blank line required between summary line and description
src/uproot/reading.py:2156:1: D400 First line should end with a period
src/uproot/reading.py:2163:1: D205 1 blank line required between summary line and description
src/uproot/reading.py:2163:1: D400 First line should end with a period
src/uproot/reading.py:2163:1: D401 First line should be in imperative mood
src/uproot/reading.py:2173:1: D205 1 blank line required between summary line and description
src/uproot/reading.py:2173:1: D400 First line should end with a period
src/uproot/reading.py:2173:1: D401 First line should be in imperative mood
src/uproot/reading.py:2180:1: D205 1 blank line required between summary line and description
src/uproot/reading.py:2180:1: D400 First line should end with a period
src/uproot/reading.py:2180:1: D401 First line should be in imperative mood
src/uproot/reading.py:2188:1: D205 1 blank line required between summary line and description
src/uproot/reading.py:2188:1: D400 First line should end with a period
src/uproot/reading.py:2188:1: D401 First line should be in imperative mood
src/uproot/reading.py:2195:1: D205 1 blank line required between summary line and description
src/uproot/reading.py:2195:1: D400 First line should end with a period
src/uproot/reading.py:2195:1: D401 First line should be in imperative mood
src/uproot/reading.py:2203:1: D205 1 blank line required between summary line and description
src/uproot/reading.py:2203:1: D400 First line should end with a period
src/uproot/reading.py:2203:1: D401 First line should be in imperative mood
src/uproot/reading.py:2218:1: D205 1 blank line required between summary line and description
src/uproot/reading.py:2218:1: D400 First line should end with a period
src/uproot/reading.py:2237:1: D107 Missing docstring in __init__
src/uproot/reading.py:2305:1: D105 Missing docstring in magic method
src/uproot/reading.py:2316:1: D205 1 blank line required between summary line and description
src/uproot/reading.py:2316:1: D400 First line should end with a period
src/uproot/reading.py:2316:1: D401 First line should be in imperative mood; try rephrasing
src/uproot/reading.py:2324:1: D205 1 blank line required between summary line and description
src/uproot/reading.py:2324:1: D400 First line should end with a period
src/uproot/reading.py:2324:1: D401 First line should be in imperative mood; try rephrasing
src/uproot/reading.py:2333:1: D200 One-line docstring should fit on one line with quotes
src/uproot/reading.py:2333:1: D401 First line should be in imperative mood; try rephrasing
src/uproot/reading.py:2340:1: D205 1 blank line required between summary line and description
src/uproot/reading.py:2340:1: D400 First line should end with a period
src/uproot/reading.py:2340:1: D401 First line should be in imperative mood; try rephrasing
src/uproot/reading.py:2347:1: D205 1 blank line required between summary line and description
src/uproot/reading.py:2347:1: D400 First line should end with a period
src/uproot/reading.py:2347:1: D401 First line should be in imperative mood; try rephrasing
src/uproot/reading.py:2357:1: D200 One-line docstring should fit on one line with quotes
src/uproot/reading.py:2357:1: D401 First line should be in imperative mood; try rephrasing
src/uproot/reading.py:2363:1: D401 First line should be in imperative mood; try rephrasing
src/uproot/reading.py:2393:1: D205 1 blank line required between summary line and description
src/uproot/reading.py:2393:1: D400 First line should end with a period
src/uproot/reading.py:2393:1: D401 First line should be in imperative mood; try rephrasing
src/uproot/reading.py:2401:1: D401 First line should be in imperative mood; try rephrasing
src/uproot/reading.py:2412:1: D401 First line should be in imperative mood; try rephrasing
src/uproot/reading.py:2422:1: D400 First line should end with a period
src/uproot/reading.py:2422:1: D401 First line should be in imperative mood; try rephrasing
src/uproot/reading.py:2431:1: D400 First line should end with a period
src/uproot/reading.py:2431:1: D401 First line should be in imperative mood; try rephrasing
src/uproot/reading.py:2450:1: D205 1 blank line required between summary line and description
src/uproot/reading.py:2450:1: D400 First line should end with a period
src/uproot/reading.py:2450:1: D401 First line should be in imperative mood
src/uproot/reading.py:2536:1: D205 1 blank line required between summary line and description
src/uproot/reading.py:2536:1: D400 First line should end with a period
src/uproot/reading.py:2536:1: D401 First line should be in imperative mood
src/uproot/reading.py:2575:1: D200 One-line docstring should fit on one line with quotes
src/uproot/reading.py:2575:1: D401 First line should be in imperative mood; try rephrasing
src/uproot/reading.py:2582:1: D200 One-line docstring should fit on one line with quotes
src/uproot/reading.py:2589:1: D205 1 blank line required between summary line and description
src/uproot/reading.py:2589:1: D400 First line should end with a period
src/uproot/reading.py:2589:1: D401 First line should be in imperative mood; try rephrasing
src/uproot/reading.py:2607:1: D200 One-line docstring should fit on one line with quotes
src/uproot/reading.py:2607:1: D401 First line should be in imperative mood; try rephrasing
src/uproot/reading.py:2614:1: D205 1 blank line required between summary line and description
src/uproot/reading.py:2614:1: D400 First line should end with a period
src/uproot/reading.py:2614:1: D401 First line should be in imperative mood; try rephrasing
src/uproot/reading.py:2622:1: D205 1 blank line required between summary line and description
src/uproot/reading.py:2622:1: D400 First line should end with a period
src/uproot/reading.py:2630:1: D205 1 blank line required between summary line and description
src/uproot/reading.py:2630:1: D400 First line should end with a period
src/uproot/reading.py:2638:1: D205 1 blank line required between summary line and description
src/uproot/reading.py:2638:1: D400 First line should end with a period
src/uproot/reading.py:2638:1: D401 First line should be in imperative mood; try rephrasing
src/uproot/reading.py:2647:1: D205 1 blank line required between summary line and description
src/uproot/reading.py:2647:1: D400 First line should end with a period
src/uproot/reading.py:2647:1: D401 First line should be in imperative mood; try rephrasing
src/uproot/reading.py:2655:1: D205 1 blank line required between summary line and description
src/uproot/reading.py:2655:1: D400 First line should end with a period
src/uproot/reading.py:2655:1: D401 First line should be in imperative mood; try rephrasing
src/uproot/reading.py:2662:1: D205 1 blank line required between summary line and description
src/uproot/reading.py:2662:1: D400 First line should end with a period
src/uproot/reading.py:2662:1: D401 First line should be in imperative mood
src/uproot/reading.py:2672:1: D205 1 blank line required between summary line and description
src/uproot/reading.py:2672:1: D400 First line should end with a period
src/uproot/reading.py:2672:1: D401 First line should be in imperative mood
src/uproot/reading.py:2684:1: D205 1 blank line required between summary line and description
src/uproot/reading.py:2684:1: D400 First line should end with a period
src/uproot/reading.py:2684:1: D401 First line should be in imperative mood
src/uproot/streamers.py:3:1: D205 1 blank line required between summary line and description
src/uproot/streamers.py:3:1: D400 First line should end with a period
src/uproot/streamers.py:142:1: D105 Missing docstring in magic method
src/uproot/streamers.py:148:1: D205 1 blank line required between summary line and description
src/uproot/streamers.py:148:1: D400 First line should end with a period
src/uproot/streamers.py:177:1: D205 1 blank line required between summary line and description
src/uproot/streamers.py:177:1: D400 First line should end with a period
src/uproot/streamers.py:177:1: D401 First line should be in imperative mood; try rephrasing
src/uproot/streamers.py:177:1: D402 First line should not be the function's "signature"
src/uproot/streamers.py:185:1: D200 One-line docstring should fit on one line with quotes
src/uproot/streamers.py:185:1: D401 First line should be in imperative mood; try rephrasing
src/uproot/streamers.py:192:1: D205 1 blank line required between summary line and description
src/uproot/streamers.py:192:1: D400 First line should end with a period
src/uproot/streamers.py:192:1: D401 First line should be in imperative mood; try rephrasing
src/uproot/streamers.py:200:1: D200 One-line docstring should fit on one line with quotes
src/uproot/streamers.py:200:1: D401 First line should be in imperative mood; try rephrasing
src/uproot/streamers.py:206:1: D205 1 blank line required between summary line and description
src/uproot/streamers.py:206:1: D400 First line should end with a period
src/uproot/streamers.py:206:1: D401 First line should be in imperative mood
src/uproot/streamers.py:353:1: D205 1 blank line required between summary line and description
src/uproot/streamers.py:353:1: D400 First line should end with a period
src/uproot/streamers.py:372:1: D205 1 blank line required between summary line and description
src/uproot/streamers.py:372:1: D400 First line should end with a period
src/uproot/streamers.py:372:1: D401 First line should be in imperative mood; try rephrasing
src/uproot/streamers.py:379:1: D205 1 blank line required between summary line and description
src/uproot/streamers.py:379:1: D400 First line should end with a period
src/uproot/streamers.py:404:1: D102 Missing docstring in public method
src/uproot/streamers.py:470:1: D205 1 blank line required between summary line and description
src/uproot/streamers.py:470:1: D400 First line should end with a period
src/uproot/streamers.py:487:1: D200 One-line docstring should fit on one line with quotes
src/uproot/streamers.py:487:1: D401 First line should be in imperative mood; try rephrasing
src/uproot/streamers.py:487:1: D402 First line should not be the function's "signature"
src/uproot/streamers.py:494:1: D200 One-line docstring should fit on one line with quotes
src/uproot/streamers.py:494:1: D401 First line should be in imperative mood; try rephrasing
src/uproot/streamers.py:494:1: D402 First line should not be the function's "signature"
src/uproot/streamers.py:501:1: D200 One-line docstring should fit on one line with quotes
src/uproot/streamers.py:501:1: D401 First line should be in imperative mood; try rephrasing
src/uproot/streamers.py:501:1: D402 First line should not be the function's "signature"
src/uproot/streamers.py:508:1: D200 One-line docstring should fit on one line with quotes
src/uproot/streamers.py:508:1: D401 First line should be in imperative mood; try rephrasing
src/uproot/streamers.py:515:1: D205 1 blank line required between summary line and description
src/uproot/streamers.py:515:1: D400 First line should end with a period
src/uproot/streamers.py:515:1: D401 First line should be in imperative mood; try rephrasing
src/uproot/streamers.py:523:1: D200 One-line docstring should fit on one line with quotes
src/uproot/streamers.py:523:1: D401 First line should be in imperative mood; try rephrasing
src/uproot/streamers.py:528:1: D102 Missing docstring in public method
src/uproot/streamers.py:602:1: D102 Missing docstring in public method
src/uproot/streamers.py:619:1: D102 Missing docstring in public method
src/uproot/streamers.py:679:1: D205 1 blank line required between summary line and description
src/uproot/streamers.py:679:1: D400 First line should end with a period
src/uproot/streamers.py:679:1: D401 First line should be in imperative mood; try rephrasing
src/uproot/streamers.py:679:1: D402 First line should not be the function's "signature"
src/uproot/streamers.py:687:1: D200 One-line docstring should fit on one line with quotes
src/uproot/streamers.py:687:1: D401 First line should be in imperative mood; try rephrasing
src/uproot/streamers.py:695:1: D102 Missing docstring in public method
src/uproot/streamers.py:698:1: D102 Missing docstring in public method
src/uproot/streamers.py:741:1: D102 Missing docstring in public method
src/uproot/streamers.py:793:1: D200 One-line docstring should fit on one line with quotes
src/uproot/streamers.py:793:1: D401 First line should be in imperative mood; try rephrasing
src/uproot/streamers.py:798:1: D102 Missing docstring in public method
src/uproot/streamers.py:864:1: D102 Missing docstring in public method
src/uproot/streamers.py:896:1: D102 Missing docstring in public method
src/uproot/streamers.py:1031:1: D102 Missing docstring in public method
src/uproot/streamers.py:1125:1: D200 One-line docstring should fit on one line with quotes
src/uproot/streamers.py:1125:1: D401 First line should be in imperative mood; try rephrasing
src/uproot/streamers.py:1130:1: D102 Missing docstring in public method
src/uproot/streamers.py:1186:1: D102 Missing docstring in public method
src/uproot/streamers.py:1230:1: D200 One-line docstring should fit on one line with quotes
src/uproot/streamers.py:1230:1: D401 First line should be in imperative mood; try rephrasing
src/uproot/streamers.py:1237:1: D200 One-line docstring should fit on one line with quotes
src/uproot/streamers.py:1237:1: D401 First line should be in imperative mood; try rephrasing
src/uproot/streamers.py:1242:1: D102 Missing docstring in public method
src/uproot/streamers.py:1293:1: D102 Missing docstring in public method
src/uproot/streamers.py:1337:1: D102 Missing docstring in public method
src/uproot/streamers.py:1356:1: D205 1 blank line required between summary line and description
src/uproot/streamers.py:1356:1: D400 First line should end with a period
src/uproot/streamers.py:1362:1: D102 Missing docstring in public method
src/uproot/streamers.py:1448:1: D102 Missing docstring in public method
src/uproot/streamers.py:1475:1: D102 Missing docstring in public method
src/uproot/streamers.py:1494:1: D205 1 blank line required between summary line and description
src/uproot/streamers.py:1494:1: D400 First line should end with a period
src/uproot/streamers.py:1501:1: D102 Missing docstring in public method
src/uproot/streamers.py:1561:1: D102 Missing docstring in public method
src/uproot/streamers.py:1588:1: D102 Missing docstring in public method
src/uproot/streamers.py:1615:1: D102 Missing docstring in public method
src/uproot/sink/__init__.py:3:1: D205 1 blank line required between summary line and description
src/uproot/sink/__init__.py:3:1: D400 First line should end with a period
src/uproot/behaviors/TGraphAsymmErrors.py:3:1: D200 One-line docstring should fit on one line with quotes
src/uproot/behaviors/TGraphAsymmErrors.py:17:1: D200 One-line docstring should fit on one line with quotes
src/uproot/behaviors/TGraphAsymmErrors.py:38:1: D205 1 blank line required between summary line and description
src/uproot/behaviors/TGraphAsymmErrors.py:38:1: D400 First line should end with a period
src/uproot/behaviors/TGraphAsymmErrors.py:60:1: D205 1 blank line required between summary line and description
src/uproot/behaviors/TGraphAsymmErrors.py:60:1: D400 First line should end with a period
src/uproot/behaviors/TH1.py:3:1: D205 1 blank line required between summary line and description
src/uproot/behaviors/TH1.py:3:1: D400 First line should end with a period
src/uproot/behaviors/TH1.py:44:1: D200 One-line docstring should fit on one line with quotes
src/uproot/behaviors/TH1.py:50:1: D200 One-line docstring should fit on one line with quotes
src/uproot/behaviors/TH1.py:50:1: D401 First line should be in imperative mood; try rephrasing
src/uproot/behaviors/TH1.py:57:1: D200 One-line docstring should fit on one line with quotes
src/uproot/behaviors/TH1.py:57:1: D401 First line should be in imperative mood; try rephrasing
src/uproot/behaviors/TH1.py:63:1: D205 1 blank line required between summary line and description
src/uproot/behaviors/TH1.py:63:1: D400 First line should end with a period
src/uproot/behaviors/TH1.py:78:1: D200 One-line docstring should fit on one line with quotes
src/uproot/behaviors/TH1.py:78:1: D401 First line should be in imperative mood; try rephrasing
src/uproot/behaviors/TH1.py:85:1: D200 One-line docstring should fit on one line with quotes
src/uproot/behaviors/TH1.py:85:1: D401 First line should be in imperative mood; try rephrasing
src/uproot/behaviors/TH1.py:91:1: D401 First line should be in imperative mood
src/uproot/behaviors/TH1.py:109:1: D200 One-line docstring should fit on one line with quotes
src/uproot/behaviors/TH1.py:109:1: D401 First line should be in imperative mood; try rephrasing
src/uproot/behaviors/TH1.py:116:1: D205 1 blank line required between summary line and description
src/uproot/behaviors/TH1.py:116:1: D400 First line should end with a period
src/uproot/behaviors/TH1.py:116:1: D401 First line should be in imperative mood; try rephrasing
src/uproot/behaviors/TH1.py:123:1: D205 1 blank line required between summary line and description
src/uproot/behaviors/TH1.py:123:1: D400 First line should end with a period
src/uproot/behaviors/TH1.py:136:1: D205 1 blank line required between summary line and description
src/uproot/behaviors/TH1.py:136:1: D400 First line should end with a period
src/uproot/behaviors/TH1.py:154:1: D205 1 blank line required between summary line and description
src/uproot/behaviors/TH1.py:154:1: D400 First line should end with a period
src/uproot/behaviors/TH1.py:173:1: D205 1 blank line required between summary line and description
src/uproot/behaviors/TH1.py:173:1: D400 First line should end with a period
src/uproot/behaviors/TH1.py:184:1: D205 1 blank line required between summary line and description
src/uproot/behaviors/TH1.py:184:1: D400 First line should end with a period
src/uproot/behaviors/TH1.py:196:1: D205 1 blank line required between summary line and description
src/uproot/behaviors/TH1.py:196:1: D400 First line should end with a period
src/uproot/behaviors/TH1.py:215:1: D205 1 blank line required between summary line and description
src/uproot/behaviors/TH1.py:215:1: D400 First line should end with a period
src/uproot/behaviors/TH1.py:221:1: D102 Missing docstring in public method
src/uproot/behaviors/TH1.py:224:1: D102 Missing docstring in public method
src/uproot/behaviors/TH1.py:231:1: D102 Missing docstring in public method
src/uproot/behaviors/TH1.py:236:1: D102 Missing docstring in public method
src/uproot/behaviors/TH1.py:239:1: D102 Missing docstring in public method
src/uproot/behaviors/TH1.py:276:1: D205 1 blank line required between summary line and description
src/uproot/behaviors/TH1.py:276:1: D400 First line should end with a period
src/uproot/behaviors/TH1.py:295:1: D102 Missing docstring in public method
src/uproot/models/TLeaf.py:3:1: D200 One-line docstring should fit on one line with quotes
src/uproot/models/TLeaf.py:23:1: D200 One-line docstring should fit on one line with quotes
src/uproot/models/TLeaf.py:27:1: D102 Missing docstring in public method
src/uproot/models/TLeaf.py:78:1: D200 One-line docstring should fit on one line with quotes
src/uproot/models/TLeaf.py:89:1: D205 1 blank line required between summary line and description
src/uproot/models/TLeaf.py:89:1: D400 First line should end with a period
src/uproot/models/TLeaf.py:94:1: D102 Missing docstring in public method
src/uproot/models/TLeaf.py:137:1: D200 One-line docstring should fit on one line with quotes
src/uproot/models/TLeaf.py:148:1: D205 1 blank line required between summary line and description
src/uproot/models/TLeaf.py:148:1: D400 First line should end with a period
src/uproot/models/TLeaf.py:153:1: D102 Missing docstring in public method
src/uproot/models/TLeaf.py:196:1: D200 One-line docstring should fit on one line with quotes
src/uproot/models/TLeaf.py:207:1: D205 1 blank line required between summary line and description
src/uproot/models/TLeaf.py:207:1: D400 First line should end with a period
src/uproot/models/TLeaf.py:212:1: D102 Missing docstring in public method
src/uproot/models/TLeaf.py:255:1: D200 One-line docstring should fit on one line with quotes
src/uproot/models/TLeaf.py:266:1: D205 1 blank line required between summary line and description
src/uproot/models/TLeaf.py:266:1: D400 First line should end with a period
src/uproot/models/TLeaf.py:271:1: D102 Missing docstring in public method
src/uproot/models/TLeaf.py:314:1: D200 One-line docstring should fit on one line with quotes
src/uproot/models/TLeaf.py:325:1: D205 1 blank line required between summary line and description
src/uproot/models/TLeaf.py:325:1: D400 First line should end with a period
src/uproot/models/TLeaf.py:330:1: D102 Missing docstring in public method
src/uproot/models/TLeaf.py:373:1: D200 One-line docstring should fit on one line with quotes
src/uproot/models/TLeaf.py:384:1: D205 1 blank line required between summary line and description
src/uproot/models/TLeaf.py:384:1: D400 First line should end with a period
src/uproot/models/TLeaf.py:389:1: D102 Missing docstring in public method
src/uproot/models/TLeaf.py:432:1: D200 One-line docstring should fit on one line with quotes
src/uproot/models/TLeaf.py:443:1: D205 1 blank line required between summary line and description
src/uproot/models/TLeaf.py:443:1: D400 First line should end with a period
src/uproot/models/TLeaf.py:448:1: D102 Missing docstring in public method
src/uproot/models/TLeaf.py:491:1: D200 One-line docstring should fit on one line with quotes
src/uproot/models/TLeaf.py:502:1: D205 1 blank line required between summary line and description
src/uproot/models/TLeaf.py:502:1: D400 First line should end with a period
src/uproot/models/TLeaf.py:507:1: D102 Missing docstring in public method
src/uproot/models/TLeaf.py:550:1: D205 1 blank line required between summary line and description
src/uproot/models/TLeaf.py:550:1: D400 First line should end with a period
src/uproot/models/TLeaf.py:559:1: D205 1 blank line required between summary line and description
src/uproot/models/TLeaf.py:559:1: D400 First line should end with a period
src/uproot/models/TLeaf.py:564:1: D102 Missing docstring in public method
src/uproot/models/TLeaf.py:592:1: D205 1 blank line required between summary line and description
src/uproot/models/TLeaf.py:592:1: D400 First line should end with a period
src/uproot/models/TLeaf.py:601:1: D205 1 blank line required between summary line and description
src/uproot/models/TLeaf.py:601:1: D400 First line should end with a period
src/uproot/models/TLeaf.py:606:1: D102 Missing docstring in public method
src/uproot/models/TLeaf.py:634:1: D205 1 blank line required between summary line and description
src/uproot/models/TLeaf.py:634:1: D400 First line should end with a period
src/uproot/models/TLeaf.py:646:1: D205 1 blank line required between summary line and description
src/uproot/models/TLeaf.py:646:1: D400 First line should end with a period
src/uproot/models/TLeaf.py:651:1: D102 Missing docstring in public method
src/uproot/models/TLeaf.py:681:1: D205 1 blank line required between summary line and description
src/uproot/models/TLeaf.py:681:1: D400 First line should end with a period
dev/example-objects.py:1:1: D200 One-line docstring should fit on one line with quotes
src/uproot/interpretation/strings.py:3:1: D205 1 blank line required between summary line and description
src/uproot/interpretation/strings.py:3:1: D400 First line should end with a period
src/uproot/interpretation/strings.py:27:1: D205 1 blank line required between summary line and description
src/uproot/interpretation/strings.py:27:1: D400 First line should end with a period
src/uproot/interpretation/strings.py:59:1: D107 Missing docstring in __init__
src/uproot/interpretation/strings.py:72:1: D200 One-line docstring should fit on one line with quotes
src/uproot/interpretation/strings.py:72:1: D401 First line should be in imperative mood; try rephrasing
src/uproot/interpretation/strings.py:79:1: D205 1 blank line required between summary line and description
src/uproot/interpretation/strings.py:79:1: D400 First line should end with a period
src/uproot/interpretation/strings.py:79:1: D401 First line should be in imperative mood; try rephrasing
src/uproot/interpretation/strings.py:88:1: D205 1 blank line required between summary line and description
src/uproot/interpretation/strings.py:88:1: D400 First line should end with a period
src/uproot/interpretation/strings.py:96:1: D105 Missing docstring in magic method
src/uproot/interpretation/strings.py:104:1: D105 Missing docstring in magic method
src/uproot/interpretation/strings.py:112:1: D102 Missing docstring in public method
src/uproot/interpretation/strings.py:119:1: D102 Missing docstring in public method
src/uproot/interpretation/strings.py:122:1: D102 Missing docstring in public method
src/uproot/interpretation/strings.py:145:1: D102 Missing docstring in public method
src/uproot/interpretation/strings.py:150:1: D102 Missing docstring in public method
src/uproot/interpretation/strings.py:245:1: D102 Missing docstring in public method
src/uproot/interpretation/strings.py:354:1: D205 1 blank line required between summary line and description
src/uproot/interpretation/strings.py:354:1: D400 First line should end with a period
src/uproot/interpretation/strings.py:368:1: D107 Missing docstring in __init__
src/uproot/interpretation/strings.py:372:1: D105 Missing docstring in magic method
src/uproot/interpretation/strings.py:382:1: D205 1 blank line required between summary line and description
src/uproot/interpretation/strings.py:382:1: D400 First line should end with a period
src/uproot/interpretation/strings.py:382:1: D401 First line should be in imperative mood
src/uproot/interpretation/strings.py:390:1: D200 One-line docstring should fit on one line with quotes
src/uproot/interpretation/strings.py:395:1: D105 Missing docstring in magic method
src/uproot/interpretation/strings.py:399:1: D105 Missing docstring in magic method
src/uproot/interpretation/strings.py:402:1: D105 Missing docstring in magic method
src/uproot/models/TBranch.py:38:1: D200 One-line docstring should fit on one line with quotes
src/uproot/models/TBranch.py:44:1: D102 Missing docstring in public method
src/uproot/models/TBranch.py:130:1: D102 Missing docstring in public method
src/uproot/models/TBranch.py:168:1: D200 One-line docstring should fit on one line with quotes
src/uproot/models/TBranch.py:174:1: D102 Missing docstring in public method
src/uproot/models/TBranch.py:261:1: D102 Missing docstring in public method
src/uproot/models/TBranch.py:300:1: D200 One-line docstring should fit on one line with quotes
src/uproot/models/TBranch.py:306:1: D102 Missing docstring in public method
src/uproot/models/TBranch.py:393:1: D102 Missing docstring in public method
src/uproot/models/TBranch.py:433:1: D200 One-line docstring should fit on one line with quotes
src/uproot/models/TBranch.py:439:1: D102 Missing docstring in public method
src/uproot/models/TBranch.py:531:1: D102 Missing docstring in public method
src/uproot/models/TBranch.py:577:1: D200 One-line docstring should fit on one line with quotes
src/uproot/models/TBranch.py:595:1: D200 One-line docstring should fit on one line with quotes
src/uproot/models/TBranch.py:601:1: D102 Missing docstring in public method
src/uproot/models/TBranch.py:669:1: D200 One-line docstring should fit on one line with quotes
src/uproot/models/TBranch.py:675:1: D102 Missing docstring in public method
src/uproot/models/TBranch.py:743:1: D200 One-line docstring should fit on one line with quotes
src/uproot/models/TBranch.py:749:1: D102 Missing docstring in public method
src/uproot/models/TBranch.py:812:1: D200 One-line docstring should fit on one line with quotes
src/uproot/models/TBranch.py:826:1: D200 One-line docstring should fit on one line with quotes
src/uproot/models/TBranch.py:832:1: D102 Missing docstring in public method
src/uproot/models/TBranch.py:861:1: D200 One-line docstring should fit on one line with quotes
src/uproot/models/TH.py:3:1: D200 One-line docstring should fit on one line with quotes
src/uproot/models/TH.py:148:1: D200 One-line docstring should fit on one line with quotes
src/uproot/models/TH.py:152:1: D102 Missing docstring in public method
src/uproot/models/TH.py:206:1: D102 Missing docstring in public method
src/uproot/models/TH.py:277:1: D102 Missing docstring in public method
src/uproot/models/TH.py:333:1: D102 Missing docstring in public method
src/uproot/models/TH.py:470:1: D200 One-line docstring should fit on one line with quotes
src/uproot/models/TH.py:478:1: D200 One-line docstring should fit on one line with quotes
src/uproot/models/TH.py:482:1: D102 Missing docstring in public method
src/uproot/models/TH.py:582:1: D102 Missing docstring in public method
src/uproot/models/TH.py:724:1: D102 Missing docstring in public method
src/uproot/models/TH.py:834:1: D102 Missing docstring in public method
src/uproot/models/TH.py:1069:1: D200 One-line docstring should fit on one line with quotes
src/uproot/models/TH.py:1077:1: D200 One-line docstring should fit on one line with quotes
src/uproot/models/TH.py:1081:1: D102 Missing docstring in public method
src/uproot/models/TH.py:1107:1: D102 Missing docstring in public method
src/uproot/models/TH.py:1138:1: D102 Missing docstring in public method
src/uproot/models/TH.py:1164:1: D102 Missing docstring in public method
src/uproot/models/TH.py:1243:1: D200 One-line docstring should fit on one line with quotes
src/uproot/models/TH.py:1251:1: D200 One-line docstring should fit on one line with quotes
src/uproot/models/TH.py:1255:1: D102 Missing docstring in public method
src/uproot/models/TH.py:1295:1: D102 Missing docstring in public method
src/uproot/models/TH.py:1350:1: D102 Missing docstring in public method
src/uproot/models/TH.py:1384:1: D102 Missing docstring in public method
src/uproot/models/TH.py:1490:1: D200 One-line docstring should fit on one line with quotes
src/uproot/models/TH.py:1498:1: D200 One-line docstring should fit on one line with quotes
src/uproot/models/TH.py:1502:1: D102 Missing docstring in public method
src/uproot/models/TH.py:1533:1: D102 Missing docstring in public method
src/uproot/models/TH.py:1560:1: D102 Missing docstring in public method
src/uproot/models/TH.py:1587:1: D102 Missing docstring in public method
src/uproot/models/TH.py:1666:1: D200 One-line docstring should fit on one line with quotes
src/uproot/models/TH.py:1674:1: D200 One-line docstring should fit on one line with quotes
src/uproot/models/TH.py:1678:1: D102 Missing docstring in public method
src/uproot/models/TH.py:1709:1: D102 Missing docstring in public method
src/uproot/models/TH.py:1736:1: D102 Missing docstring in public method
src/uproot/models/TH.py:1763:1: D102 Missing docstring in public method
src/uproot/models/TH.py:1837:1: D200 One-line docstring should fit on one line with quotes
src/uproot/models/TH.py:1845:1: D200 One-line docstring should fit on one line with quotes
src/uproot/models/TH.py:1849:1: D102 Missing docstring in public method
src/uproot/models/TH.py:1880:1: D102 Missing docstring in public method
src/uproot/models/TH.py:1907:1: D102 Missing docstring in public method
src/uproot/models/TH.py:1934:1: D102 Missing docstring in public method
src/uproot/models/TH.py:2008:1: D200 One-line docstring should fit on one line with quotes
src/uproot/models/TH.py:2016:1: D200 One-line docstring should fit on one line with quotes
src/uproot/models/TH.py:2020:1: D102 Missing docstring in public method
src/uproot/models/TH.py:2051:1: D102 Missing docstring in public method
src/uproot/models/TH.py:2078:1: D102 Missing docstring in public method
src/uproot/models/TH.py:2105:1: D102 Missing docstring in public method
src/uproot/models/TH.py:2184:1: D200 One-line docstring should fit on one line with quotes
src/uproot/models/TH.py:2192:1: D200 One-line docstring should fit on one line with quotes
src/uproot/models/TH.py:2196:1: D102 Missing docstring in public method
src/uproot/models/TH.py:2227:1: D102 Missing docstring in public method
src/uproot/models/TH.py:2254:1: D102 Missing docstring in public method
src/uproot/models/TH.py:2281:1: D102 Missing docstring in public method
src/uproot/models/TH.py:2360:1: D200 One-line docstring should fit on one line with quotes
src/uproot/models/TH.py:2368:1: D200 One-line docstring should fit on one line with quotes
src/uproot/models/TH.py:2372:1: D102 Missing docstring in public method
src/uproot/models/TH.py:2403:1: D102 Missing docstring in public method
src/uproot/models/TH.py:2430:1: D102 Missing docstring in public method
src/uproot/models/TH.py:2457:1: D102 Missing docstring in public method
src/uproot/models/TH.py:2537:1: D200 One-line docstring should fit on one line with quotes
src/uproot/models/TH.py:2545:1: D200 One-line docstring should fit on one line with quotes
src/uproot/models/TH.py:2549:1: D102 Missing docstring in public method
src/uproot/models/TH.py:2580:1: D102 Missing docstring in public method
src/uproot/models/TH.py:2607:1: D102 Missing docstring in public method
src/uproot/models/TH.py:2634:1: D102 Missing docstring in public method
src/uproot/models/TH.py:2709:1: D200 One-line docstring should fit on one line with quotes
src/uproot/models/TH.py:2717:1: D200 One-line docstring should fit on one line with quotes
src/uproot/models/TH.py:2721:1: D102 Missing docstring in public method
src/uproot/models/TH.py:2752:1: D102 Missing docstring in public method
src/uproot/models/TH.py:2779:1: D102 Missing docstring in public method
src/uproot/models/TH.py:2806:1: D102 Missing docstring in public method
src/uproot/models/TH.py:2886:1: D200 One-line docstring should fit on one line with quotes
src/uproot/models/TH.py:2894:1: D200 One-line docstring should fit on one line with quotes
src/uproot/models/TH.py:2898:1: D102 Missing docstring in public method
src/uproot/models/TH.py:2929:1: D102 Missing docstring in public method
src/uproot/models/TH.py:2956:1: D102 Missing docstring in public method
src/uproot/models/TH.py:2983:1: D102 Missing docstring in public method
src/uproot/models/TH.py:3063:1: D200 One-line docstring should fit on one line with quotes
src/uproot/models/TH.py:3071:1: D200 One-line docstring should fit on one line with quotes
src/uproot/models/TH.py:3075:1: D102 Missing docstring in public method
src/uproot/models/TH.py:3106:1: D102 Missing docstring in public method
src/uproot/models/TH.py:3133:1: D102 Missing docstring in public method
src/uproot/models/TH.py:3160:1: D102 Missing docstring in public method
src/uproot/models/TH.py:3240:1: D200 One-line docstring should fit on one line with quotes
src/uproot/models/TH.py:3248:1: D200 One-line docstring should fit on one line with quotes
src/uproot/models/TH.py:3252:1: D102 Missing docstring in public method
src/uproot/models/TH.py:3283:1: D102 Missing docstring in public method
src/uproot/models/TH.py:3310:1: D102 Missing docstring in public method
src/uproot/models/TH.py:3337:1: D102 Missing docstring in public method
src/uproot/models/TH.py:3418:1: D200 One-line docstring should fit on one line with quotes
src/uproot/models/TH.py:3426:1: D200 One-line docstring should fit on one line with quotes
src/uproot/models/TH.py:3430:1: D102 Missing docstring in public method
src/uproot/models/TH.py:3461:1: D102 Missing docstring in public method
src/uproot/models/TH.py:3488:1: D102 Missing docstring in public method
src/uproot/models/TH.py:3515:1: D102 Missing docstring in public method
src/uproot/models/TH.py:3591:1: D200 One-line docstring should fit on one line with quotes
src/uproot/models/TH.py:3599:1: D200 One-line docstring should fit on one line with quotes
src/uproot/models/TH.py:3603:1: D102 Missing docstring in public method
src/uproot/models/TH.py:3634:1: D102 Missing docstring in public method
src/uproot/models/TH.py:3661:1: D102 Missing docstring in public method
src/uproot/models/TH.py:3688:1: D102 Missing docstring in public method
src/uproot/models/TH.py:3769:1: D200 One-line docstring should fit on one line with quotes
src/uproot/models/TH.py:3777:1: D200 One-line docstring should fit on one line with quotes
src/uproot/models/TH.py:3781:1: D102 Missing docstring in public method
src/uproot/models/TH.py:3812:1: D102 Missing docstring in public method
src/uproot/models/TH.py:3839:1: D102 Missing docstring in public method
src/uproot/models/TH.py:3866:1: D102 Missing docstring in public method
src/uproot/models/TH.py:3947:1: D200 One-line docstring should fit on one line with quotes
src/uproot/models/TH.py:3955:1: D200 One-line docstring should fit on one line with quotes
src/uproot/models/TH.py:3959:1: D102 Missing docstring in public method
src/uproot/models/TH.py:3990:1: D102 Missing docstring in public method
src/uproot/models/TH.py:4017:1: D102 Missing docstring in public method
src/uproot/models/TH.py:4044:1: D102 Missing docstring in public method
src/uproot/models/TH.py:4125:1: D200 One-line docstring should fit on one line with quotes
src/uproot/models/TH.py:4135:1: D200 One-line docstring should fit on one line with quotes
src/uproot/models/TH.py:4139:1: D102 Missing docstring in public method
src/uproot/models/TH.py:4172:1: D102 Missing docstring in public method
src/uproot/models/TH.py:4215:1: D102 Missing docstring in public method
src/uproot/models/TH.py:4258:1: D102 Missing docstring in public method
src/uproot/models/TH.py:4379:1: D200 One-line docstring should fit on one line with quotes
src/uproot/models/TH.py:4389:1: D200 One-line docstring should fit on one line with quotes
src/uproot/models/TH.py:4393:1: D102 Missing docstring in public method
src/uproot/models/TH.py:4426:1: D102 Missing docstring in public method
src/uproot/models/TH.py:4469:1: D102 Missing docstring in public method
src/uproot/models/TH.py:4512:1: D102 Missing docstring in public method
src/uproot/models/TH.py:4635:1: D200 One-line docstring should fit on one line with quotes
src/uproot/models/TH.py:4645:1: D200 One-line docstring should fit on one line with quotes
src/uproot/models/TH.py:4649:1: D102 Missing docstring in public method
src/uproot/models/TH.py:4682:1: D102 Missing docstring in public method
src/uproot/models/TH.py:4725:1: D102 Missing docstring in public method
src/uproot/models/TH.py:4768:1: D102 Missing docstring in public method
src/uproot/models/TH.py:4892:1: D200 One-line docstring should fit on one line with quotes
src/uproot/behaviors/TGraph.py:3:1: D200 One-line docstring should fit on one line with quotes
src/uproot/behaviors/TGraph.py:12:1: D200 One-line docstring should fit on one line with quotes
src/uproot/behaviors/TGraph.py:12:1: D400 First line should end with a period
src/uproot/behaviors/TGraph.py:23:1: D205 1 blank line required between summary line and description
src/uproot/behaviors/TGraph.py:23:1: D400 First line should end with a period
src/uproot/serialization.py:3:1: D205 1 blank line required between summary line and description
src/uproot/serialization.py:3:1: D400 First line should end with a period
src/uproot/serialization.py:20:1: D301 Use r""" if any backslashes in a docstring
src/uproot/serialization.py:20:1: D401 First line should be in imperative mood
src/uproot/serialization.py:36:1: D205 1 blank line required between summary line and description
src/uproot/serialization.py:36:1: D400 First line should end with a period
src/uproot/serialization.py:74:1: D205 1 blank line required between summary line and description
src/uproot/serialization.py:74:1: D400 First line should end with a period
src/uproot/behaviors/TBranchElement.py:3:1: D205 1 blank line required between summary line and description
src/uproot/behaviors/TBranchElement.py:3:1: D400 First line should end with a period
src/uproot/behaviors/TBranchElement.py:13:1: D205 1 blank line required between summary line and description
src/uproot/behaviors/TBranchElement.py:13:1: D400 First line should end with a period
src/uproot/behaviors/RooHist.py:3:1: D200 One-line docstring should fit on one line with quotes
src/uproot/behaviors/RooHist.py:29:1: D200 One-line docstring should fit on one line with quotes
src/uproot/behaviors/RooHist.py:29:1: D401 First line should be in imperative mood; try rephrasing
src/uproot/behaviors/RooHist.py:36:1: D200 One-line docstring should fit on one line with quotes
src/uproot/behaviors/RooHist.py:36:1: D401 First line should be in imperative mood; try rephrasing
src/uproot/behaviors/RooHist.py:42:1: D205 1 blank line required between summary line and description
src/uproot/behaviors/RooHist.py:42:1: D400 First line should end with a period
src/uproot/behaviors/RooHist.py:64:1: D200 One-line docstring should fit on one line with quotes
src/uproot/behaviors/RooHist.py:64:1: D401 First line should be in imperative mood
src/uproot/behaviors/RooHist.py:95:1: D200 One-line docstring should fit on one line with quotes
src/uproot/behaviors/RooHist.py:95:1: D401 First line should be in imperative mood
src/uproot/models/TRef.py:3:1: D200 One-line docstring should fit on one line with quotes
src/uproot/models/TRef.py:30:1: D200 One-line docstring should fit on one line with quotes
src/uproot/models/TRef.py:30:1: D401 First line should be in imperative mood; try rephrasing
src/uproot/models/TRef.py:35:1: D102 Missing docstring in public method
src/uproot/models/TRef.py:38:1: D102 Missing docstring in public method
src/uproot/models/TRef.py:48:1: D105 Missing docstring in magic method
src/uproot/models/TRef.py:52:1: D102 Missing docstring in public method
src/uproot/models/TRef.py:66:1: D102 Missing docstring in public method
src/uproot/models/TRef.py:120:1: D200 One-line docstring should fit on one line with quotes
src/uproot/models/TRef.py:120:1: D401 First line should be in imperative mood; try rephrasing
src/uproot/models/TRef.py:127:1: D200 One-line docstring should fit on one line with quotes
src/uproot/models/TRef.py:127:1: D401 First line should be in imperative mood; try rephrasing
src/uproot/models/TRef.py:134:1: D200 One-line docstring should fit on one line with quotes
src/uproot/models/TRef.py:134:1: D401 First line should be in imperative mood; try rephrasing
src/uproot/models/TRef.py:139:1: D102 Missing docstring in public method
src/uproot/models/TRef.py:155:1: D105 Missing docstring in magic method
src/uproot/models/TRef.py:158:1: D105 Missing docstring in magic method
src/uproot/models/TRef.py:161:1: D105 Missing docstring in magic method
src/uproot/models/TRef.py:180:1: D102 Missing docstring in public method
src/uproot/writing/__init__.py:3:1: D205 1 blank line required between summary line and description
src/uproot/writing/__init__.py:3:1: D400 First line should end with a period
src/uproot/behaviors/TGraphErrors.py:3:1: D200 One-line docstring should fit on one line with quotes
src/uproot/behaviors/TGraphErrors.py:9:1: D200 One-line docstring should fit on one line with quotes
src/uproot/writing/writable.py:40:1: D205 1 blank line required between summary line and description
src/uproot/writing/writable.py:40:1: D400 First line should end with a period
src/uproot/writing/writable.py:72:1: D205 1 blank line required between summary line and description
src/uproot/writing/writable.py:72:1: D400 First line should end with a period
src/uproot/writing/writable.py:129:1: D205 1 blank line required between summary line and description
src/uproot/writing/writable.py:129:1: D400 First line should end with a period
src/uproot/writing/writable.py:184:1: D205 1 blank line required between summary line and description
src/uproot/writing/writable.py:184:1: D400 First line should end with a period
src/uproot/writing/writable.py:200:1: D107 Missing docstring in __init__
src/uproot/writing/writable.py:214:1: D105 Missing docstring in magic method
src/uproot/writing/writable.py:219:1: D205 1 blank line required between summary line and description
src/uproot/writing/writable.py:219:1: D400 First line should end with a period
src/uproot/writing/writable.py:219:1: D401 First line should be in imperative mood
src/uproot/writing/writable.py:227:1: D205 1 blank line required between summary line and description
src/uproot/writing/writable.py:227:1: D400 First line should end with a period
src/uproot/writing/writable.py:227:1: D401 First line should be in imperative mood; try rephrasing
src/uproot/writing/writable.py:239:1: D200 One-line docstring should fit on one line with quotes
src/uproot/writing/writable.py:239:1: D401 First line should be in imperative mood; try rephrasing
src/uproot/writing/writable.py:250:1: D205 1 blank line required between summary line and description
src/uproot/writing/writable.py:250:1: D400 First line should end with a period
src/uproot/writing/writable.py:250:1: D401 First line should be in imperative mood; try rephrasing
src/uproot/writing/writable.py:262:1: D200 One-line docstring should fit on one line with quotes
src/uproot/writing/writable.py:262:1: D401 First line should be in imperative mood; try rephrasing
src/uproot/writing/writable.py:269:1: D205 1 blank line required between summary line and description
src/uproot/writing/writable.py:269:1: D400 First line should end with a period
src/uproot/writing/writable.py:293:1: D205 1 blank line required between summary line and description
src/uproot/writing/writable.py:293:1: D400 First line should end with a period
src/uproot/writing/writable.py:293:1: D401 First line should be in imperative mood; try rephrasing
src/uproot/writing/writable.py:301:1: D205 1 blank line required between summary line and description
src/uproot/writing/writable.py:301:1: D400 First line should end with a period
src/uproot/writing/writable.py:301:1: D401 First line should be in imperative mood; try rephrasing
src/uproot/writing/writable.py:309:1: D205 1 blank line required between summary line and description
src/uproot/writing/writable.py:309:1: D400 First line should end with a period
src/uproot/writing/writable.py:309:1: D401 First line should be in imperative mood; try rephrasing
src/uproot/writing/writable.py:317:1: D205 1 blank line required between summary line and description
src/uproot/writing/writable.py:317:1: D400 First line should end with a period
src/uproot/writing/writable.py:317:1: D401 First line should be in imperative mood; try rephrasing
src/uproot/writing/writable.py:325:1: D205 1 blank line required between summary line and description
src/uproot/writing/writable.py:325:1: D400 First line should end with a period
src/uproot/writing/writable.py:341:1: D205 1 blank line required between summary line and description
src/uproot/writing/writable.py:341:1: D400 First line should end with a period
src/uproot/writing/writable.py:341:1: D401 First line should be in imperative mood; try rephrasing
src/uproot/writing/writable.py:349:1: D205 1 blank line required between summary line and description
src/uproot/writing/writable.py:349:1: D400 First line should end with a period
src/uproot/writing/writable.py:349:1: D401 First line should be in imperative mood; try rephrasing
src/uproot/writing/writable.py:357:1: D205 1 blank line required between summary line and description
src/uproot/writing/writable.py:357:1: D400 First line should end with a period
src/uproot/writing/writable.py:357:1: D401 First line should be in imperative mood; try rephrasing
src/uproot/writing/writable.py:365:1: D200 One-line docstring should fit on one line with quotes
src/uproot/writing/writable.py:365:1: D401 First line should be in imperative mood; try rephrasing
src/uproot/writing/writable.py:371:1: D205 1 blank line required between summary line and description
src/uproot/writing/writable.py:371:1: D400 First line should end with a period
src/uproot/writing/writable.py:379:1: D200 One-line docstring should fit on one line with quotes
src/uproot/writing/writable.py:397:1: D401 First line should be in imperative mood; try rephrasing
src/uproot/writing/writable.py:408:1: D105 Missing docstring in magic method
src/uproot/writing/writable.py:412:1: D105 Missing docstring in magic method
src/uproot/writing/writable.py:431:1: D205 1 blank line required between summary line and description
src/uproot/writing/writable.py:431:1: D400 First line should end with a period
src/uproot/writing/writable.py:499:1: D107 Missing docstring in __init__
src/uproot/writing/writable.py:505:1: D105 Missing docstring in magic method
src/uproot/writing/writable.py:512:1: D205 1 blank line required between summary line and description
src/uproot/writing/writable.py:512:1: D400 First line should end with a period
src/uproot/writing/writable.py:520:1: D205 1 blank line required between summary line and description
src/uproot/writing/writable.py:520:1: D400 First line should end with a period
src/uproot/writing/writable.py:528:1: D200 One-line docstring should fit on one line with quotes
src/uproot/writing/writable.py:535:1: D205 1 blank line required between summary line and description
src/uproot/writing/writable.py:535:1: D400 First line should end with a period
src/uproot/writing/writable.py:554:1: D401 First line should be in imperative mood; try rephrasing
src/uproot/writing/writable.py:565:1: D105 Missing docstring in magic method
src/uproot/writing/writable.py:569:1: D105 Missing docstring in magic method
src/uproot/writing/writable.py:574:1: D205 1 blank line required between summary line and description
src/uproot/writing/writable.py:574:1: D400 First line should end with a period
src/uproot/writing/writable.py:587:1: D105 Missing docstring in magic method
src/uproot/writing/writable.py:592:1: D105 Missing docstring in magic method
src/uproot/writing/writable.py:600:1: D105 Missing docstring in magic method
src/uproot/writing/writable.py:604:1: D200 One-line docstring should fit on one line with quotes
src/uproot/writing/writable.py:616:1: D205 1 blank line required between summary line and description
src/uproot/writing/writable.py:616:1: D400 First line should end with a period
src/uproot/writing/writable.py:647:1: D205 1 blank line required between summary line and description
src/uproot/writing/writable.py:647:1: D400 First line should end with a period
src/uproot/writing/writable.py:678:1: D205 1 blank line required between summary line and description
src/uproot/writing/writable.py:678:1: D400 First line should end with a period
src/uproot/writing/writable.py:711:1: D205 1 blank line required between summary line and description
src/uproot/writing/writable.py:711:1: D400 First line should end with a period
src/uproot/writing/writable.py:743:1: D205 1 blank line required between summary line and description
src/uproot/writing/writable.py:743:1: D400 First line should end with a period
src/uproot/writing/writable.py:788:1: D205 1 blank line required between summary line and description
src/uproot/writing/writable.py:788:1: D400 First line should end with a period
src/uproot/writing/writable.py:819:1: D205 1 blank line required between summary line and description
src/uproot/writing/writable.py:819:1: D400 First line should end with a period
src/uproot/writing/writable.py:854:1: D205 1 blank line required between summary line and description
src/uproot/writing/writable.py:854:1: D400 First line should end with a period
src/uproot/writing/writable.py:947:1: D105 Missing docstring in magic method
src/uproot/writing/writable.py:952:1: D105 Missing docstring in magic method
src/uproot/writing/writable.py:957:1: D105 Missing docstring in magic method
src/uproot/writing/writable.py:1140:1: D205 1 blank line required between summary line and description
src/uproot/writing/writable.py:1140:1: D400 First line should end with a period
src/uproot/writing/writable.py:1213:1: D205 1 blank line required between summary line and description
src/uproot/writing/writable.py:1213:1: D400 First line should end with a period
src/uproot/writing/writable.py:1307:1: D205 1 blank line required between summary line and description
src/uproot/writing/writable.py:1307:1: D400 First line should end with a period
src/uproot/writing/writable.py:1425:1: D205 1 blank line required between summary line and description
src/uproot/writing/writable.py:1425:1: D400 First line should end with a period
src/uproot/writing/writable.py:1462:1: D205 1 blank line required between summary line and description
src/uproot/writing/writable.py:1462:1: D400 First line should end with a period
src/uproot/writing/writable.py:1541:1: D107 Missing docstring in __init__
src/uproot/writing/writable.py:1546:1: D105 Missing docstring in magic method
src/uproot/writing/writable.py:1553:1: D200 One-line docstring should fit on one line with quotes
src/uproot/writing/writable.py:1560:1: D205 1 blank line required between summary line and description
src/uproot/writing/writable.py:1560:1: D400 First line should end with a period
src/uproot/writing/writable.py:1568:1: D200 One-line docstring should fit on one line with quotes
src/uproot/writing/writable.py:1575:1: D205 1 blank line required between summary line and description
src/uproot/writing/writable.py:1575:1: D400 First line should end with a period
src/uproot/writing/writable.py:1594:1: D401 First line should be in imperative mood; try rephrasing
src/uproot/writing/writable.py:1605:1: D105 Missing docstring in magic method
src/uproot/writing/writable.py:1609:1: D105 Missing docstring in magic method
src/uproot/writing/writable.py:1614:1: D205 1 blank line required between summary line and description
src/uproot/writing/writable.py:1614:1: D400 First line should end with a period
src/uproot/writing/writable.py:1683:1: D105 Missing docstring in magic method
src/uproot/writing/writable.py:1696:1: D200 One-line docstring should fit on one line with quotes
src/uproot/writing/writable.py:1696:1: D401 First line should be in imperative mood; try rephrasing
src/uproot/writing/writable.py:1703:1: D200 One-line docstring should fit on one line with quotes
src/uproot/writing/writable.py:1703:1: D401 First line should be in imperative mood; try rephrasing
src/uproot/writing/writable.py:1709:1: D205 1 blank line required between summary line and description
src/uproot/writing/writable.py:1709:1: D400 First line should end with a period
src/uproot/writing/writable.py:1749:1: D205 1 blank line required between summary line and description
src/uproot/writing/writable.py:1749:1: D400 First line should end with a period
src/uproot/writing/writable.py:1749:1: D401 First line should be in imperative mood
src/uproot/writing/writable.py:1787:1: D107 Missing docstring in __init__
src/uproot/writing/writable.py:1791:1: D105 Missing docstring in magic method
src/uproot/writing/writable.py:1798:1: D200 One-line docstring should fit on one line with quotes
src/uproot/writing/writable.py:1798:1: D401 First line should be in imperative mood; try rephrasing
src/uproot/writing/writable.py:1805:1: D205 1 blank line required between summary line and description
src/uproot/writing/writable.py:1805:1: D400 First line should end with a period
src/uproot/models/TTree.py:42:1: D200 One-line docstring should fit on one line with quotes
src/uproot/models/TTree.py:48:1: D102 Missing docstring in public method
src/uproot/models/TTree.py:148:1: D102 Missing docstring in public method
src/uproot/models/TTree.py:194:1: D200 One-line docstring should fit on one line with quotes
src/uproot/models/TTree.py:200:1: D102 Missing docstring in public method
src/uproot/models/TTree.py:300:1: D102 Missing docstring in public method
src/uproot/models/TTree.py:347:1: D200 One-line docstring should fit on one line with quotes
src/uproot/models/TTree.py:353:1: D102 Missing docstring in public method
src/uproot/models/TTree.py:455:1: D102 Missing docstring in public method
src/uproot/models/TTree.py:506:1: D200 One-line docstring should fit on one line with quotes
src/uproot/models/TTree.py:512:1: D102 Missing docstring in public method
src/uproot/models/TTree.py:627:1: D102 Missing docstring in public method
src/uproot/models/TTree.py:681:1: D200 One-line docstring should fit on one line with quotes
src/uproot/models/TTree.py:687:1: D102 Missing docstring in public method
src/uproot/models/TTree.py:805:1: D102 Missing docstring in public method
src/uproot/models/TTree.py:873:1: D200 One-line docstring should fit on one line with quotes
src/uproot/models/TTree.py:890:1: D200 One-line docstring should fit on one line with quotes
src/uproot/models/TTree.py:894:1: D102 Missing docstring in public method
src/uproot/behaviors/TH3.py:3:1: D205 1 blank line required between summary line and description
src/uproot/behaviors/TH3.py:3:1: D400 First line should end with a period
src/uproot/behaviors/TH3.py:16:1: D205 1 blank line required between summary line and description
src/uproot/behaviors/TH3.py:16:1: D400 First line should end with a period
src/uproot/behaviors/TH3.py:24:1: D102 Missing docstring in public method
src/uproot/behaviors/TH3.py:27:1: D102 Missing docstring in public method
src/uproot/behaviors/TH3.py:40:1: D102 Missing docstring in public method
src/uproot/behaviors/TH3.py:45:1: D102 Missing docstring in public method
src/uproot/behaviors/TH3.py:48:1: D102 Missing docstring in public method
src/uproot/behaviors/TH3.py:92:1: D205 1 blank line required between summary line and description
src/uproot/behaviors/TH3.py:92:1: D400 First line should end with a period
src/uproot/behaviors/TH3.py:113:1: D102 Missing docstring in public method
src/uproot/behaviors/TH2Poly.py:3:1: D200 One-line docstring should fit on one line with quotes
src/uproot/behaviors/TH2Poly.py:12:1: D200 One-line docstring should fit on one line with quotes
src/uproot/behaviors/TH2Poly.py:19:1: D102 Missing docstring in public method
src/uproot/behaviors/TH2Poly.py:22:1: D102 Missing docstring in public method
src/uproot/behaviors/TParameter.py:3:1: D200 One-line docstring should fit on one line with quotes
src/uproot/behaviors/TParameter.py:9:1: D200 One-line docstring should fit on one line with quotes
src/uproot/behaviors/TParameter.py:14:1: D102 Missing docstring in public method
src/uproot/behaviors/TParameter.py:17:1: D105 Missing docstring in magic method
src/uproot/behaviors/TParameter.py:20:1: D105 Missing docstring in magic method
src/uproot/behaviors/TParameter.py:23:1: D105 Missing docstring in magic method
src/uproot/behaviors/TParameter.py:28:1: D200 One-line docstring should fit on one line with quotes
src/uproot/behaviors/TParameter.py:33:1: D102 Missing docstring in public method
src/uproot/behaviors/TParameter.py:36:1: D105 Missing docstring in magic method
src/uproot/behaviors/TParameter.py:39:1: D105 Missing docstring in magic method
src/uproot/behaviors/TParameter.py:42:1: D105 Missing docstring in magic method
src/uproot/behaviors/TParameter.py:45:1: D105 Missing docstring in magic method
src/uproot/behaviors/TParameter.py:50:1: D200 One-line docstring should fit on one line with quotes
src/uproot/behaviors/TParameter.py:55:1: D102 Missing docstring in public method
src/uproot/behaviors/TParameter.py:58:1: D105 Missing docstring in magic method
src/uproot/behaviors/TParameter.py:61:1: D105 Missing docstring in magic method
src/uproot/behaviors/TParameter.py:64:1: D105 Missing docstring in magic method
src/uproot/behaviors/TParameter.py:69:1: D200 One-line docstring should fit on one line with quotes
src/uproot/behaviors/TParameter.py:69:1: D401 First line should be in imperative mood
src/uproot/exceptions.py:3:1: D205 1 blank line required between summary line and description
src/uproot/exceptions.py:3:1: D400 First line should end with a period
src/uproot/exceptions.py:13:1: D205 1 blank line required between summary line and description
src/uproot/exceptions.py:13:1: D400 First line should end with a period
src/uproot/exceptions.py:32:1: D107 Missing docstring in __init__
src/uproot/exceptions.py:43:1: D105 Missing docstring in magic method
src/uproot/models/TObjArray.py:3:1: D200 One-line docstring should fit on one line with quotes
src/uproot/models/TObjArray.py:30:1: D102 Missing docstring in public method
src/uproot/models/TObjArray.py:84:1: D105 Missing docstring in magic method
src/uproot/models/TObjArray.py:96:1: D105 Missing docstring in magic method
src/uproot/models/TObjArray.py:99:1: D105 Missing docstring in magic method
src/uproot/models/TObjArray.py:102:1: D102 Missing docstring in public method
src/uproot/models/TObjArray.py:114:1: D200 One-line docstring should fit on one line with quotes
src/uproot/models/TObjArray.py:118:1: D102 Missing docstring in public method
src/uproot/writing/_cascade.py:3:1: D205 1 blank line required between summary line and description
src/uproot/writing/_cascade.py:3:1: D400 First line should end with a period
src/uproot/writing/_cascade.py:62:1: D107 Missing docstring in __init__
src/uproot/writing/_cascade.py:68:1: D102 Missing docstring in public method
src/uproot/writing/_cascade.py:78:1: D102 Missing docstring in public method
src/uproot/writing/_cascade.py:84:1: D102 Missing docstring in public method
src/uproot/writing/_cascade.py:87:1: D102 Missing docstring in public method
src/uproot/writing/_cascade.py:91:1: D102 Missing docstring in public method
src/uproot/writing/_cascade.py:96:1: D102 Missing docstring in public method
src/uproot/writing/_cascade.py:125:1: D107 Missing docstring in __init__
src/uproot/writing/_cascade.py:128:1: D102 Missing docstring in public method
src/uproot/writing/_cascade.py:131:1: D102 Missing docstring in public method
src/uproot/writing/_cascade.py:140:1: D205 1 blank line required between summary line and description
src/uproot/writing/_cascade.py:140:1: D301 Use r""" if any backslashes in a docstring
src/uproot/writing/_cascade.py:140:1: D400 First line should end with a period
src/uproot/writing/_cascade.py:149:1: D107 Missing docstring in __init__
src/uproot/writing/_cascade.py:154:1: D105 Missing docstring in magic method
src/uproot/writing/_cascade.py:162:1: D102 Missing docstring in public method
src/uproot/writing/_cascade.py:165:1: D102 Missing docstring in public method
src/uproot/writing/_cascade.py:168:1: D102 Missing docstring in public method
src/uproot/writing/_cascade.py:172:1: D102 Missing docstring in public method
src/uproot/writing/_cascade.py:199:1: D107 Missing docstring in __init__
src/uproot/writing/_cascade.py:225:1: D105 Missing docstring in magic method
src/uproot/writing/_cascade.py:240:1: D102 Missing docstring in public method
src/uproot/writing/_cascade.py:244:1: D102 Missing docstring in public method
src/uproot/writing/_cascade.py:254:1: D102 Missing docstring in public method
src/uproot/writing/_cascade.py:264:1: D102 Missing docstring in public method
src/uproot/writing/_cascade.py:268:1: D102 Missing docstring in public method
src/uproot/writing/_cascade.py:272:1: D102 Missing docstring in public method
src/uproot/writing/_cascade.py:276:1: D102 Missing docstring in public method
src/uproot/writing/_cascade.py:280:1: D102 Missing docstring in public method
src/uproot/writing/_cascade.py:291:1: D102 Missing docstring in public method
src/uproot/writing/_cascade.py:301:1: D102 Missing docstring in public method
src/uproot/writing/_cascade.py:331:1: D102 Missing docstring in public method
src/uproot/writing/_cascade.py:347:1: D102 Missing docstring in public method
src/uproot/writing/_cascade.py:363:1: D102 Missing docstring in public method
src/uproot/writing/_cascade.py:398:1: D102 Missing docstring in public method
src/uproot/writing/_cascade.py:476:1: D200 One-line docstring should fit on one line with quotes
src/uproot/writing/_cascade.py:476:1: D400 First line should end with a period
src/uproot/writing/_cascade.py:482:1: D107 Missing docstring in __init__
src/uproot/writing/_cascade.py:487:1: D105 Missing docstring in magic method
src/uproot/writing/_cascade.py:496:1: D102 Missing docstring in public method
src/uproot/writing/_cascade.py:506:1: D102 Missing docstring in public method
src/uproot/writing/_cascade.py:516:1: D102 Missing docstring in public method
src/uproot/writing/_cascade.py:528:1: D102 Missing docstring in public method
src/uproot/writing/_cascade.py:548:1: D102 Missing docstring in public method
src/uproot/writing/_cascade.py:577:1: D102 Missing docstring in public method
src/uproot/writing/_cascade.py:637:1: D107 Missing docstring in __init__
src/uproot/writing/_cascade.py:643:1: D105 Missing docstring in magic method
src/uproot/writing/_cascade.py:652:1: D102 Missing docstring in public method
src/uproot/writing/_cascade.py:656:1: D102 Missing docstring in public method
src/uproot/writing/_cascade.py:660:1: D102 Missing docstring in public method
src/uproot/writing/_cascade.py:664:1: D102 Missing docstring in public method
src/uproot/writing/_cascade.py:669:1: D102 Missing docstring in public method
src/uproot/writing/_cascade.py:755:1: D102 Missing docstring in public method
src/uproot/writing/_cascade.py:791:1: D102 Missing docstring in public method
src/uproot/writing/_cascade.py:814:1: D107 Missing docstring in __init__
src/uproot/writing/_cascade.py:819:1: D105 Missing docstring in magic method
src/uproot/writing/_cascade.py:825:1: D102 Missing docstring in public method
src/uproot/writing/_cascade.py:835:1: D102 Missing docstring in public method
src/uproot/writing/_cascade.py:845:1: D102 Missing docstring in public method
src/uproot/writing/_cascade.py:848:1: D102 Missing docstring in public method
src/uproot/writing/_cascade.py:861:1: D205 1 blank line required between summary line and description
src/uproot/writing/_cascade.py:861:1: D400 First line should end with a period
src/uproot/writing/_cascade.py:867:1: D107 Missing docstring in __init__
src/uproot/writing/_cascade.py:874:1: D102 Missing docstring in public method
src/uproot/writing/_cascade.py:878:1: D102 Missing docstring in public method
src/uproot/writing/_cascade.py:881:1: D105 Missing docstring in magic method
src/uproot/writing/_cascade.py:890:1: D102 Missing docstring in public method
src/uproot/writing/_cascade.py:895:1: D102 Missing docstring in public method
src/uproot/writing/_cascade.py:908:1: D107 Missing docstring in __init__
src/uproot/writing/_cascade.py:912:1: D105 Missing docstring in magic method
src/uproot/writing/_cascade.py:919:1: D102 Missing docstring in public method
src/uproot/writing/_cascade.py:922:1: D102 Missing docstring in public method
src/uproot/writing/_cascade.py:933:1: D107 Missing docstring in __init__
src/uproot/writing/_cascade.py:946:1: D105 Missing docstring in magic method
src/uproot/writing/_cascade.py:957:1: D102 Missing docstring in public method
src/uproot/writing/_cascade.py:961:1: D102 Missing docstring in public method
src/uproot/writing/_cascade.py:965:1: D102 Missing docstring in public method
src/uproot/writing/_cascade.py:969:1: D102 Missing docstring in public method
src/uproot/writing/_cascade.py:973:1: D102 Missing docstring in public method
src/uproot/writing/_cascade.py:976:1: D102 Missing docstring in public method
src/uproot/writing/_cascade.py:1020:1: D102 Missing docstring in public method
src/uproot/writing/_cascade.py:1047:1: D102 Missing docstring in public method
src/uproot/writing/_cascade.py:1153:1: D205 1 blank line required between summary line and description
src/uproot/writing/_cascade.py:1153:1: D400 First line should end with a period
src/uproot/writing/_cascade.py:1165:1: D107 Missing docstring in __init__
src/uproot/writing/_cascade.py:1178:1: D105 Missing docstring in magic method
src/uproot/writing/_cascade.py:1187:1: D102 Missing docstring in public method
src/uproot/writing/_cascade.py:1198:1: D102 Missing docstring in public method
src/uproot/writing/_cascade.py:1204:1: D102 Missing docstring in public method
src/uproot/writing/_cascade.py:1217:1: D102 Missing docstring in public method
src/uproot/writing/_cascade.py:1233:1: D102 Missing docstring in public method
src/uproot/writing/_cascade.py:1249:1: D102 Missing docstring in public method
src/uproot/writing/_cascade.py:1252:1: D102 Missing docstring in public method
src/uproot/writing/_cascade.py:1255:1: D102 Missing docstring in public method
src/uproot/writing/_cascade.py:1270:1: D102 Missing docstring in public method
src/uproot/writing/_cascade.py:1274:1: D102 Missing docstring in public method
src/uproot/writing/_cascade.py:1278:1: D102 Missing docstring in public method
src/uproot/writing/_cascade.py:1286:1: D102 Missing docstring in public method
src/uproot/writing/_cascade.py:1294:1: D102 Missing docstring in public method
src/uproot/writing/_cascade.py:1298:1: D102 Missing docstring in public method
src/uproot/writing/_cascade.py:1301:1: D102 Missing docstring in public method
src/uproot/writing/_cascade.py:1327:1: D102 Missing docstring in public method
src/uproot/writing/_cascade.py:1334:1: D102 Missing docstring in public method
src/uproot/writing/_cascade.py:1354:1: D205 1 blank line required between summary line and description
src/uproot/writing/_cascade.py:1354:1: D400 First line should end with a period
src/uproot/writing/_cascade.py:1368:1: D107 Missing docstring in __init__
src/uproot/writing/_cascade.py:1390:1: D105 Missing docstring in magic method
src/uproot/writing/_cascade.py:1403:1: D102 Missing docstring in public method
src/uproot/writing/_cascade.py:1413:1: D102 Missing docstring in public method
src/uproot/writing/_cascade.py:1423:1: D102 Missing docstring in public method
src/uproot/writing/_cascade.py:1433:1: D102 Missing docstring in public method
src/uproot/writing/_cascade.py:1443:1: D102 Missing docstring in public method
src/uproot/writing/_cascade.py:1447:1: D102 Missing docstring in public method
src/uproot/writing/_cascade.py:1451:1: D102 Missing docstring in public method
src/uproot/writing/_cascade.py:1461:1: D102 Missing docstring in public method
src/uproot/writing/_cascade.py:1469:1: D102 Missing docstring in public method
src/uproot/writing/_cascade.py:1475:1: D102 Missing docstring in public method
src/uproot/writing/_cascade.py:1501:1: D102 Missing docstring in public method
src/uproot/writing/_cascade.py:1588:1: D102 Missing docstring in public method
src/uproot/writing/_cascade.py:1689:1: D102 Missing docstring in public method
src/uproot/writing/_cascade.py:1773:1: D102 Missing docstring in public method
src/uproot/writing/_cascade.py:1807:1: D107 Missing docstring in __init__
src/uproot/writing/_cascade.py:1817:1: D105 Missing docstring in magic method
src/uproot/writing/_cascade.py:1830:1: D102 Missing docstring in public method
src/uproot/writing/_cascade.py:1834:1: D102 Missing docstring in public method
src/uproot/writing/_cascade.py:1838:1: D102 Missing docstring in public method
src/uproot/writing/_cascade.py:1842:1: D102 Missing docstring in public method
src/uproot/writing/_cascade.py:1846:1: D102 Missing docstring in public method
src/uproot/writing/_cascade.py:1850:1: D102 Missing docstring in public method
src/uproot/writing/_cascade.py:1854:1: D102 Missing docstring in public method
src/uproot/writing/_cascade.py:1858:1: D102 Missing docstring in public method
src/uproot/writing/_cascade.py:1862:1: D102 Missing docstring in public method
src/uproot/writing/_cascade.py:1865:1: D102 Missing docstring in public method
src/uproot/writing/_cascade.py:1892:1: D107 Missing docstring in __init__
src/uproot/writing/_cascade.py:1901:1: D105 Missing docstring in magic method
src/uproot/writing/_cascade.py:1913:1: D102 Missing docstring in public method
src/uproot/writing/_cascade.py:1917:1: D102 Missing docstring in public method
src/uproot/writing/_cascade.py:1921:1: D102 Missing docstring in public method
src/uproot/writing/_cascade.py:1925:1: D102 Missing docstring in public method
src/uproot/writing/_cascade.py:1929:1: D102 Missing docstring in public method
src/uproot/writing/_cascade.py:1933:1: D102 Missing docstring in public method
src/uproot/writing/_cascade.py:1937:1: D102 Missing docstring in public method
src/uproot/writing/_cascade.py:1940:1: D102 Missing docstring in public method
src/uproot/writing/_cascade.py:1973:1: D107 Missing docstring in __init__
src/uproot/writing/_cascade.py:1998:1: D105 Missing docstring in magic method
src/uproot/writing/_cascade.py:2013:1: D102 Missing docstring in public method
src/uproot/writing/_cascade.py:2023:1: D102 Missing docstring in public method
src/uproot/writing/_cascade.py:2033:1: D102 Missing docstring in public method
src/uproot/writing/_cascade.py:2043:1: D102 Missing docstring in public method
src/uproot/writing/_cascade.py:2053:1: D102 Missing docstring in public method
src/uproot/writing/_cascade.py:2063:1: D102 Missing docstring in public method
src/uproot/writing/_cascade.py:2073:1: D102 Missing docstring in public method
src/uproot/writing/_cascade.py:2083:1: D102 Missing docstring in public method
src/uproot/writing/_cascade.py:2093:1: D102 Missing docstring in public method
src/uproot/writing/_cascade.py:2103:1: D102 Missing docstring in public method
src/uproot/writing/_cascade.py:2110:1: D102 Missing docstring in public method
src/uproot/writing/_cascade.py:2114:1: D102 Missing docstring in public method
src/uproot/writing/_cascade.py:2123:1: D102 Missing docstring in public method
src/uproot/writing/_cascade.py:2129:1: D102 Missing docstring in public method
src/uproot/writing/_cascade.py:2162:1: D102 Missing docstring in public method
src/uproot/writing/_cascade.py:2235:1: D107 Missing docstring in __init__
src/uproot/writing/_cascade.py:2249:1: D105 Missing docstring in magic method
src/uproot/writing/_cascade.py:2259:1: D102 Missing docstring in public method
src/uproot/writing/_cascade.py:2263:1: D102 Missing docstring in public method
src/uproot/writing/_cascade.py:2267:1: D102 Missing docstring in public method
src/uproot/writing/_cascade.py:2271:1: D102 Missing docstring in public method
src/uproot/writing/_cascade.py:2275:1: D102 Missing docstring in public method
src/uproot/writing/_cascade.py:2286:1: D200 One-line docstring should fit on one line with quotes
src/uproot/writing/_cascade.py:2286:1: D401 First line should be in imperative mood; try rephrasing
src/uproot/writing/_cascade.py:2405:1: D205 1 blank line required between summary line and description
src/uproot/writing/_cascade.py:2405:1: D400 First line should end with a period
src/uproot/writing/_cascade.py:2405:1: D401 First line should be in imperative mood; try rephrasing
src/uproot/source/xrootd.py:22:1: D205 1 blank line required between summary line and description
src/uproot/source/xrootd.py:22:1: D400 First line should end with a period
src/uproot/source/xrootd.py:72:1: D205 1 blank line required between summary line and description
src/uproot/source/xrootd.py:72:1: D400 First line should end with a period
src/uproot/source/xrootd.py:80:1: D107 Missing docstring in __init__
src/uproot/source/xrootd.py:94:1: D105 Missing docstring in magic method
src/uproot/source/xrootd.py:99:1: D105 Missing docstring in magic method
src/uproot/source/xrootd.py:127:1: D200 One-line docstring should fit on one line with quotes
src/uproot/source/xrootd.py:127:1: D401 First line should be in imperative mood; try rephrasing
src/uproot/source/xrootd.py:134:1: D200 One-line docstring should fit on one line with quotes
src/uproot/source/xrootd.py:134:1: D401 First line should be in imperative mood; try rephrasing
src/uproot/source/xrootd.py:140:1: D102 Missing docstring in public method
src/uproot/source/xrootd.py:147:1: D102 Missing docstring in public method
src/uproot/source/xrootd.py:150:1: D105 Missing docstring in magic method
src/uproot/source/xrootd.py:153:1: D105 Missing docstring in magic method
src/uproot/source/xrootd.py:157:1: D205 1 blank line required between summary line and description
src/uproot/source/xrootd.py:157:1: D400 First line should end with a period
src/uproot/source/xrootd.py:174:1: D205 1 blank line required between summary line and description
src/uproot/source/xrootd.py:174:1: D400 First line should end with a period
src/uproot/source/xrootd.py:194:1: D205 1 blank line required between summary line and description
src/uproot/source/xrootd.py:194:1: D400 First line should end with a period
src/uproot/source/xrootd.py:194:1: D401 First line should be in imperative mood
src/uproot/source/xrootd.py:210:1: D205 1 blank line required between summary line and description
src/uproot/source/xrootd.py:210:1: D400 First line should end with a period
src/uproot/source/xrootd.py:210:1: D401 First line should be in imperative mood
src/uproot/source/xrootd.py:228:1: D205 1 blank line required between summary line and description
src/uproot/source/xrootd.py:228:1: D400 First line should end with a period
src/uproot/source/xrootd.py:228:1: D401 First line should be in imperative mood
src/uproot/source/xrootd.py:251:1: D205 1 blank line required between summary line and description
src/uproot/source/xrootd.py:251:1: D400 First line should end with a period
src/uproot/source/xrootd.py:262:1: D107 Missing docstring in __init__
src/uproot/source/xrootd.py:291:1: D105 Missing docstring in magic method
src/uproot/source/xrootd.py:296:1: D105 Missing docstring in magic method
src/uproot/source/xrootd.py:300:1: D105 Missing docstring in magic method
src/uproot/source/xrootd.py:306:1: D102 Missing docstring in public method
src/uproot/source/xrootd.py:315:1: D102 Missing docstring in public method
src/uproot/source/xrootd.py:395:1: D200 One-line docstring should fit on one line with quotes
src/uproot/source/xrootd.py:395:1: D401 First line should be in imperative mood; try rephrasing
src/uproot/source/xrootd.py:402:1: D200 One-line docstring should fit on one line with quotes
src/uproot/source/xrootd.py:402:1: D401 First line should be in imperative mood; try rephrasing
src/uproot/source/xrootd.py:409:1: D200 One-line docstring should fit on one line with quotes
src/uproot/source/xrootd.py:409:1: D401 First line should be in imperative mood; try rephrasing
src/uproot/source/xrootd.py:415:1: D102 Missing docstring in public method
src/uproot/source/xrootd.py:418:1: D105 Missing docstring in magic method
src/uproot/source/xrootd.py:421:1: D105 Missing docstring in magic method
src/uproot/source/xrootd.py:426:1: D102 Missing docstring in public method
src/uproot/source/xrootd.py:433:1: D205 1 blank line required between summary line and description
src/uproot/source/xrootd.py:433:1: D400 First line should end with a period
src/uproot/source/xrootd.py:444:1: D107 Missing docstring in __init__
src/uproot/source/xrootd.py:463:1: D105 Missing docstring in magic method
src/uproot/source/xrootd.py:468:1: D105 Missing docstring in magic method
src/uproot/source/xrootd.py:474:1: D200 One-line docstring should fit on one line with quotes
src/uproot/source/xrootd.py:474:1: D401 First line should be in imperative mood; try rephrasing
src/uproot/source/xrootd.py:480:1: D102 Missing docstring in public method
src/uproot/models/TClonesArray.py:3:1: D200 One-line docstring should fit on one line with quotes
src/uproot/models/TClonesArray.py:20:1: D102 Missing docstring in public method
src/uproot/models/TClonesArray.py:68:1: D102 Missing docstring in public method
src/uproot/models/TClonesArray.py:72:1: D102 Missing docstring in public method
src/uproot/models/TClonesArray.py:76:1: D102 Missing docstring in public method
src/uproot/models/TClonesArray.py:79:1: D105 Missing docstring in magic method
src/uproot/models/TClonesArray.py:82:1: D105 Missing docstring in magic method
```

</details>